{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from __future__ import annotations\n",
    "\n",
    "import os\n",
    "import pandas as pd\n",
    "import subprocess\n",
    "import numpy as np\n",
    "import matplotlib as mpl\n",
    "import matplotlib.cm as cm\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prepare Data Input\n",
    "\n",
    "You will need ABE8e and evoCDA endogenous data from [Zenodo](https://doi.org/10.5281/zenodo.13737880)\n",
    "- CRISPR-CLEAR-data/data/raw_FASTQs/endogenous_sequencing/ABE8e_pooled.zip\n",
    "- CRISPR-CLEAR-data/data/raw_FASTQs/endogenous_sequencing/EvoCDA_pooled.zip\n",
    "\n",
    "As well as the guide information and mismatch corrected count table from [Zenodo](https://doi.org/10.5281/zenodo.13737880):\n",
    "- CRISPR-CLEAR-data/data/surf_inputs/CD19_guide_info.tsv\n",
    "- CRISPR-CLEAR-data/data/surf_inputs/20230908_CD19_count_matrix.tsv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Filepath Definitions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ABE8E_PRESORT_FASTQ_PATHS = []\n",
    "ABE8E_LOW_FASTQ_PATHS = []\n",
    "ABE8E_HIGH_FASTQ_PATHS = []\n",
    "\n",
    "EVOCDA_PRESORT_FASTQ_PATHS = []\n",
    "EVOCDA_LOW_FASTQ_PATHS = []\n",
    "EVOCDA_HIGH_FASTQ_PATHS = []\n",
    "\n",
    "WORKING_DIR = \"\"\n",
    "RESULTS_DIR = \"\"\n",
    "\n",
    "GUIDE_INFO_PATH = \"\"\n",
    "CORRECTED_COUNT_MATRIX_PATH = \"\"\n",
    "\n",
    "UNCORRECTED_ABE8E_DIR = f\"{RESULTS_DIR}/Uncorrected_ABE8e\"\n",
    "CORRECTED_ABE8E_DIR = f\"{RESULTS_DIR}/Corrected_ABE8e\"\n",
    "os.makedirs(UNCORRECTED_ABE8E_DIR, exist_ok=True)\n",
    "os.makedirs(CORRECTED_ABE8E_DIR, exist_ok=True)\n",
    "\n",
    "UNCORRECTED_EVOCDA_DIR = f\"{RESULTS_DIR}/Uncorrected_evoCDA\"\n",
    "CORRECTED_EVOCDA_DIR = f\"{RESULTS_DIR}/Corrected_evoCDA\"\n",
    "os.makedirs(UNCORRECTED_EVOCDA_DIR, exist_ok=True)\n",
    "os.makedirs(CORRECTED_EVOCDA_DIR, exist_ok=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "def adjust_tracks_for_plotting(\n",
    "    results_path: str,\n",
    "    output_path: str = \"\",\n",
    "):\n",
    "    if output_path == \"\":\n",
    "        output_path = f\"{results_path}tracks_for_plotting/\"\n",
    "    os.makedirs(output_path, exist_ok=True)\n",
    "\n",
    "    try:\n",
    "        pos_regions = pd.read_csv(\n",
    "            f\"{results_path}positive_significant_regions.bed\", header=None, sep=\"\\t\"\n",
    "        )\n",
    "        pos_regions[2] += 1\n",
    "        pos_regions = pos_regions.loc[pos_regions.groupby([0, 1, 2])[3].idxmin()]\n",
    "        pos_regions[4] = 0\n",
    "        pos_regions[5] = \".\"\n",
    "        pos_regions[6] = pos_regions[1]\n",
    "        pos_regions[7] = pos_regions[2]\n",
    "        pos_regions.insert(8, 8, -np.log10(pos_regions[3]))\n",
    "    except pd.errors.EmptyDataError:\n",
    "        pos_regions = pd.DataFrame(columns=[0, 1, 2, 3, 4, 5, 6, 7, 8])\n",
    "\n",
    "    try:\n",
    "        neg_regions = pd.read_csv(\n",
    "            f\"{results_path}negative_significant_regions.bed\", header=None, sep=\"\\t\"\n",
    "        )\n",
    "        neg_regions[2] += 1\n",
    "        neg_regions = neg_regions.loc[neg_regions.groupby([0, 1, 2])[3].idxmin()]\n",
    "        neg_regions[4] = 0\n",
    "        neg_regions[5] = \".\"\n",
    "        neg_regions[6] = neg_regions[1]\n",
    "        neg_regions[7] = neg_regions[2]\n",
    "        neg_regions.insert(8, 8, np.log10(neg_regions[3]))\n",
    "    except pd.errors.EmptyDataError:\n",
    "        neg_regions = pd.DataFrame(columns=[0, 1, 2, 3, 4, 5, 6, 7, 8])\n",
    "\n",
    "    sig_regions = pd.concat([pos_regions, neg_regions], ignore_index=True)\n",
    "    cmap = plt.get_cmap(\"coolwarm\")\n",
    "    cnorm = mpl.colors.Normalize(vmin=np.log10(0.0001), vmax=-np.log10(0.0001))\n",
    "    scalar_map = cm.ScalarMappable(norm=cnorm, cmap=cmap)\n",
    "    sig_regions[8] = sig_regions[8].apply(\n",
    "        lambda x: \",\".join([str(val) for val in scalar_map.to_rgba(x, bytes=True)[:3]])\n",
    "    )\n",
    "    sig_regions.to_csv(\n",
    "        f\"{output_path}significant_regions.bed\", sep=\"\\t\", header=None, index=None\n",
    "    )\n",
    "\n",
    "    raw_scores = pd.read_csv(\n",
    "        f\"{results_path}raw_scores.bedgraph\", header=None, sep=\"\\t\"\n",
    "    )\n",
    "    raw_scores[2] += 1\n",
    "    raw_scores.loc[raw_scores[3] > 3, 3] = 3\n",
    "    raw_scores.loc[raw_scores[3] < -3, 3] = -3\n",
    "    raw_scores[\"rep\"] = raw_scores.groupby([0, 1, 2]).cumcount() + 1\n",
    "    raw_scores = raw_scores[raw_scores[\"rep\"] <= 3]\n",
    "\n",
    "    for rep in raw_scores[\"rep\"].unique():\n",
    "        temp_scores = raw_scores[raw_scores[\"rep\"] == rep].drop(\"rep\", axis=1)\n",
    "        # temp_scores = pd.merge(temp_scores, pvals, left_on=4, right_index=True)\n",
    "        temp_scores.to_csv(\n",
    "            f\"{output_path}rep{rep}_raw_scores.bedgraph\",\n",
    "            sep=\"\\t\",\n",
    "            header=False,\n",
    "            index=False,\n",
    "        )\n",
    "\n",
    "    deconv_scores = pd.read_csv(\n",
    "        f\"{results_path}deconvolved_scores.bedgraph\", header=None, sep=\"\\t\"\n",
    "    )\n",
    "    deconv_scores[2] += 1\n",
    "    deconv_scores.to_csv(\n",
    "        f\"{output_path}deconvolved_scores.bedgraph\",\n",
    "        sep=\"\\t\",\n",
    "        header=False,\n",
    "        index=False,\n",
    "    )\n",
    "\n",
    "    power = pd.read_csv(\n",
    "        f\"{results_path}neglog10_pvals.bedgraph\", header=None, sep=\"\\t\"\n",
    "    )\n",
    "    power[2] += 1\n",
    "    power.to_csv(\n",
    "        f\"{output_path}neglog10_pvals.bedgraph\",\n",
    "        sep=\"\\t\",\n",
    "        header=False,\n",
    "        index=False\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Chr</th>\n",
       "      <th>Start</th>\n",
       "      <th>Stop</th>\n",
       "      <th>sgRNA_Sequence.1</th>\n",
       "      <th>Strand</th>\n",
       "      <th>sgRNA_Type_ABE</th>\n",
       "      <th>sgRNA_Type_CBE</th>\n",
       "      <th>Cutsite</th>\n",
       "      <th>amplicon_position</th>\n",
       "      <th>pam</th>\n",
       "      <th>is_ngn</th>\n",
       "      <th>is_ngg</th>\n",
       "      <th>is_nnn</th>\n",
       "      <th>is_ABE</th>\n",
       "      <th>is_CBE</th>\n",
       "      <th>is_ABE_editable</th>\n",
       "      <th>is_CBE_editable</th>\n",
       "      <th>ABE_explainable</th>\n",
       "      <th>CBE_explainable</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sgRNA_Sequence</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>GGGGAATGACATGCTCTAGT</th>\n",
       "      <td>chr16</td>\n",
       "      <td>28930678.0</td>\n",
       "      <td>28930701.0</td>\n",
       "      <td>GGGGAATGACATGCTCTAGT</td>\n",
       "      <td>+</td>\n",
       "      <td>observation</td>\n",
       "      <td>observation</td>\n",
       "      <td>28930692.0</td>\n",
       "      <td>42.0</td>\n",
       "      <td>GAA</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GAATGACATGCTCTAGTGAA</th>\n",
       "      <td>chr16</td>\n",
       "      <td>28930681.0</td>\n",
       "      <td>28930704.0</td>\n",
       "      <td>GAATGACATGCTCTAGTGAA</td>\n",
       "      <td>+</td>\n",
       "      <td>observation</td>\n",
       "      <td>observation</td>\n",
       "      <td>28930695.0</td>\n",
       "      <td>45.0</td>\n",
       "      <td>AGC</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>TGACATGCTCTAGTGAAAGC</th>\n",
       "      <td>chr16</td>\n",
       "      <td>28930684.0</td>\n",
       "      <td>28930707.0</td>\n",
       "      <td>TGACATGCTCTAGTGAAAGC</td>\n",
       "      <td>+</td>\n",
       "      <td>observation</td>\n",
       "      <td>observation</td>\n",
       "      <td>28930698.0</td>\n",
       "      <td>48.0</td>\n",
       "      <td>CAG</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>CATGCTCTAGTGAAAGCCAG</th>\n",
       "      <td>chr16</td>\n",
       "      <td>28930687.0</td>\n",
       "      <td>28930710.0</td>\n",
       "      <td>CATGCTCTAGTGAAAGCCAG</td>\n",
       "      <td>+</td>\n",
       "      <td>observation</td>\n",
       "      <td>observation</td>\n",
       "      <td>28930701.0</td>\n",
       "      <td>51.0</td>\n",
       "      <td>TCT</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GCTCTAGTGAAAGCCAGTCT</th>\n",
       "      <td>chr16</td>\n",
       "      <td>28930690.0</td>\n",
       "      <td>28930713.0</td>\n",
       "      <td>GCTCTAGTGAAAGCCAGTCT</td>\n",
       "      <td>+</td>\n",
       "      <td>observation</td>\n",
       "      <td>observation</td>\n",
       "      <td>28930704.0</td>\n",
       "      <td>54.0</td>\n",
       "      <td>GGG</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                        Chr       Start        Stop      sgRNA_Sequence.1  \\\n",
       "sgRNA_Sequence                                                              \n",
       "GGGGAATGACATGCTCTAGT  chr16  28930678.0  28930701.0  GGGGAATGACATGCTCTAGT   \n",
       "GAATGACATGCTCTAGTGAA  chr16  28930681.0  28930704.0  GAATGACATGCTCTAGTGAA   \n",
       "TGACATGCTCTAGTGAAAGC  chr16  28930684.0  28930707.0  TGACATGCTCTAGTGAAAGC   \n",
       "CATGCTCTAGTGAAAGCCAG  chr16  28930687.0  28930710.0  CATGCTCTAGTGAAAGCCAG   \n",
       "GCTCTAGTGAAAGCCAGTCT  chr16  28930690.0  28930713.0  GCTCTAGTGAAAGCCAGTCT   \n",
       "\n",
       "                     Strand sgRNA_Type_ABE sgRNA_Type_CBE     Cutsite  \\\n",
       "sgRNA_Sequence                                                          \n",
       "GGGGAATGACATGCTCTAGT      +    observation    observation  28930692.0   \n",
       "GAATGACATGCTCTAGTGAA      +    observation    observation  28930695.0   \n",
       "TGACATGCTCTAGTGAAAGC      +    observation    observation  28930698.0   \n",
       "CATGCTCTAGTGAAAGCCAG      +    observation    observation  28930701.0   \n",
       "GCTCTAGTGAAAGCCAGTCT      +    observation    observation  28930704.0   \n",
       "\n",
       "                      amplicon_position  pam  is_ngn  is_ngg  is_nnn  is_ABE  \\\n",
       "sgRNA_Sequence                                                                 \n",
       "GGGGAATGACATGCTCTAGT               42.0  GAA   False   False    True    True   \n",
       "GAATGACATGCTCTAGTGAA               45.0  AGC    True   False    True    True   \n",
       "TGACATGCTCTAGTGAAAGC               48.0  CAG   False   False    True    True   \n",
       "CATGCTCTAGTGAAAGCCAG               51.0  TCT   False   False    True    True   \n",
       "GCTCTAGTGAAAGCCAGTCT               54.0  GGG    True    True    True    True   \n",
       "\n",
       "                      is_CBE  is_ABE_editable  is_CBE_editable  \\\n",
       "sgRNA_Sequence                                                   \n",
       "GGGGAATGACATGCTCTAGT   False             True            False   \n",
       "GAATGACATGCTCTAGTGAA    True             True            False   \n",
       "TGACATGCTCTAGTGAAAGC    True             True            False   \n",
       "CATGCTCTAGTGAAAGCCAG    True             True            False   \n",
       "GCTCTAGTGAAAGCCAGTCT    True             True             True   \n",
       "\n",
       "                      ABE_explainable  CBE_explainable  \n",
       "sgRNA_Sequence                                          \n",
       "GGGGAATGACATGCTCTAGT             True            False  \n",
       "GAATGACATGCTCTAGTGAA             True            False  \n",
       "TGACATGCTCTAGTGAAAGC             True            False  \n",
       "CATGCTCTAGTGAAAGCCAG             True            False  \n",
       "GCTCTAGTGAAAGCCAGTCT             True             True  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "guide_info = pd.read_csv(GUIDE_INFO_PATH, sep=\"\\t\", index_col=0)\n",
    "display(guide_info.head())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Uncorrected ABE8e SURF Count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "UNCORRECTED_ABE8E_SURF_COLUMNS = [\n",
    "    \"Chr\",\n",
    "    \"Start\",\n",
    "    \"Stop\",\n",
    "    \"Strand\",\n",
    "    \"sgRNA_Type_ABE\",\n",
    "    \"sgRNA_Sequence.1\",\n",
    "]\n",
    "# Prepare FastQs (old)\n",
    "# presort_fastqs = [f\"{ABE8E_FASTQ_DIR}/AAGTAGAG_AAGTAGAG_R1-CD19_presort_1_ABE8.fastq\",\n",
    "#                 f\"{ABE8E_FASTQ_DIR}/AAGTAGAG_CATGATCG_R1-CD19_presort_2_ABE8.fastq\",\n",
    "#                 f\"{ABE8E_FASTQ_DIR}/AAGTAGAG_AACGCATT_R1-CD19_presort_3_ABE8.fastq\",]\n",
    "\n",
    "# low_fastqs = [f\"{ABE8E_FASTQ_DIR}/AAGTAGAG_ACACGATC_R1-CD19_low_1_ABE8.fastq\",\n",
    "#             f\"{ABE8E_FASTQ_DIR}/AAGTAGAG_CGTTACCA_R1-CD19_low_2_ABE8.fastq\",\n",
    "#             f\"{ABE8E_FASTQ_DIR}/AAGTAGAG_AGGTAAGG_R1-CD19_low_3_ABE8.fastq\",]\n",
    "\n",
    "# high_fastqs = [f\"{ABE8E_FASTQ_DIR}/AAGTAGAG_CGCGCGGT_R1-CD19_high_1_ABE8.fastq\",\n",
    "#             f\"{ABE8E_FASTQ_DIR}/AAGTAGAG_TCCTTGGT_R1-CD19_high_2_ABE8.fastq\",\n",
    "#             f\"{ABE8E_FASTQ_DIR}/AAGTAGAG_AACAATGG_R1-CD19_high_3_ABE8.fastq\",]\n",
    "\n",
    "# Set non-explainable guides as negative controls\n",
    "library_file = guide_info[UNCORRECTED_ABE8E_SURF_COLUMNS]\n",
    "library_file.loc[~library_file[\"sgRNA_Sequence.1\"].str[2:9].str.contains(\"A\"), \"sgRNA_Type_ABE\"] = \"negative_control\"\n",
    "\n",
    "library_file.rename(\n",
    "    columns={\n",
    "        \"sgRNA_Type_ABE\": \"sgRNA_Type\",\n",
    "        \"sgRNA_Sequence.1\": \"sgRNA_Sequence\"},\n",
    "    ).to_csv(f\"{UNCORRECTED_ABE8E_DIR}/library_file.csv\", index=False)\n",
    "\n",
    "\n",
    "# Generate Presort vs. Low Counts\n",
    "os.makedirs(f\"{UNCORRECTED_ABE8E_DIR}/presort_vs_low\", exist_ok=True)\n",
    "subprocess.run(\n",
    "        \"module load singularity\\n\"+f\"singularity run -B {WORKING_DIR} docker://jdd056/crisprsurf:vectorization SURF_count -f {UNCORRECTED_ABE8E_DIR}/library_file.csv -control_fastqs {' '.join(ABE8E_PRESORT_FASTQ_PATHS)} -sample_fastqs {' '.join(ABE8E_LOW_FASTQ_PATHS)} -pert be -out_dir {UNCORRECTED_ABE8E_DIR}/presort_vs_low\",\n",
    "        shell=True,\n",
    "        check=True,\n",
    "        stdout=subprocess.PIPE,\n",
    "        stderr=subprocess.PIPE,\n",
    "        )\n",
    "\n",
    "# Filter out guides with less than 500 counts\n",
    "library_file = pd.read_csv(f\"{UNCORRECTED_ABE8E_DIR}/presort_vs_low/sgRNAs_summary_table.csv\")\n",
    "library_file = library_file.loc[(library_file.filter(like=\"_Count\").apply(lambda x: x > 500).all(axis=1)), :]\n",
    "library_file.to_csv(f\"{UNCORRECTED_ABE8E_DIR}/presort_vs_low/sgRNAs_summary_table.csv\", index=False)\n",
    "\n",
    "# Generate Low vs. High Counts\n",
    "os.makedirs(f\"{UNCORRECTED_ABE8E_DIR}/low_vs_high\", exist_ok=True)\n",
    "subprocess.run(\n",
    "        \"module load singularity\\n\"+f\"singularity run -B {WORKING_DIR} docker://jdd056/crisprsurf:vectorization SURF_count -f {UNCORRECTED_ABE8E_DIR}/library_file.csv -control_fastqs {' '.join(ABE8E_LOW_FASTQ_PATHS)} -sample_fastqs {' '.join(ABE8E_HIGH_FASTQ_PATHS)} -pert be -out_dir {UNCORRECTED_ABE8E_DIR}/low_vs_high\",\n",
    "        shell=True,\n",
    "        check=True,\n",
    "        stdout=subprocess.PIPE,\n",
    "        stderr=subprocess.PIPE,\n",
    "        )\n",
    "\n",
    "library_file = pd.read_csv(f\"{UNCORRECTED_ABE8E_DIR}/low_vs_high/sgRNAs_summary_table.csv\")\n",
    "library_file = library_file.loc[(library_file.filter(like=\"_Count\").apply(lambda x: x > 500).all(axis=1)), :]\n",
    "library_file.to_csv(f\"{UNCORRECTED_ABE8E_DIR}/low_vs_high/sgRNAs_summary_table.csv\", index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Uncorrected ABE8e SURF Deconvolution"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "chrom = \"chr16\"\n",
    "start = guide_info[guide_info[\"Chr\"] == chrom][\"Start\"].min()\n",
    "stop = guide_info[guide_info[\"Chr\"] == chrom][\"Stop\"].max()\n",
    "region = f\"{chrom}:{int(start)}-{int(stop)}\"\n",
    "\n",
    "genome = \"hg38\"\n",
    "null_dist = \"negative_control\"\n",
    "pert_range = \"6\"\n",
    "sim_n = 2000\n",
    "\n",
    "for comparison in [\"presort_vs_low\", \"low_vs_high\"]:\n",
    "    subprocess.run(\n",
    "        \"module load singularity\\n\"+f\"singularity run -B {WORKING_DIR} docker://jdd056/crisprsurf:vectorization SURF_deconvolution -f {UNCORRECTED_ABE8E_DIR}/{comparison}/sgRNAs_summary_table.csv -pert be -range {pert_range} -null_dist {null_dist} -genome {genome} -lambda_val 1 -sim_n {sim_n} -out_dir {UNCORRECTED_ABE8E_DIR}/{comparison}/\",\n",
    "        shell=True,\n",
    "        check=True,\n",
    "        stdout=subprocess.PIPE,\n",
    "        stderr=subprocess.PIPE,\n",
    "    )\n",
    "    adjust_tracks_for_plotting(\n",
    "        results_path=f\"{UNCORRECTED_ABE8E_DIR}/{comparison}/\"\n",
    "    )\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Uncorrected evoCDA SURF Count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "UNCORRECTED_EVOCDA_SURF_COLUMNS = [\n",
    "    \"Chr\",\n",
    "    \"Start\",\n",
    "    \"Stop\",\n",
    "    \"Strand\",\n",
    "    \"is_ngg\",\n",
    "    \"CBE_explainable\",\n",
    "    \"sgRNA_Sequence.1\",\n",
    "    \"sgRNA_Type_CBE\",\n",
    "]\n",
    "\n",
    "# Prepare FastQs (old)\n",
    "# FASTQ_BASENAME = \"BSF_1264_HKGCWDRX2_1\"\n",
    "\n",
    "# presort_indices = [20, 23, 26]\n",
    "# low_indices = [22, 25, 28]\n",
    "# high_indices = [21, 24, 27]\n",
    "\n",
    "# presort_fastqs = [f\"{EVOCDA_FASTQ_DIR}/{FASTQ_BASENAME}#p7{EVOCDA_FASTQ_BARCODE_SHEET.iloc[index]['barcode P7 rev']}_p5{EVOCDA_FASTQ_BARCODE_SHEET.iloc[index]['barcode P5 fwd']}.R1.fastq\" for index in presort_indices]\n",
    "# low_fastqs = [f\"{EVOCDA_FASTQ_DIR}/{FASTQ_BASENAME}#p7{EVOCDA_FASTQ_BARCODE_SHEET.iloc[index]['barcode P7 rev']}_p5{EVOCDA_FASTQ_BARCODE_SHEET.iloc[index]['barcode P5 fwd']}.R1.fastq\" for index in low_indices]\n",
    "# high_fastqs = [f\"{EVOCDA_FASTQ_DIR}/{FASTQ_BASENAME}#p7{EVOCDA_FASTQ_BARCODE_SHEET.iloc[index]['barcode P7 rev']}_p5{EVOCDA_FASTQ_BARCODE_SHEET.iloc[index]['barcode P5 fwd']}.R1.fastq\" for index in high_indices]\n",
    "\n",
    "# Set non-explainable guides as negative controls\n",
    "library_file = guide_info.loc[(guide_info[\"CBE_explainable\"]) | (~guide_info[\"is_ngg\"]), UNCORRECTED_EVOCDA_SURF_COLUMNS]\n",
    "library_file.loc[~library_file[\"is_ngg\"], \"sgRNA_Type_CBE\"] = \"negative_control\"\n",
    "library_file.loc[~library_file[\"sgRNA_Sequence.1\"].str[0:14].str.contains(\"C\"), \"sgRNA_Type_CBE\"] = \"negative_control\"\n",
    "\n",
    "library_file.drop(columns=[\"is_ngg\", \"CBE_explainable\"], inplace=True)\n",
    "library_file.rename(\n",
    "    columns={\n",
    "        \"sgRNA_Type_CBE\": \"sgRNA_Type\",\n",
    "        \"sgRNA_Sequence.1\": \"sgRNA_Sequence\"}, inplace=True\n",
    "    )\n",
    "library_file.to_csv(f\"{UNCORRECTED_EVOCDA_DIR}/library_file.csv\", index=False)\n",
    "\n",
    "# Generate Presort vs. Low Counts\n",
    "os.makedirs(f\"{UNCORRECTED_EVOCDA_DIR}/presort_vs_low\", exist_ok=True)\n",
    "subprocess.run(\n",
    "        \"module load singularity\\n\"+f\"singularity run -B {WORKING_DIR} docker://jdd056/crisprsurf:vectorization SURF_count -f {UNCORRECTED_EVOCDA_DIR}/library_file.csv -control_fastqs {' '.join(EVOCDA_PRESORT_FASTQ_PATHS)} -sample_fastqs {' '.join(EVOCDA_LOW_FASTQ_PATHS)} -pert be -out_dir {UNCORRECTED_EVOCDA_DIR}/presort_vs_low\",\n",
    "        shell=True,\n",
    "        check=True,\n",
    "        stdout=subprocess.PIPE,\n",
    "        stderr=subprocess.PIPE,\n",
    "        )\n",
    "\n",
    "#Filter out guides with less than 500 counts\n",
    "library_file = pd.read_csv(f\"{UNCORRECTED_EVOCDA_DIR}/presort_vs_low/sgRNAs_summary_table.csv\")\n",
    "library_file = library_file.loc[(library_file.filter(like=\"_Count\").apply(lambda x: x > 500).all(axis=1)), :]\n",
    "library_file.to_csv(f\"{UNCORRECTED_EVOCDA_DIR}/presort_vs_low/sgRNAs_summary_table.csv\", index=False)\n",
    "\n",
    "# Generate Low vs. High Counts\n",
    "os.makedirs(f\"{UNCORRECTED_EVOCDA_DIR}/low_vs_high\", exist_ok=True)\n",
    "subprocess.run(\n",
    "        \"module load singularity\\n\"+f\"singularity run -B {WORKING_DIR} docker://jdd056/crisprsurf:vectorization SURF_count -f {UNCORRECTED_EVOCDA_DIR}/library_file.csv -control_fastqs {' '.join(EVOCDA_LOW_FASTQ_PATHS)} -sample_fastqs {' '.join(EVOCDA_HIGH_FASTQ_PATHS)} -pert be -out_dir {UNCORRECTED_EVOCDA_DIR}/low_vs_high\",\n",
    "        shell=True,\n",
    "        check=True,\n",
    "        stdout=subprocess.PIPE,\n",
    "        stderr=subprocess.PIPE,\n",
    "        )\n",
    "\n",
    "#Filter out guides with less than 500 counts\n",
    "library_file = pd.read_csv(f\"{UNCORRECTED_EVOCDA_DIR}/low_vs_high/sgRNAs_summary_table.csv\")\n",
    "library_file = library_file.loc[(library_file.filter(like=\"_Count\").apply(lambda x: x > 500).all(axis=1)), :]\n",
    "library_file.to_csv(f\"{UNCORRECTED_EVOCDA_DIR}/low_vs_high/sgRNAs_summary_table.csv\", index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Uncorrected evoCDA SURF Deconvolution"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "chrom = \"chr16\"\n",
    "start = guide_info[guide_info[\"Chr\"] == chrom][\"Start\"].min()\n",
    "stop = guide_info[guide_info[\"Chr\"] == chrom][\"Stop\"].max()\n",
    "region = f\"{chrom}:{int(start)}-{int(stop)}\"\n",
    "\n",
    "lambdas = [0.2, 0.5, 1, 10, 30, 50]\n",
    "genome = \"hg38\"\n",
    "null_dist = \"negative_control\"\n",
    "pert_range = 13\n",
    "sim_n = 2000\n",
    "\n",
    "for comparison in [\"presort_vs_low\", \"low_vs_high\"]:\n",
    "    subprocess.run(\n",
    "        \"module load singularity\\n\"+f\"singularity run -B {WORKING_DIR} docker://jdd056/crisprsurf:vectorization SURF_deconvolution -f {UNCORRECTED_EVOCDA_DIR}/{comparison}/sgRNAs_summary_table.csv -pert be -range {pert_range} -null_dist {null_dist} -genome {genome} -lambda_val 1 -sim_n {sim_n} -out_dir {UNCORRECTED_EVOCDA_DIR}/{comparison}/\",\n",
    "        shell=True,\n",
    "        check=True,\n",
    "        stdout=subprocess.PIPE,\n",
    "        stderr=subprocess.PIPE,\n",
    "    )\n",
    "    adjust_tracks_for_plotting(\n",
    "        results_path=f\"{UNCORRECTED_EVOCDA_DIR}/{comparison}/\"\n",
    "    )\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Mismatch Corrected Data Cleaning"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ABE8e_presort_1</th>\n",
       "      <th>ABE8e_presort_2</th>\n",
       "      <th>ABE8e_presort_3</th>\n",
       "      <th>ABE8e_low_1</th>\n",
       "      <th>ABE8e_low_2</th>\n",
       "      <th>ABE8e_low_3</th>\n",
       "      <th>ABE8e_high_1</th>\n",
       "      <th>ABE8e_high_2</th>\n",
       "      <th>ABE8e_high_3</th>\n",
       "      <th>evoCDA_presort_1</th>\n",
       "      <th>evoCDA_presort_2</th>\n",
       "      <th>evoCDA_presort_3</th>\n",
       "      <th>evoCDA_low_1</th>\n",
       "      <th>evoCDA_low_2</th>\n",
       "      <th>evoCDA_low_3</th>\n",
       "      <th>evoCDA_high_1</th>\n",
       "      <th>evoCDA_high_2</th>\n",
       "      <th>evoCDA_high_3</th>\n",
       "      <th>None_plasmid_None</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>GGGGAATGACATGCTCTAGT</th>\n",
       "      <td>24716.0</td>\n",
       "      <td>26162.0</td>\n",
       "      <td>42719.0</td>\n",
       "      <td>44645.0</td>\n",
       "      <td>27974.0</td>\n",
       "      <td>29535.0</td>\n",
       "      <td>19093.0</td>\n",
       "      <td>26987.0</td>\n",
       "      <td>29450.0</td>\n",
       "      <td>21567.0</td>\n",
       "      <td>21371.0</td>\n",
       "      <td>19518.0</td>\n",
       "      <td>21613.0</td>\n",
       "      <td>18859.0</td>\n",
       "      <td>10665.0</td>\n",
       "      <td>15123.0</td>\n",
       "      <td>21071.0</td>\n",
       "      <td>9354.0</td>\n",
       "      <td>31089.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GAATGACATGCTCTAGTGAA</th>\n",
       "      <td>11750.0</td>\n",
       "      <td>28955.0</td>\n",
       "      <td>15677.0</td>\n",
       "      <td>30217.0</td>\n",
       "      <td>23626.0</td>\n",
       "      <td>12471.0</td>\n",
       "      <td>34673.0</td>\n",
       "      <td>21088.0</td>\n",
       "      <td>29720.0</td>\n",
       "      <td>9854.0</td>\n",
       "      <td>12133.0</td>\n",
       "      <td>12043.0</td>\n",
       "      <td>20517.0</td>\n",
       "      <td>24400.0</td>\n",
       "      <td>13395.0</td>\n",
       "      <td>13876.0</td>\n",
       "      <td>8678.0</td>\n",
       "      <td>10417.0</td>\n",
       "      <td>20674.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>TGACATGCTCTAGTGAAAGC</th>\n",
       "      <td>7392.0</td>\n",
       "      <td>19317.0</td>\n",
       "      <td>20414.0</td>\n",
       "      <td>62691.0</td>\n",
       "      <td>31555.0</td>\n",
       "      <td>27629.0</td>\n",
       "      <td>9114.0</td>\n",
       "      <td>28375.0</td>\n",
       "      <td>42553.0</td>\n",
       "      <td>21156.0</td>\n",
       "      <td>18809.0</td>\n",
       "      <td>17363.0</td>\n",
       "      <td>14987.0</td>\n",
       "      <td>20133.0</td>\n",
       "      <td>4850.0</td>\n",
       "      <td>11327.0</td>\n",
       "      <td>23534.0</td>\n",
       "      <td>14495.0</td>\n",
       "      <td>30476.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>CATGCTCTAGTGAAAGCCAG</th>\n",
       "      <td>44819.0</td>\n",
       "      <td>71795.0</td>\n",
       "      <td>16880.0</td>\n",
       "      <td>50966.0</td>\n",
       "      <td>38807.0</td>\n",
       "      <td>40022.0</td>\n",
       "      <td>43136.0</td>\n",
       "      <td>57876.0</td>\n",
       "      <td>49008.0</td>\n",
       "      <td>35697.0</td>\n",
       "      <td>33642.0</td>\n",
       "      <td>27713.0</td>\n",
       "      <td>32942.0</td>\n",
       "      <td>20695.0</td>\n",
       "      <td>18093.0</td>\n",
       "      <td>20641.0</td>\n",
       "      <td>28744.0</td>\n",
       "      <td>13410.0</td>\n",
       "      <td>39160.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GCTCTAGTGAAAGCCAGTCT</th>\n",
       "      <td>100774.0</td>\n",
       "      <td>49923.0</td>\n",
       "      <td>46733.0</td>\n",
       "      <td>98625.0</td>\n",
       "      <td>48256.0</td>\n",
       "      <td>81950.0</td>\n",
       "      <td>73324.0</td>\n",
       "      <td>56437.0</td>\n",
       "      <td>71415.0</td>\n",
       "      <td>30653.0</td>\n",
       "      <td>27542.0</td>\n",
       "      <td>28107.0</td>\n",
       "      <td>32660.0</td>\n",
       "      <td>30404.0</td>\n",
       "      <td>33348.0</td>\n",
       "      <td>17810.0</td>\n",
       "      <td>23773.0</td>\n",
       "      <td>15340.0</td>\n",
       "      <td>38905.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                      ABE8e_presort_1  ABE8e_presort_2  ABE8e_presort_3  \\\n",
       "GGGGAATGACATGCTCTAGT          24716.0          26162.0          42719.0   \n",
       "GAATGACATGCTCTAGTGAA          11750.0          28955.0          15677.0   \n",
       "TGACATGCTCTAGTGAAAGC           7392.0          19317.0          20414.0   \n",
       "CATGCTCTAGTGAAAGCCAG          44819.0          71795.0          16880.0   \n",
       "GCTCTAGTGAAAGCCAGTCT         100774.0          49923.0          46733.0   \n",
       "\n",
       "                      ABE8e_low_1  ABE8e_low_2  ABE8e_low_3  ABE8e_high_1  \\\n",
       "GGGGAATGACATGCTCTAGT      44645.0      27974.0      29535.0       19093.0   \n",
       "GAATGACATGCTCTAGTGAA      30217.0      23626.0      12471.0       34673.0   \n",
       "TGACATGCTCTAGTGAAAGC      62691.0      31555.0      27629.0        9114.0   \n",
       "CATGCTCTAGTGAAAGCCAG      50966.0      38807.0      40022.0       43136.0   \n",
       "GCTCTAGTGAAAGCCAGTCT      98625.0      48256.0      81950.0       73324.0   \n",
       "\n",
       "                      ABE8e_high_2  ABE8e_high_3  evoCDA_presort_1  \\\n",
       "GGGGAATGACATGCTCTAGT       26987.0       29450.0           21567.0   \n",
       "GAATGACATGCTCTAGTGAA       21088.0       29720.0            9854.0   \n",
       "TGACATGCTCTAGTGAAAGC       28375.0       42553.0           21156.0   \n",
       "CATGCTCTAGTGAAAGCCAG       57876.0       49008.0           35697.0   \n",
       "GCTCTAGTGAAAGCCAGTCT       56437.0       71415.0           30653.0   \n",
       "\n",
       "                      evoCDA_presort_2  evoCDA_presort_3  evoCDA_low_1  \\\n",
       "GGGGAATGACATGCTCTAGT           21371.0           19518.0       21613.0   \n",
       "GAATGACATGCTCTAGTGAA           12133.0           12043.0       20517.0   \n",
       "TGACATGCTCTAGTGAAAGC           18809.0           17363.0       14987.0   \n",
       "CATGCTCTAGTGAAAGCCAG           33642.0           27713.0       32942.0   \n",
       "GCTCTAGTGAAAGCCAGTCT           27542.0           28107.0       32660.0   \n",
       "\n",
       "                      evoCDA_low_2  evoCDA_low_3  evoCDA_high_1  \\\n",
       "GGGGAATGACATGCTCTAGT       18859.0       10665.0        15123.0   \n",
       "GAATGACATGCTCTAGTGAA       24400.0       13395.0        13876.0   \n",
       "TGACATGCTCTAGTGAAAGC       20133.0        4850.0        11327.0   \n",
       "CATGCTCTAGTGAAAGCCAG       20695.0       18093.0        20641.0   \n",
       "GCTCTAGTGAAAGCCAGTCT       30404.0       33348.0        17810.0   \n",
       "\n",
       "                      evoCDA_high_2  evoCDA_high_3  None_plasmid_None  \n",
       "GGGGAATGACATGCTCTAGT        21071.0         9354.0            31089.0  \n",
       "GAATGACATGCTCTAGTGAA         8678.0        10417.0            20674.0  \n",
       "TGACATGCTCTAGTGAAAGC        23534.0        14495.0            30476.0  \n",
       "CATGCTCTAGTGAAAGCCAG        28744.0        13410.0            39160.0  \n",
       "GCTCTAGTGAAAGCCAGTCT        23773.0        15340.0            38905.0  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "corrected_count_matrix = pd.read_csv(CORRECTED_COUNT_MATRIX_PATH, sep=\"\\t\", index_col=0\n",
    ").T\n",
    "\n",
    "display(corrected_count_matrix.head())\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Mismatch Corrected ABE8e SURF Count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "CORRECTED_ABE8E_SURF_COLUMNS = [\n",
    "    \"Chr\",\n",
    "    \"Start\",\n",
    "    \"Stop\",\n",
    "    \"Strand\",\n",
    "    \"sgRNA_Type_ABE\",\n",
    "    \"sgRNA_Sequence.1\",\n",
    "]\n",
    "\n",
    "# Set non-explainable guides as negative controls\n",
    "corrected_ABE8e_guides = guide_info.loc[guide_info[\"ABE_explainable\"], CORRECTED_ABE8E_SURF_COLUMNS]\n",
    "corrected_ABE8e_guides.loc[~corrected_ABE8e_guides[\"sgRNA_Sequence.1\"].str[2:9].str.contains(\"A\"), \"sgRNA_Type_ABE\"] = \"negative_control\"\n",
    "\n",
    "corrected_ABE8e_presort_vs_low = pd.merge(\n",
    "    corrected_ABE8e_guides,\n",
    "    corrected_count_matrix[\n",
    "        [\n",
    "            \"ABE8e_presort_1\",\n",
    "            \"ABE8e_presort_2\",\n",
    "            \"ABE8e_presort_3\",\n",
    "            \"ABE8e_low_1\",\n",
    "            \"ABE8e_low_2\",\n",
    "            \"ABE8e_low_3\",\n",
    "        ]\n",
    "    ],\n",
    "    left_index=True,\n",
    "    right_index=True,\n",
    ")\n",
    "corrected_ABE8e_presort_vs_low.reset_index(inplace=True, drop=True)\n",
    "corrected_ABE8e_presort_vs_low.rename(\n",
    "    columns={\n",
    "        \"sgRNA_Type_ABE\": \"sgRNA_Type\",\n",
    "        \"sgRNA_Sequence.1\": \"sgRNA_Sequence\",\n",
    "        \"ABE8e_presort_1\": \"Replicate1_Control_Count\",\n",
    "        \"ABE8e_presort_2\": \"Replicate2_Control_Count\",\n",
    "        \"ABE8e_presort_3\": \"Replicate3_Control_Count\",\n",
    "        \"ABE8e_low_1\": \"Replicate1_Sample_Count\",\n",
    "        \"ABE8e_low_2\": \"Replicate2_Sample_Count\",\n",
    "        \"ABE8e_low_3\": \"Replicate3_Sample_Count\",\n",
    "    },\n",
    "    inplace=True,\n",
    ")\n",
    "\n",
    "# Filter out guides with less than 500 counts\n",
    "corrected_ABE8e_presort_vs_low = corrected_ABE8e_presort_vs_low.loc[(corrected_ABE8e_presort_vs_low.filter(like=\"_Count\").apply(lambda x: x > 500).all(axis=1)), :]\n",
    "\n",
    "os.makedirs(f\"{CORRECTED_ABE8E_DIR}/presort_vs_low\", exist_ok=True)\n",
    "corrected_ABE8e_presort_vs_low.to_csv(\n",
    "    f\"{CORRECTED_ABE8E_DIR}/presort_vs_low/presort_vs_low.csv\", index=False\n",
    ")\n",
    "\n",
    "corrected_ABE8e_low_vs_high = pd.merge(\n",
    "    corrected_ABE8e_guides,\n",
    "    corrected_count_matrix[\n",
    "        [\n",
    "            \"ABE8e_low_1\",\n",
    "            \"ABE8e_low_2\",\n",
    "            \"ABE8e_low_3\",\n",
    "            \"ABE8e_high_1\",\n",
    "            \"ABE8e_high_2\",\n",
    "            \"ABE8e_high_3\",\n",
    "        ]\n",
    "    ],\n",
    "    left_index=True,\n",
    "    right_index=True,\n",
    ")\n",
    "corrected_ABE8e_low_vs_high.reset_index(inplace=True, drop=True)\n",
    "corrected_ABE8e_low_vs_high.rename(\n",
    "    columns={\n",
    "        \"sgRNA_Type_ABE\": \"sgRNA_Type\",\n",
    "        \"sgRNA_Sequence.1\": \"sgRNA_Sequence\",\n",
    "        \"ABE8e_low_1\": \"Replicate1_Control_Count\",\n",
    "        \"ABE8e_low_2\": \"Replicate2_Control_Count\",\n",
    "        \"ABE8e_low_3\": \"Replicate3_Control_Count\",\n",
    "        \"ABE8e_high_1\": \"Replicate1_Sample_Count\",\n",
    "        \"ABE8e_high_2\": \"Replicate2_Sample_Count\",\n",
    "        \"ABE8e_high_3\": \"Replicate3_Sample_Count\",\n",
    "    },\n",
    "    inplace=True,\n",
    ")\n",
    "\n",
    "# Filter out guides with less than 500 counts\n",
    "corrected_ABE8e_low_vs_high = corrected_ABE8e_low_vs_high.loc[(corrected_ABE8e_low_vs_high.filter(like=\"_Count\").apply(lambda x: x > 500).all(axis=1)), :]\n",
    "\n",
    "os.makedirs(f\"{CORRECTED_ABE8E_DIR}/low_vs_high\", exist_ok=True)\n",
    "corrected_ABE8e_low_vs_high.to_csv(\n",
    "    f\"{CORRECTED_ABE8E_DIR}/low_vs_high/low_vs_high.csv\", index=False\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "for comparison in [\"presort_vs_low\", \"low_vs_high\"]:\n",
    "    subprocess.run(\n",
    "        \"module load singularity\\n\"+f\"singularity run -B {WORKING_DIR} docker://jdd056/crisprsurf:vectorization SURF_count -f {CORRECTED_ABE8E_DIR}/{comparison}/{comparison}.csv -pert be -out_dir {CORRECTED_ABE8E_DIR}/{comparison}\",\n",
    "        shell=True,\n",
    "        check=True,\n",
    "        stdout=subprocess.PIPE,\n",
    "        stderr=subprocess.PIPE,\n",
    "    )"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Mismatch Corrected ABE8e SURF Deconvolution"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "chrom = \"chr16\"\n",
    "start = guide_info[guide_info[\"Chr\"] == chrom][\"Start\"].min()\n",
    "stop = guide_info[guide_info[\"Chr\"] == chrom][\"Stop\"].max()\n",
    "region = f\"{chrom}:{int(start)}-{int(stop)}\"\n",
    "\n",
    "genome = \"hg38\"\n",
    "null_dist = \"negative_control\"\n",
    "pert_range = 6\n",
    "sim_n = 2000\n",
    "\n",
    "for comparison in [\"presort_vs_low\", \"low_vs_high\"]:\n",
    "    subprocess.run(\n",
    "        \"module load singularity\\n\"+f\"singularity run -B {WORKING_DIR} docker://jdd056/crisprsurf:vectorization SURF_deconvolution -f {CORRECTED_ABE8E_DIR}/{comparison}/sgRNAs_summary_table.csv -pert be -range {pert_range} -null_dist {null_dist} -genome {genome} -lambda_val 1 -sim_n {sim_n} -out_dir {CORRECTED_ABE8E_DIR}/{comparison}/\",\n",
    "        shell=True,\n",
    "        check=True,\n",
    "        stdout=subprocess.PIPE,\n",
    "        stderr=subprocess.PIPE,\n",
    "\n",
    "    )\n",
    "    adjust_tracks_for_plotting(\n",
    "        results_path=f\"{CORRECTED_ABE8E_DIR}/{comparison}/\"\n",
    "    )"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Mismatch Corrected evoCDA SURF Count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "CORRECTED_EVOCDA_SURF_COLUMNS = [\n",
    "    \"Chr\",\n",
    "    \"Start\",\n",
    "    \"Stop\",\n",
    "    \"Strand\",\n",
    "    \"sgRNA_Type_CBE\",\n",
    "    \"sgRNA_Sequence.1\",\n",
    "]\n",
    "\n",
    "# Set non-explainable guides as negative controls\n",
    "corrected_evoCDA_guides = guide_info.loc[(guide_info[\"CBE_explainable\"]) | (~guide_info[\"is_ngg\"]), CORRECTED_EVOCDA_SURF_COLUMNS]\n",
    "corrected_evoCDA_guides.loc[~guide_info[\"is_ngg\"], \"sgRNA_Type_CBE\"] = \"negative_control\"\n",
    "corrected_evoCDA_guides.loc[~corrected_evoCDA_guides[\"sgRNA_Sequence.1\"].str[0:14].str.contains(\"C\"), \"sgRNA_Type_CBE\"] = \"negative_control\"\n",
    "\n",
    "corrected_evoCDA_presort_vs_low = pd.merge(\n",
    "    corrected_evoCDA_guides,\n",
    "    corrected_count_matrix[\n",
    "        [\n",
    "            \"evoCDA_presort_1\",\n",
    "            \"evoCDA_presort_2\",\n",
    "            \"evoCDA_presort_3\",\n",
    "            \"evoCDA_low_1\",\n",
    "            \"evoCDA_low_2\",\n",
    "            \"evoCDA_low_3\",\n",
    "        ]\n",
    "    ],\n",
    "    left_index=True,\n",
    "    right_index=True,\n",
    ")\n",
    "corrected_evoCDA_presort_vs_low.reset_index(inplace=True, drop=True)\n",
    "corrected_evoCDA_presort_vs_low.rename(\n",
    "    columns={\n",
    "        \"sgRNA_Type_CBE\": \"sgRNA_Type\",\n",
    "        \"sgRNA_Sequence.1\": \"sgRNA_Sequence\",\n",
    "        \"evoCDA_presort_1\": \"Replicate1_Control_Count\",\n",
    "        \"evoCDA_presort_2\": \"Replicate2_Control_Count\",\n",
    "        \"evoCDA_presort_3\": \"Replicate3_Control_Count\",\n",
    "        \"evoCDA_low_1\": \"Replicate1_Sample_Count\",\n",
    "        \"evoCDA_low_2\": \"Replicate2_Sample_Count\",\n",
    "        \"evoCDA_low_3\": \"Replicate3_Sample_Count\",\n",
    "    },\n",
    "    inplace=True,\n",
    ")\n",
    "\n",
    "corrected_evoCDA_presort_vs_low = corrected_evoCDA_presort_vs_low.loc[(corrected_evoCDA_presort_vs_low.filter(like=\"_Count\").apply(lambda x: x > 500).all(axis=1)), :]\n",
    "\n",
    "os.makedirs(f\"{CORRECTED_EVOCDA_DIR}/presort_vs_low\", exist_ok=True)\n",
    "corrected_evoCDA_presort_vs_low.to_csv(\n",
    "    f\"{CORRECTED_EVOCDA_DIR}/presort_vs_low/presort_vs_low.csv\", index=False\n",
    ")\n",
    "\n",
    "corrected_evoCDA_low_vs_high = pd.merge(\n",
    "    corrected_evoCDA_guides,\n",
    "    corrected_count_matrix[\n",
    "        [\n",
    "            \"evoCDA_low_1\",\n",
    "            \"evoCDA_low_2\",\n",
    "            \"evoCDA_low_3\",\n",
    "            \"evoCDA_high_1\",\n",
    "            \"evoCDA_high_2\",\n",
    "            \"evoCDA_high_3\",\n",
    "        ]\n",
    "    ],\n",
    "    left_index=True,\n",
    "    right_index=True,\n",
    ")\n",
    "corrected_evoCDA_low_vs_high.reset_index(inplace=True, drop=True)\n",
    "corrected_evoCDA_low_vs_high.rename(\n",
    "    columns={\n",
    "        \"sgRNA_Type_CBE\": \"sgRNA_Type\",\n",
    "        \"sgRNA_Sequence.1\": \"sgRNA_Sequence\",\n",
    "        \"evoCDA_low_1\": \"Replicate1_Control_Count\",\n",
    "        \"evoCDA_low_2\": \"Replicate2_Control_Count\",\n",
    "        \"evoCDA_low_3\": \"Replicate3_Control_Count\",\n",
    "        \"evoCDA_high_1\": \"Replicate1_Sample_Count\",\n",
    "        \"evoCDA_high_2\": \"Replicate2_Sample_Count\",\n",
    "        \"evoCDA_high_3\": \"Replicate3_Sample_Count\",\n",
    "    },\n",
    "    inplace=True,\n",
    ")\n",
    "\n",
    "corrected_evoCDA_low_vs_high = corrected_evoCDA_low_vs_high.loc[(corrected_evoCDA_low_vs_high.filter(like=\"_Count\").apply(lambda x: x > 500).all(axis=1)), :]\n",
    "\n",
    "os.makedirs(f\"{CORRECTED_EVOCDA_DIR}/low_vs_high\", exist_ok=True)\n",
    "corrected_evoCDA_low_vs_high.to_csv(\n",
    "    f\"{CORRECTED_EVOCDA_DIR}/low_vs_high/low_vs_high.csv\", index=False\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "for comparison in [\"presort_vs_low\", \"low_vs_high\"]:\n",
    "    subprocess.run(\n",
    "        \"module load singularity\\n\"+f\"singularity run -B {WORKING_DIR} docker://jdd056/crisprsurf:vectorization SURF_count -f {CORRECTED_EVOCDA_DIR}/{comparison}/{comparison}.csv -pert be -out_dir {CORRECTED_EVOCDA_DIR}/{comparison}\",\n",
    "        shell=True,\n",
    "        check=True,\n",
    "        stdout=subprocess.PIPE,\n",
    "        stderr=subprocess.PIPE,\n",
    "    )"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Mismatch Corrected evoCDA SURF Deconvolution"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "chrom = \"chr16\"\n",
    "start = guide_info[guide_info[\"Chr\"] == chrom][\"Start\"].min()\n",
    "stop = guide_info[guide_info[\"Chr\"] == chrom][\"Stop\"].max()\n",
    "region = f\"{chrom}:{int(start)}-{int(stop)}\"\n",
    "\n",
    "genome = \"hg38\"\n",
    "null_dist = \"negative_control\"\n",
    "pert_range = 13\n",
    "sim_n=2000\n",
    "\n",
    "for comparison in [\"presort_vs_low\", \"low_vs_high\"]:\n",
    "    subprocess.run(\n",
    "        \"module load singularity\\n\"+f\"singularity run -B {WORKING_DIR} docker://jdd056/crisprsurf:vectorization SURF_deconvolution -f {CORRECTED_EVOCDA_DIR}/{comparison}/sgRNAs_summary_table.csv -pert be -range {pert_range} -null_dist {null_dist} -genome {genome} -lambda_val 1  -sim_n {sim_n} -out_dir {CORRECTED_EVOCDA_DIR}/{comparison}/\",\n",
    "        shell=True,\n",
    "        check=True,\n",
    "        stdout=subprocess.PIPE,\n",
    "        stderr=subprocess.PIPE,\n",
    "\n",
    "    )\n",
    "    adjust_tracks_for_plotting(\n",
    "        results_path=f\"{CORRECTED_EVOCDA_DIR}/{comparison}/\"\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Add Negative Control Points\n",
    "for screen in [\"Uncorrected_evoCDA\", \"Uncorrected_ABE8e\", \"Corrected_evoCDA\", \"Corrected_ABE8e\"]:\n",
    "    guides_file = pd.read_csv(f\"{RESULTS_DIR}/{screen}/low_vs_high/sgRNAs_summary_table_updated.csv\")\n",
    "    guides_file = guides_file[guides_file[\"Chr\"] == \"chr16\"]\n",
    "    neg_controls = guides_file[guides_file[\"sgRNA_Type\"] == \"negative_control\"]\n",
    "    neg_controls = pd.concat([neg_controls[[\"Chr\", \"Start\", \"Stop\", \"Log2FC_Replicate1\", \"sgRNA_Sequence\"]].rename(columns={\"Log2FC_Replicate1\":\"Score\"}),neg_controls[[\"Chr\", \"Start\", \"Stop\", \"Log2FC_Replicate2\", \"sgRNA_Sequence\"]].rename(columns={\"Log2FC_Replicate2\":\"Score\"}), neg_controls[[\"Chr\", \"Start\", \"Stop\", \"Log2FC_Replicate3\", \"sgRNA_Sequence\"]].rename(columns={\"Log2FC_Replicate3\":\"Score\"})])\n",
    "    neg_controls[\"Start\"] = guides_file[\"Perturbation_Index\"].astype(int)\n",
    "    neg_controls[\"Stop\"] = (guides_file[\"Perturbation_Index\"]+1).astype(int)\n",
    "    neg_controls.to_csv(f\"{RESULTS_DIR}/{screen}/low_vs_high/tracks_for_plotting/neg_raw_scores.bedgraph\", index=False, sep=\"\\t\", header=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Optional: sgRNAs per Significant Region"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "import glob\n",
    "import shutil\n",
    "guide_info = pd.read_csv(GUIDE_INFO_PATH, sep=\"\\t\", index_col=0).drop(columns=\"sgRNA_Sequence.1\")\n",
    "\n",
    "for sig_region_path in glob.glob(f\"{RESULTS_DIR}/*/*/significant_regions.csv\"):\n",
    "    if os.path.exists(sig_region_path.replace(\"significant_regions.csv\", \"significant_regions\")):\n",
    "        shutil.rmtree(sig_region_path.replace(\"significant_regions.csv\", \"significant_regions\"))\n",
    "    os.makedirs(sig_region_path.replace(\"significant_regions.csv\", \"significant_regions\"), exist_ok=True)\n",
    "\n",
    "    library_file = pd.read_csv(sig_region_path.replace(\"significant_regions.csv\", \"sgRNAs_summary_table_updated.csv\"), index_col=\"sgRNA_Sequence\")\n",
    "    if \"Unnamed: 0\" in library_file.columns:\n",
    "        library_file.drop(columns=\"Unnamed: 0\")\n",
    "\n",
    "    with open(sig_region_path, \"r\") as sig_region_file:\n",
    "        sig_regions = sig_region_file.readlines()[1:]\n",
    "        if len(sig_regions) == 0:\n",
    "            continue\n",
    "        for idx, sig_region in enumerate(sig_regions):\n",
    "            sig_guides = sig_region.strip().split(\",\")[9:]\n",
    "            sig_guide_df = pd.concat([guide_info.loc[sig_guides], library_file.loc[sig_guides].iloc[:,6:]], axis=1)\n",
    "            sig_guide_df.to_csv(sig_region_path.replace(\"significant_regions.csv\", f\"significant_regions/sig_region_{idx}_{sig_region.strip().split(',')[4]}.csv\"))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
