{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "08cd525e",
   "metadata": {},
   "source": [
    "# Run CRISPResso2 on ABE8e sg218 Data"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "14763fbd",
   "metadata": {},
   "source": [
    "### Import Packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "e1e13dcd",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "46327532",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Install BioPython \"conda install conda-forge::biopython\" \n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bdfb73ab",
   "metadata": {},
   "source": [
    "### Download Data\n",
    "\n",
    "As an example, download the sg219 endogenous data from GEO that was used in the CRISPR-CLEAR manuscript: [GSE278581](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE278581)\n",
    "\n",
    "\n",
    "\n",
    "- GSM8549747\tABE8e, unedited, pooled, endogenous\n",
    "- GSM8549748\tABE8e, replicate 1, presort, sg218\n",
    "- GSM8549749\tABE8e, replicate 1, CD19 positive, sg218\n",
    "- GSM8549750\tABE8e, replicate 1, CD19 negative, sg218\n",
    "- GSM8549751\tABE8e, replicate 2, presort, sg218\n",
    "- GSM8549752\tABE8e, replicate 2, CD19 positive, sg218\n",
    "- GSM8549753\tABE8e, replicate 2, CD19 negative, sg218\n",
    "- GSM8549754\tABE8e, replicate 3, presort, sg218\n",
    "- GSM8549755\tABE8e, replicate 3, CD19 positive, sg218\n",
    "- GSM8549756\tABE8e, replicate 3, CD19 negative, sg218\n",
    "\n",
    "You can alternatively download the data from [Zenodo](https://doi.org/10.5281/zenodo.13737880) which may be easier: Unzip the file CRISPR-CLEAR-data/data/raw_FASTQs/endogenous_sequencing/sg219_fastqs.zip\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ef5fee43",
   "metadata": {},
   "source": [
    "### Prepare sample sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "1497f13a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Set the directory of the downloaded data, replace with your directory\n",
    "unmerged_all_dir = \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "26835e9c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1\n"
     ]
    }
   ],
   "source": [
    "# OPTIONAL: In all filenames, need to replace '#' character with '_' to allow correct parsing of filename in CRISPResso2\n",
    "def replicate_hashtag(directory_path):\n",
    "    print(directory_path)\n",
    "    file_list = os.listdir(directory_path)\n",
    "    for filename in file_list:\n",
    "        # Check if the '#' character is present in the filename\n",
    "        if '#' in filename:\n",
    "            # Replace '#' with '_'\n",
    "            new_filename = filename.replace('#', '_')\n",
    "\n",
    "            # Create the full old and new file paths\n",
    "            old_file_path = os.path.join(directory_path, filename)\n",
    "            new_file_path = os.path.join(directory_path, new_filename)\n",
    "\n",
    "            # Rename the file\n",
    "            os.rename(old_file_path, new_file_path)\n",
    "            print('Renamed: {} to {}'.format(filename, new_filename))\n",
    "            \n",
    "replicate_hashtag(unmerged_all_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "472dfcbc",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get filenames from directory\n",
    "unmerged_all_fns = os.listdir(unmerged_all_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "e509ecb4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get the R1 and R2 files\n",
    "r1_unmerged_fastq = [fn for fn in unmerged_all_fns if (\"R1.fastq\" in fn) and (\"PAX5sg\" in fn)]\n",
    "r2_unmerged_fastq = [fn for fn in unmerged_all_fns if (\"R2.fastq\" in fn) and (\"PAX5sg\" in fn)]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "979fa936",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Sort the R1 and R2 files so that corresponding files are in same list index\n",
    "r1_unmerged_fastq = sorted(r1_unmerged_fastq)\n",
    "r2_unmerged_fastq = sorted(r2_unmerged_fastq)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "5ccf44eb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_S139294.R1.fastq',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_S139299.R1.fastq',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_S139300.R1.fastq',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S139295.R1.fastq',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S139296.R1.fastq',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S139301.R1.fastq',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_1_S139292.R1.fastq',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_2_S139297.R1.fastq',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_3_S139298.R1.fastq']"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "r1_unmerged_fastq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "7abc61aa",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Prepare the sample metadata values\n",
    "editors_list = [\"ABE8e\"] * 9\n",
    "population_list = [\"high\"] * 3 + [\"low\"] * 3 + [\"presort\"] * 3\n",
    "replicate_list = [1,2,3] * 3\n",
    "experiment_list = [\"ABE8e_PAX5sg\"] * 9"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "88e673be",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Create dataframe from metadata files and save sample sheet\n",
    "sample_sheet_df = pd.DataFrame({\n",
    "    \"editors\": editors_list,\n",
    "    \"population\": population_list,\n",
    "    \"replicate\": replicate_list,\n",
    "    \"experiment\": experiment_list,\n",
    "    \"R1_FASTQ\": r1_unmerged_fastq,\n",
    "    \"R2_FASTQ\": r2_unmerged_fastq\n",
    "})\n",
    "\n",
    "sample_sheet_df = sample_sheet_df.loc[:, [\"editors\", \"population\", \"replicate\", \"experiment\", \"R1_FASTQ\", \"R2_FASTQ\"]]\n",
    "sample_sheet_df.to_csv(\"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/MetaData/20230922_sample_sheet_2.tsv\", sep=\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "516cacd5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>editors</th>\n",
       "      <th>population</th>\n",
       "      <th>replicate</th>\n",
       "      <th>experiment</th>\n",
       "      <th>R1_FASTQ</th>\n",
       "      <th>R2_FASTQ</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>high</td>\n",
       "      <td>1</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>high</td>\n",
       "      <td>2</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>high</td>\n",
       "      <td>3</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>low</td>\n",
       "      <td>1</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>low</td>\n",
       "      <td>2</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>low</td>\n",
       "      <td>3</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>presort</td>\n",
       "      <td>1</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>presort</td>\n",
       "      <td>2</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>presort</td>\n",
       "      <td>3</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  editors population  replicate    experiment  \\\n",
       "0   ABE8e       high          1  ABE8e_PAX5sg   \n",
       "1   ABE8e       high          2  ABE8e_PAX5sg   \n",
       "2   ABE8e       high          3  ABE8e_PAX5sg   \n",
       "3   ABE8e        low          1  ABE8e_PAX5sg   \n",
       "4   ABE8e        low          2  ABE8e_PAX5sg   \n",
       "5   ABE8e        low          3  ABE8e_PAX5sg   \n",
       "6   ABE8e    presort          1  ABE8e_PAX5sg   \n",
       "7   ABE8e    presort          2  ABE8e_PAX5sg   \n",
       "8   ABE8e    presort          3  ABE8e_PAX5sg   \n",
       "\n",
       "                                            R1_FASTQ  \\\n",
       "0  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_...   \n",
       "1  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_...   \n",
       "2  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_...   \n",
       "3  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S...   \n",
       "4  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S...   \n",
       "5  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S...   \n",
       "6  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...   \n",
       "7  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...   \n",
       "8  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...   \n",
       "\n",
       "                                            R2_FASTQ  \n",
       "0  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_...  \n",
       "1  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_...  \n",
       "2  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_...  \n",
       "3  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S...  \n",
       "4  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S...  \n",
       "5  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S...  \n",
       "6  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...  \n",
       "7  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...  \n",
       "8  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...  "
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sample_sheet_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "672b0ddb",
   "metadata": {},
   "source": [
    "### Read in guide sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "5c19094b",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/ipykernel_launcher.py:3: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n"
     ]
    }
   ],
   "source": [
    "# Read in the guide library file (contain in the repository), replace with filepath:\n",
    "guide_library_fn = \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/CD19_sgRNA_count_libraries_DS.txt\"\n",
    "pooled_guide_sequence_df = pd.read_table(guide_library_fn)\n",
    "pooled_guide_sequence_df.columns = [col.strip() for col in pooled_guide_sequence_df.columns]\n",
    "pooled_guide_sequences = [str(val) for val in pooled_guide_sequence_df[\"sgRNA\"].values.tolist()]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "5380bcf4",
   "metadata": {},
   "outputs": [],
   "source": [
    "editor = [\"ABE8e\"]\n",
    "sgID = [\"219\"]\n",
    "guide_sequence = [\"ACAGGAAGTCACAGCCTGGT\"]\n",
    "\n",
    "validation_origin = [\"ABE8e_v1_endogenous\"]\n",
    "\n",
    "\n",
    "CD19_guide_validation_sequences_df = pd.DataFrame({\n",
    "    \"editor\": editor,\n",
    "    \"sgID\": sgID,\n",
    "    \"screen_origin\": validation_origin,\n",
    "    \"guide_sequence\": guide_sequence\n",
    "})\n",
    "\n",
    "CD19_guide_validation_sequences_df = CD19_guide_validation_sequences_df.loc[:, [\"editor\", \"sgID\", \"screen_origin\", \"guide_sequence\"]]\n",
    "\n",
    "# Add the field corresponding to the CRISPRessoBatch sample name:\n",
    "CD19_guide_validation_sequences_df[\"crispresso_name\"] = CD19_guide_validation_sequences_df[\"editor\"] + \"_\" + CD19_guide_validation_sequences_df[\"sgID\"]\n",
    "CD19_guide_validation_sequences_df[\"crispresso_name\"][0] = \"ABE8e_PAX5sg\"\n",
    "CD19_guide_validation_sequences_df.to_csv(\"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/20230922_CD19_guide_validation_sequences_2.tsv\", sep=\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "b74620a8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>editor</th>\n",
       "      <th>sgID</th>\n",
       "      <th>screen_origin</th>\n",
       "      <th>guide_sequence</th>\n",
       "      <th>crispresso_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>219</td>\n",
       "      <td>ABE8e_v1_endogenous</td>\n",
       "      <td>ACAGGAAGTCACAGCCTGGT</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  editor sgID        screen_origin        guide_sequence crispresso_name\n",
       "0  ABE8e  219  ABE8e_v1_endogenous  ACAGGAAGTCACAGCCTGGT    ABE8e_PAX5sg"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "CD19_guide_validation_sequences_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d8013102",
   "metadata": {},
   "source": [
    "## Run CRISPRessoBatch"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0f32d99b",
   "metadata": {},
   "source": [
    "Prepare amplicon sequence and guide list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "a5a82406",
   "metadata": {},
   "outputs": [],
   "source": [
    "pooled_amplicon_sequence = Seq(\"ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG\")\n",
    "\n",
    "pooled_amplicon_sequence_revcomp = pooled_amplicon_sequence.reverse_complement() \n",
    "pooled_amplicon_sequence = str(pooled_amplicon_sequence)\n",
    "pooled_amplicon_sequence_revcomp = str(pooled_amplicon_sequence_revcomp)\n",
    "\n",
    "guide_sequences_present = [seq for seq in pooled_guide_sequences if ((seq in pooled_amplicon_sequence) or (seq in pooled_amplicon_sequence_revcomp))]\n",
    "\n",
    "pooled_middle_guide = \"GAGGCCCTGCCACCTGTAGG\" # Set guide in middle of amplicon for visualization\n",
    "r1_middle_guide = \"ATTCATGCCTCATTCTTCCG\"\n",
    "r2_middle_guide = \"ACAGCAAACAGGAAGTCACA\"\n",
    "# TODO (20230925): Might need another middle guide for R1 and R2 individually"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "dfc802d8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['CATGCTCTAGTGAAAGCCAG',\n",
       " 'GCTCTAGTGAAAGCCAGTCT',\n",
       " 'CTAGTGAAAGCCAGTCTGGG',\n",
       " 'GTGAAAGCCAGTCTGGGCAG',\n",
       " 'AAAGCCAGTCTGGGCAGCTG',\n",
       " 'GCCAGTCTGGGCAGCTGGGT',\n",
       " 'AGTCTGGGCAGCTGGGTAGC',\n",
       " 'CTGGGCAGCTGGGTAGCTAA',\n",
       " 'GGCAGCTGGGTAGCTAATGA',\n",
       " 'AGCTGGGTAGCTAATGAGGG',\n",
       " 'TGGGTAGCTAATGAGGGGAT',\n",
       " 'GCTAATGAGGGGATTAGAGA',\n",
       " 'AATGAGGGGATTAGAGAGAT',\n",
       " 'GAGGGGATTAGAGAGATTTT',\n",
       " 'GGGATTAGAGAGATTTTGTT',\n",
       " 'ATTAGAGAGATTTTGTTGAA',\n",
       " 'GAGATTTTGTTGAATGAAAG',\n",
       " 'ATTTTGTTGAATGAAAGGCA',\n",
       " 'TTGTTGAATGAAAGGCAGAT',\n",
       " 'TTGAATGAAAGGCAGATTGA',\n",
       " 'AATGAAAGGCAGATTGAGTC',\n",
       " 'GAAAGGCAGATTGAGTCCTG',\n",
       " 'AGGCAGATTGAGTCCTGCTA',\n",
       " 'CAGATTGAGTCCTGCTACTC',\n",
       " 'ATTGAGTCCTGCTACTCGCC',\n",
       " 'GAGTCCTGCTACTCGCCCCC',\n",
       " 'TCCTGCTACTCGCCCCCTTC',\n",
       " 'TGCTACTCGCCCCCTTCATT',\n",
       " 'TACTCGCCCCCTTCATTCCC',\n",
       " 'TCGCCCCCTTCATTCCCCTT',\n",
       " 'CCCCCTTCATTCCCCTTCAT',\n",
       " 'CCTTCATTCCCCTTCATTCA',\n",
       " 'TCATTCCCCTTCATTCATGC',\n",
       " 'TTCCCCTTCATTCATGCCTC',\n",
       " 'CCCTTCATTCATGCCTCATT',\n",
       " 'TTCATTCATGCCTCATTCTT',\n",
       " 'ATTCATGCCTCATTCTTCCG',\n",
       " 'CATGCCTCATTCTTCCGCCT',\n",
       " 'GCCTCATTCTTCCGCCTCCC',\n",
       " 'TCATTCTTCCGCCTCCCAGC',\n",
       " 'TTCTTCCGCCTCCCAGCCGC',\n",
       " 'TTCCGCCTCCCAGCCGCCTC',\n",
       " 'CGCCTCCCAGCCGCCTCAAC',\n",
       " 'CTCCCAGCCGCCTCAACTGG',\n",
       " 'CCAGCCGCCTCAACTGGCCA',\n",
       " 'GCCGCCTCAACTGGCCAAAG',\n",
       " 'TCAACTGGCCAAAGGGAAGT',\n",
       " 'ACTGGCCAAAGGGAAGTGGA',\n",
       " 'GGCCAAAGGGAAGTGGAGGC',\n",
       " 'CAAAGGGAAGTGGAGGCCCT',\n",
       " 'AGGGAAGTGGAGGCCCTGCC',\n",
       " 'GAAGTGGAGGCCCTGCCACC',\n",
       " 'GTGGAGGCCCTGCCACCTGT',\n",
       " 'GAGGCCCTGCCACCTGTAGG',\n",
       " 'CTGCCACCTGTAGGGAGGGT',\n",
       " 'CCACCTGTAGGGAGGGTCCC',\n",
       " 'CCTGTAGGGAGGGTCCCCTG',\n",
       " 'GTAGGGAGGGTCCCCTGGGG',\n",
       " 'GGGAGGGTCCCCTGGGGCTT',\n",
       " 'AGGGTCCCCTGGGGCTTGCC',\n",
       " 'GTCCCCTGGGGCTTGCCCAC',\n",
       " 'CCCTGGGGCTTGCCCACAGC',\n",
       " 'TGGGGCTTGCCCACAGCAAA',\n",
       " 'GGCTTGCCCACAGCAAACAG',\n",
       " 'CCCACAGCAAACAGGAAGTC',\n",
       " 'ACAGCAAACAGGAAGTCACA',\n",
       " 'AACAGGAAGTCACAGCCTGG',\n",
       " 'AGGAAGTCACAGCCTGGTGA',\n",
       " 'AAGTCACAGCCTGGTGAGAT',\n",
       " 'TCACAGCCTGGTGAGATGGG',\n",
       " 'CAGCCTGGTGAGATGGGCCT',\n",
       " 'CCTGGTGAGATGGGCCTGGG',\n",
       " 'GGTGAGATGGGCCTGGGAAT',\n",
       " 'ATGGGCCTGGGAATCAGCCA',\n",
       " 'GGCCTGGGAATCAGCCACTG',\n",
       " 'CTGGGAATCAGCCACTGAGA',\n",
       " 'ATCAGCCACTGAGAAAGTGG',\n",
       " 'AGCCACTGAGAAAGTGGGTC',\n",
       " 'CACTGAGAAAGTGGGTCTCT',\n",
       " 'TGAGAAAGTGGGTCTCTTGG',\n",
       " 'GAAAGTGGGTCTCTTGGGTC',\n",
       " 'GGGTCTCTTGGGTCCCTGAA',\n",
       " 'TCTCTTGGGTCCCTGAATTC',\n",
       " 'CTTGGGTCCCTGAATTCTTT',\n",
       " 'GGGTCCCTGAATTCTTTTTC',\n",
       " 'TCCCTGAATTCTTTTTCTGA',\n",
       " 'CTGAATTCTTTTTCTGAGTC',\n",
       " 'AATTCTTTTTCTGAGTCCCT',\n",
       " 'TCTTTTTCTGAGTCCCTGCA',\n",
       " 'GAGTCCCTGCAGCAGTGAAA',\n",
       " 'TCCCTGCAGCAGTGAAAAAG',\n",
       " 'CTGCAGCAGTGAAAAAGACA',\n",
       " 'CAGTGAAAAAGACACAGAGG',\n",
       " 'TGAAAAAGACACAGAGGCAC',\n",
       " 'AAAAGACACAGAGGCACATA',\n",
       " 'AGACACAGAGGCACATAGAG',\n",
       " 'CACAGAGGCACATAGAGAGT',\n",
       " 'CTGGCTTTCACTAGAGCATG',\n",
       " 'AGACTGGCTTTCACTAGAGC',\n",
       " 'CCCAGACTGGCTTTCACTAG',\n",
       " 'CTGCCCAGACTGGCTTTCAC',\n",
       " 'ACCCAGCTGCCCAGACTGGC',\n",
       " 'TCATTAGCTACCCAGCTGCC',\n",
       " 'CCCTCATTAGCTACCCAGCT',\n",
       " 'ATCCCCTCATTAGCTACCCA',\n",
       " 'CTAATCCCCTCATTAGCTAC',\n",
       " 'TCTCTAATCCCCTCATTAGC',\n",
       " 'ATCTCTCTAATCCCCTCATT',\n",
       " 'AAAATCTCTCTAATCCCCTC',\n",
       " 'AACAAAATCTCTCTAATCCC',\n",
       " 'TCATTCAACAAAATCTCTCT',\n",
       " 'CTTTCATTCAACAAAATCTC',\n",
       " 'TGCCTTTCATTCAACAAAAT',\n",
       " 'ATCTGCCTTTCATTCAACAA',\n",
       " 'TCAATCTGCCTTTCATTCAA',\n",
       " 'GACTCAATCTGCCTTTCATT',\n",
       " 'CAGGACTCAATCTGCCTTTC',\n",
       " 'GAGTAGCAGGACTCAATCTG',\n",
       " 'GGCGAGTAGCAGGACTCAAT',\n",
       " 'GGGGGCGAGTAGCAGGACTC',\n",
       " 'AATGAAGGGGGCGAGTAGCA',\n",
       " 'AAGGGGAATGAAGGGGGCGA',\n",
       " 'ATGAAGGGGAATGAAGGGGG',\n",
       " 'TGAATGAAGGGGAATGAAGG',\n",
       " 'GCATGAATGAAGGGGAATGA',\n",
       " 'GAGGCATGAATGAAGGGGAA',\n",
       " 'AATGAGGCATGAATGAAGGG',\n",
       " 'AAGAATGAGGCATGAATGAA',\n",
       " 'CGGAAGAATGAGGCATGAAT',\n",
       " 'AGGCGGAAGAATGAGGCATG',\n",
       " 'GGGAGGCGGAAGAATGAGGC',\n",
       " 'GCTGGGAGGCGGAAGAATGA',\n",
       " 'GCGGCTGGGAGGCGGAAGAA',\n",
       " 'GAGGCGGCTGGGAGGCGGAA',\n",
       " 'GTTGAGGCGGCTGGGAGGCG',\n",
       " 'CCAGTTGAGGCGGCTGGGAG',\n",
       " 'TGGCCAGTTGAGGCGGCTGG',\n",
       " 'CTTTGGCCAGTTGAGGCGGC',\n",
       " 'ACTTCCCTTTGGCCAGTTGA',\n",
       " 'TCCACTTCCCTTTGGCCAGT',\n",
       " 'GCCTCCACTTCCCTTTGGCC',\n",
       " 'AGGGCCTCCACTTCCCTTTG',\n",
       " 'GGCAGGGCCTCCACTTCCCT',\n",
       " 'GGTGGCAGGGCCTCCACTTC',\n",
       " 'ACAGGTGGCAGGGCCTCCAC',\n",
       " 'CCTACAGGTGGCAGGGCCTC',\n",
       " 'CTCCCTACAGGTGGCAGGGC',\n",
       " 'ACCCTCCCTACAGGTGGCAG',\n",
       " 'GGGACCCTCCCTACAGGTGG',\n",
       " 'CAGGGGACCCTCCCTACAGG',\n",
       " 'CCCCAGGGGACCCTCCCTAC',\n",
       " 'AAGCCCCAGGGGACCCTCCC',\n",
       " 'GGCAAGCCCCAGGGGACCCT',\n",
       " 'GCTGTGGGCAAGCCCCAGGG',\n",
       " 'TTTGCTGTGGGCAAGCCCCA',\n",
       " 'CTGTTTGCTGTGGGCAAGCC',\n",
       " 'TTCCTGTTTGCTGTGGGCAA',\n",
       " 'GACTTCCTGTTTGCTGTGGG',\n",
       " 'TGTGACTTCCTGTTTGCTGT',\n",
       " 'GGCTGTGACTTCCTGTTTGC',\n",
       " 'CCAGGCTGTGACTTCCTGTT',\n",
       " 'TCACCAGGCTGTGACTTCCT',\n",
       " 'ATCTCACCAGGCTGTGACTT',\n",
       " 'CCCATCTCACCAGGCTGTGA',\n",
       " 'AGGCCCATCTCACCAGGCTG',\n",
       " 'CCCAGGCCCATCTCACCAGG',\n",
       " 'ATTCCCAGGCCCATCTCACC',\n",
       " 'CTGATTCCCAGGCCCATCTC',\n",
       " 'TGGCTGATTCCCAGGCCCAT',\n",
       " 'TCTCAGTGGCTGATTCCCAG',\n",
       " 'CCACTTTCTCAGTGGCTGAT',\n",
       " 'GACCCACTTTCTCAGTGGCT',\n",
       " 'AGAGACCCACTTTCTCAGTG',\n",
       " 'CCAAGAGACCCACTTTCTCA',\n",
       " 'GACCCAAGAGACCCACTTTC',\n",
       " 'AGGGACCCAAGAGACCCACT',\n",
       " 'TTCAGGGACCCAAGAGACCC',\n",
       " 'GAATTCAGGGACCCAAGAGA',\n",
       " 'AAAGAATTCAGGGACCCAAG',\n",
       " 'GAAAAAGAATTCAGGGACCC',\n",
       " 'TCAGAAAAAGAATTCAGGGA',\n",
       " 'GACTCAGAAAAAGAATTCAG',\n",
       " 'AGGGACTCAGAAAAAGAATT',\n",
       " 'TGCTGCAGGGACTCAGAAAA',\n",
       " 'CACTGCTGCAGGGACTCAGA',\n",
       " 'TTTCACTGCTGCAGGGACTC',\n",
       " 'CTTTTTCACTGCTGCAGGGA',\n",
       " 'TGTCTTTTTCACTGCTGCAG',\n",
       " 'CTGTGTCTTTTTCACTGCTG',\n",
       " 'CCTCTGTGTCTTTTTCACTG',\n",
       " 'GTGCCTCTGTGTCTTTTTCA',\n",
       " 'TATGTGCCTCTGTGTCTTTT']"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "guide_sequences_present"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "97e6a8c7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 192 guides present in amplicon sequence out of 206\n"
     ]
    }
   ],
   "source": [
    "print(\"There are {} guides present in amplicon sequence out of {}\".format(len(guide_sequences_present), len(pooled_guide_sequences)))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6cf55400",
   "metadata": {},
   "source": [
    "Prepare filenames"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "f5e427c8",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Prepare R1 and R2 FASTQ files for CRISPResso2Batch settings file\n",
    "filenames_single_end_dir = \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/\"\n",
    "\n",
    "names = sample_sheet_df[\"experiment\"].str.cat(sample_sheet_df[[\"population\", \"replicate\"]].astype(str), sep=\"_\")\n",
    "sample_sheet_df[\"names\"] = names\n",
    "filenames_R1 = filenames_single_end_dir + sample_sheet_df[\"R1_FASTQ\"] # TODO, prefix with directory\n",
    "filenames_R2 = filenames_single_end_dir + sample_sheet_df[\"R2_FASTQ\"] # TODO, prefix with directory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "373c980f",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Create directory for all the different settings file\n",
    "settings_output_dir = \"./20241004_CRISPResso2Batch_Settings\"\n",
    "os.makedirs(settings_output_dir)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1469cac4",
   "metadata": {},
   "source": [
    "Create settings files for pooled screen"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "aa5bf78e",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Prepare sample names that will be batched\n",
    "pooled_names_enumeration = [(index, name) for index, name in enumerate(names) if (\"pooled\" in name) or (\"mock_presort_1\" in name)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "e76bb8a6",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pooled_names_enumeration"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d5a945ff",
   "metadata": {},
   "source": [
    "Create settings files for validation guides"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "50eba465",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ABE8e_sg219\n",
      "[(0, 'ABE8e_PAX5sg_high_1'), (1, 'ABE8e_PAX5sg_high_2'), (2, 'ABE8e_PAX5sg_high_3'), (3, 'ABE8e_PAX5sg_low_1'), (4, 'ABE8e_PAX5sg_low_2'), (5, 'ABE8e_PAX5sg_low_3'), (6, 'ABE8e_PAX5sg_presort_1'), (7, 'ABE8e_PAX5sg_presort_2'), (8, 'ABE8e_PAX5sg_presort_3')]\n"
     ]
    }
   ],
   "source": [
    "global skipped_ids, available_ids\n",
    "skipped_ids = []\n",
    "available_ids = []\n",
    "def generate_validation_guide_crispresso_settings(validation_guide_row):\n",
    "    validation_name = validation_guide_row[\"editor\"] + \"_sg\" + validation_guide_row[\"sgID\"]\n",
    "    crispresso_name = validation_guide_row[\"crispresso_name\"]\n",
    "    print(validation_name)\n",
    "    \n",
    "    unmerged_PE_settings_fn = 'Davide_CD19_enhancer_BE_screen_{}_unmerged_PE.settings'.format(validation_name)\n",
    "    unmerged_R1_settings_fn = 'Davide_CD19_enhancer_BE_screen_{}_unmerged_R1.settings'.format(validation_name)\n",
    "    unmerged_R2_settings_fn = 'Davide_CD19_enhancer_BE_screen_{}_unmerged_R2.settings'.format(validation_name)\n",
    "    \n",
    "    validation_names_enumeration = [(index, name) for index, name in enumerate(names) if crispresso_name in name]\n",
    "    print(validation_names_enumeration)\n",
    "    \n",
    "    if len(validation_names_enumeration) == 0:\n",
    "        skipped_ids.append(validation_name)\n",
    "    else:\n",
    "        available_ids.append(validation_name)\n",
    "\n",
    "        # Un-merged settings\n",
    "        with open(settings_output_dir + \"/\" + unmerged_PE_settings_fn, 'w') as out_handle:\n",
    "            out_handle.write('name\\tfastq_r1\\tfastq_r2\\tguide_seq\\n')\n",
    "            for sample_index, _ in validation_names_enumeration:\n",
    "                out_handle.write('sample_%s\\t%s\\t%s\\t%s\\n' % (names[sample_index], filenames_R1[sample_index], filenames_R2[sample_index], validation_guide_row[\"guide_sequence\"]))\n",
    "\n",
    "        # R1 settings\n",
    "        with open(settings_output_dir + \"/\" + unmerged_R1_settings_fn, 'w') as out_handle:\n",
    "            out_handle.write('name\\tfastq_r1\\tguide_seq\\n')\n",
    "            for sample_index, _ in validation_names_enumeration:\n",
    "                out_handle.write('sample_%s\\t%s\\t%s\\n' % (names[sample_index], filenames_R1[sample_index], validation_guide_row[\"guide_sequence\"]))\n",
    "\n",
    "        # R2 settings\n",
    "        with open(settings_output_dir + \"/\" + unmerged_R2_settings_fn, 'w') as out_handle:\n",
    "            out_handle.write('name\\tfastq_r1\\tguide_seq\\n')\n",
    "            for sample_index, _ in validation_names_enumeration:\n",
    "                out_handle.write('sample_%s\\t%s\\t%s\\n' % (names[sample_index], filenames_R2[sample_index], validation_guide_row[\"guide_sequence\"]))\n",
    "\n",
    "for _, row in CD19_guide_validation_sequences_df.iterrows():\n",
    "    generate_validation_guide_crispresso_settings(row)            "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "26d126c0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "skipped_ids"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "dddfd762",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['ABE8e_sg219']"
      ]
     },
     "execution_count": 52,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "available_ids"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "48cffa12",
   "metadata": {},
   "source": [
    "Run CRISPResso2Batch"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "id": "52e4de89",
   "metadata": {},
   "outputs": [],
   "source": [
    "root_results_output_dir = \"./20231002_v3_CRISPResso2Batch_Results\"\n",
    "os.makedirs(root_results_output_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "id": "a88fbe54",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing ABE8e_sg219\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Running CRISPResso with 20 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle --name sample_ABE8e_PAX5sg_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_S139294.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle --name sample_ABE8e_PAX5sg_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_S139299.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle --name sample_ABE8e_PAX5sg_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_S139300.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle --name sample_ABE8e_PAX5sg_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S139295.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle --name sample_ABE8e_PAX5sg_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S139296.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle --name sample_ABE8e_PAX5sg_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S139301.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle --name sample_ABE8e_PAX5sg_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_1_S139292.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle --name sample_ABE8e_PAX5sg_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_2_S139297.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:30:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle --name sample_ABE8e_PAX5sg_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_3_S139298.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by tINFO  @ Thu, 05 Oct 2023 15:34:35:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_1 \n",
      "\n",
      "he quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:35:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:35:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:35:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:35:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:35:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:35:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:35:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:35:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/premerged/CRISPRessoBatch_on_ABE8e_sg219_premerged_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:36:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:36:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:36:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:36:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:36:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:36:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:36:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:36:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:34:37:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 25 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 27 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:04:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:04:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:08:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3049 N_CACHED_ALN: 6378 N_COMPUTED_NOTALN: 413 N_CACHED_NOTALN: 160 \n",
      "\n",
      "Completed in 32 seconds\n",
      "\n",
      "Completed in 33 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:09:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:09:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:09:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:09:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:11:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4211 N_CACHED_ALN: 5281 N_COMPUTED_NOTALN: 433 N_CACHED_NOTALN: 75 \n",
      "\n",
      "Completed in 36 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:13:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5910 N_CACHED_ALN: 12947 N_COMPUTED_NOTALN: 773 N_CACHED_NOTALN: 370 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:13:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:13:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 36 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:14:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:14:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 37 seconds\n",
      "\n",
      "Completed in 37 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:14:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:14:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:14:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:14:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 38 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:16:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:16:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:16:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4006 N_CACHED_ALN: 5459 N_COMPUTED_NOTALN: 449 N_CACHED_NOTALN: 86 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:17:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4407 N_CACHED_ALN: 4931 N_COMPUTED_NOTALN: 563 N_CACHED_NOTALN: 99 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:17:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7912 N_CACHED_ALN: 11025 N_COMPUTED_NOTALN: 864 N_CACHED_NOTALN: 199 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:18:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 8900 N_CACHED_ALN: 19392 N_COMPUTED_NOTALN: 1133 N_CACHED_NOTALN: 575 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:20:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3623 N_CACHED_ALN: 5672 N_COMPUTED_NOTALN: 555 N_CACHED_NOTALN: 150 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:20:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4155 N_CACHED_ALN: 5143 N_COMPUTED_NOTALN: 602 N_CACHED_NOTALN: 100 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:20:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3377 N_CACHED_ALN: 5881 N_COMPUTED_NOTALN: 557 N_CACHED_NOTALN: 185 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:21:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3934 N_CACHED_ALN: 5565 N_COMPUTED_NOTALN: 429 N_CACHED_NOTALN: 72 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:22:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7600 N_CACHED_ALN: 11367 N_COMPUTED_NOTALN: 822 N_CACHED_NOTALN: 211 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:22:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 10675 N_CACHED_ALN: 17650 N_COMPUTED_NOTALN: 1256 N_CACHED_NOTALN: 419 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:23:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3829 N_CACHED_ALN: 5463 N_COMPUTED_NOTALN: 574 N_CACHED_NOTALN: 134 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:23:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 11666 N_CACHED_ALN: 26052 N_COMPUTED_NOTALN: 1482 N_CACHED_NOTALN: 800 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:24:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8530 N_CACHED_ALN: 10196 N_COMPUTED_NOTALN: 1007 N_CACHED_NOTALN: 267 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:26:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6881 N_CACHED_ALN: 11713 N_COMPUTED_NOTALN: 1027 N_CACHED_NOTALN: 379 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:26:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6627 N_CACHED_ALN: 11918 N_COMPUTED_NOTALN: 1037 N_CACHED_NOTALN: 418 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:27:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 12918 N_CACHED_ALN: 24772 N_COMPUTED_NOTALN: 1632 N_CACHED_NOTALN: 678 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:27:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 10677 N_CACHED_ALN: 17769 N_COMPUTED_NOTALN: 1218 N_CACHED_NOTALN: 336 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:27:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8009 N_CACHED_ALN: 10546 N_COMPUTED_NOTALN: 1188 N_CACHED_NOTALN: 257 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:28:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7750 N_CACHED_ALN: 11255 N_COMPUTED_NOTALN: 831 N_CACHED_NOTALN: 164 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:28:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 14662 N_CACHED_ALN: 32515 N_COMPUTED_NOTALN: 1845 N_CACHED_NOTALN: 978 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:29:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7557 N_CACHED_ALN: 10958 N_COMPUTED_NOTALN: 1127 N_CACHED_NOTALN: 358 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:30:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 12393 N_CACHED_ALN: 15665 N_COMPUTED_NOTALN: 1483 N_CACHED_NOTALN: 459 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:31:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 10138 N_CACHED_ALN: 17767 N_COMPUTED_NOTALN: 1480 N_CACHED_NOTALN: 615 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:32:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 15448 N_CACHED_ALN: 31657 N_COMPUTED_NOTALN: 1970 N_CACHED_NOTALN: 925 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:32:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 13213 N_CACHED_ALN: 24670 N_COMPUTED_NOTALN: 1560 N_CACHED_NOTALN: 557 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:32:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9949 N_CACHED_ALN: 17842 N_COMPUTED_NOTALN: 1529 N_CACHED_NOTALN: 680 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:33:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 17319 N_CACHED_ALN: 39296 N_COMPUTED_NOTALN: 2172 N_CACHED_NOTALN: 1213 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:34:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 11454 N_CACHED_ALN: 17014 N_COMPUTED_NOTALN: 1218 N_CACHED_NOTALN: 314 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:35:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 11594 N_CACHED_ALN: 16203 N_COMPUTED_NOTALN: 1774 N_CACHED_NOTALN: 429 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:35:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 15195 N_CACHED_ALN: 22149 N_COMPUTED_NOTALN: 1929 N_CACHED_NOTALN: 727 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:36:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 11328 N_CACHED_ALN: 16491 N_COMPUTED_NOTALN: 1617 N_CACHED_NOTALN: 564 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:36:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 15466 N_CACHED_ALN: 31882 N_COMPUTED_NOTALN: 1875 N_CACHED_NOTALN: 777 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:37:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 13040 N_CACHED_ALN: 24074 N_COMPUTED_NOTALN: 1969 N_CACHED_NOTALN: 917 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:37:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 18288 N_CACHED_ALN: 38257 N_COMPUTED_NOTALN: 2294 N_CACHED_NOTALN: 1161 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:38:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 20094 N_CACHED_ALN: 45953 N_COMPUTED_NOTALN: 2513 N_CACHED_NOTALN: 1440 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:38:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 12986 N_CACHED_ALN: 24071 N_COMPUTED_NOTALN: 1988 N_CACHED_NOTALN: 955 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:39:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 14444 N_CACHED_ALN: 23525 N_COMPUTED_NOTALN: 1535 N_CACHED_NOTALN: 496 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:40:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 17642 N_CACHED_ALN: 39132 N_COMPUTED_NOTALN: 2173 N_CACHED_NOTALN: 1053 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:40:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 17797 N_CACHED_ALN: 28833 N_COMPUTED_NOTALN: 2320 N_CACHED_NOTALN: 1050 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:41:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 14356 N_CACHED_ALN: 22686 N_COMPUTED_NOTALN: 2260 N_CACHED_NOTALN: 698 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:41:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 14574 N_CACHED_ALN: 22531 N_COMPUTED_NOTALN: 2068 N_CACHED_NOTALN: 827 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:42:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 15847 N_CACHED_ALN: 30543 N_COMPUTED_NOTALN: 2345 N_CACHED_NOTALN: 1265 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:42:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 21277 N_CACHED_ALN: 44706 N_COMPUTED_NOTALN: 2634 N_CACHED_NOTALN: 1383 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:43:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 15707 N_CACHED_ALN: 30633 N_COMPUTED_NOTALN: 2417 N_CACHED_NOTALN: 1243 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:43:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 23127 N_CACHED_ALN: 52365 N_COMPUTED_NOTALN: 2864 N_CACHED_NOTALN: 1644 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:44:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 20055 N_CACHED_ALN: 46159 N_COMPUTED_NOTALN: 2478 N_CACHED_NOTALN: 1308 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:44:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 17220 N_CACHED_ALN: 30209 N_COMPUTED_NOTALN: 1847 N_CACHED_NOTALN: 724 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:45:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 20665 N_CACHED_ALN: 35301 N_COMPUTED_NOTALN: 2702 N_CACHED_NOTALN: 1332 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:47:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 16924 N_CACHED_ALN: 29361 N_COMPUTED_NOTALN: 2698 N_CACHED_NOTALN: 1017 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:47:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 18425 N_CACHED_ALN: 37266 N_COMPUTED_NOTALN: 2714 N_CACHED_NOTALN: 1595 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:47:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 18149 N_CACHED_ALN: 28264 N_COMPUTED_NOTALN: 2541 N_CACHED_NOTALN: 1046 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:48:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 24351 N_CACHED_ALN: 51113 N_COMPUTED_NOTALN: 2943 N_CACHED_NOTALN: 1593 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:48:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 18478 N_CACHED_ALN: 37153 N_COMPUTED_NOTALN: 2831 N_CACHED_NOTALN: 1538 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:49:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 22546 N_CACHED_ALN: 53148 N_COMPUTED_NOTALN: 2761 N_CACHED_NOTALN: 1545 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:49:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 19964 N_CACHED_ALN: 36923 N_COMPUTED_NOTALN: 2162 N_CACHED_NOTALN: 951 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:49:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 26376 N_CACHED_ALN: 58574 N_COMPUTED_NOTALN: 3233 N_CACHED_NOTALN: 1817 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:51:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 23545 N_CACHED_ALN: 41722 N_COMPUTED_NOTALN: 3105 N_CACHED_NOTALN: 1628 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:52:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 19668 N_CACHED_ALN: 35852 N_COMPUTED_NOTALN: 3124 N_CACHED_NOTALN: 1356 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:52:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 21092 N_CACHED_ALN: 43899 N_COMPUTED_NOTALN: 3084 N_CACHED_NOTALN: 1925 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:52:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 20958 N_CACHED_ALN: 34743 N_COMPUTED_NOTALN: 2961 N_CACHED_NOTALN: 1338 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:53:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 27117 N_CACHED_ALN: 57778 N_COMPUTED_NOTALN: 3284 N_CACHED_NOTALN: 1821 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:53:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 21059 N_CACHED_ALN: 43861 N_COMPUTED_NOTALN: 3216 N_CACHED_NOTALN: 1864 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:54:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 25428 N_CACHED_ALN: 59757 N_COMPUTED_NOTALN: 3066 N_CACHED_NOTALN: 1749 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:54:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 22535 N_CACHED_ALN: 43746 N_COMPUTED_NOTALN: 2503 N_CACHED_NOTALN: 1216 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:55:\n",
      "\t Finished reads; N_TOT_READS: 99103 N_COMPUTED_ALN: 29182 N_CACHED_ALN: 64400 N_COMPUTED_NOTALN: 3564 N_CACHED_NOTALN: 1957 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:55:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:56:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 22059 N_CACHED_ALN: 42731 N_COMPUTED_NOTALN: 3495 N_CACHED_NOTALN: 1715 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:57:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 26588 N_CACHED_ALN: 47971 N_COMPUTED_NOTALN: 3501 N_CACHED_NOTALN: 1940 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:57:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 24213 N_CACHED_ALN: 50103 N_COMPUTED_NOTALN: 3519 N_CACHED_NOTALN: 2165 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:58:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 24119 N_CACHED_ALN: 40875 N_COMPUTED_NOTALN: 3420 N_CACHED_NOTALN: 1586 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:58:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 28046 N_CACHED_ALN: 66587 N_COMPUTED_NOTALN: 3387 N_CACHED_NOTALN: 1980 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:59:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 23957 N_CACHED_ALN: 50300 N_COMPUTED_NOTALN: 3620 N_CACHED_NOTALN: 2123 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:35:59:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 25184 N_CACHED_ALN: 50534 N_COMPUTED_NOTALN: 2822 N_CACHED_NOTALN: 1460 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:00:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 30262 N_CACHED_ALN: 64111 N_COMPUTED_NOTALN: 3616 N_CACHED_NOTALN: 2011 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:01:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 24275 N_CACHED_ALN: 49767 N_COMPUTED_NOTALN: 3869 N_CACHED_NOTALN: 2089 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:01:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 29378 N_CACHED_ALN: 54553 N_COMPUTED_NOTALN: 3829 N_CACHED_NOTALN: 2240 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:02:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 26980 N_CACHED_ALN: 56594 N_COMPUTED_NOTALN: 3968 N_CACHED_NOTALN: 2458 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:03:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 26739 N_CACHED_ALN: 47536 N_COMPUTED_NOTALN: 3815 N_CACHED_NOTALN: 1910 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:03:\n",
      "\t Finished reads; N_TOT_READS: 106120 N_COMPUTED_ALN: 32146 N_CACHED_ALN: 67996 N_COMPUTED_NOTALN: 3863 N_CACHED_NOTALN: 2115 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:03:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:04:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 27792 N_CACHED_ALN: 57416 N_COMPUTED_NOTALN: 3114 N_CACHED_NOTALN: 1678 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:04:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 26830 N_CACHED_ALN: 56764 N_COMPUTED_NOTALN: 4012 N_CACHED_NOTALN: 2394 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:05:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 31042 N_CACHED_ALN: 73090 N_COMPUTED_NOTALN: 3710 N_CACHED_NOTALN: 2158 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:06:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:06:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:06:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 27309 N_CACHED_ALN: 55967 N_COMPUTED_NOTALN: 4316 N_CACHED_NOTALN: 2408 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:07:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:07:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 32353 N_CACHED_ALN: 60887 N_COMPUTED_NOTALN: 4222 N_CACHED_NOTALN: 2538 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:08:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 29555 N_CACHED_ALN: 63314 N_COMPUTED_NOTALN: 4340 N_CACHED_NOTALN: 2791 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:08:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 29493 N_CACHED_ALN: 54052 N_COMPUTED_NOTALN: 4240 N_CACHED_NOTALN: 2215 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:08:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 29202 N_CACHED_ALN: 63657 N_COMPUTED_NOTALN: 4378 N_CACHED_NOTALN: 2763 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:09:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 30571 N_CACHED_ALN: 64106 N_COMPUTED_NOTALN: 3435 N_CACHED_NOTALN: 1888 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:10:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 33798 N_CACHED_ALN: 79817 N_COMPUTED_NOTALN: 4011 N_CACHED_NOTALN: 2374 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:11:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 29802 N_CACHED_ALN: 62765 N_COMPUTED_NOTALN: 4709 N_CACHED_NOTALN: 2724 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:12:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 32111 N_CACHED_ALN: 70029 N_COMPUTED_NOTALN: 4727 N_CACHED_NOTALN: 3133 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:13:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 32208 N_CACHED_ALN: 60649 N_COMPUTED_NOTALN: 4628 N_CACHED_NOTALN: 2515 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:13:\n",
      "\t Finished reads; N_TOT_READS: 125724 N_COMPUTED_ALN: 35494 N_CACHED_ALN: 83554 N_COMPUTED_NOTALN: 4202 N_CACHED_NOTALN: 2474 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:13:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:13:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 35250 N_CACHED_ALN: 67312 N_COMPUTED_NOTALN: 4600 N_CACHED_NOTALN: 2838 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:14:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 33121 N_CACHED_ALN: 71044 N_COMPUTED_NOTALN: 3733 N_CACHED_NOTALN: 2102 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:15:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 32172 N_CACHED_ALN: 69981 N_COMPUTED_NOTALN: 4796 N_CACHED_NOTALN: 3051 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:15:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:15:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:16:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 32216 N_CACHED_ALN: 69591 N_COMPUTED_NOTALN: 5161 N_CACHED_NOTALN: 3032 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:16:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:17:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 34997 N_CACHED_ALN: 67118 N_COMPUTED_NOTALN: 5032 N_CACHED_NOTALN: 2853 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:18:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 35370 N_CACHED_ALN: 78250 N_COMPUTED_NOTALN: 4018 N_CACHED_NOTALN: 2362 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:18:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 37842 N_CACHED_ALN: 73979 N_COMPUTED_NOTALN: 4983 N_CACHED_NOTALN: 3196 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:18:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 34711 N_CACHED_ALN: 76672 N_COMPUTED_NOTALN: 5145 N_CACHED_NOTALN: 3472 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:19:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 34857 N_CACHED_ALN: 76533 N_COMPUTED_NOTALN: 5205 N_CACHED_NOTALN: 3405 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:21:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 34582 N_CACHED_ALN: 76451 N_COMPUTED_NOTALN: 5572 N_CACHED_NOTALN: 3395 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:21:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 37347 N_CACHED_ALN: 74062 N_COMPUTED_NOTALN: 5381 N_CACHED_NOTALN: 3210 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:23:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 37130 N_CACHED_ALN: 83511 N_COMPUTED_NOTALN: 5525 N_CACHED_NOTALN: 3834 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:23:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 37924 N_CACHED_ALN: 85145 N_COMPUTED_NOTALN: 4316 N_CACHED_NOTALN: 2615 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:24:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 37544 N_CACHED_ALN: 83134 N_COMPUTED_NOTALN: 5599 N_CACHED_NOTALN: 3723 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:24:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 40724 N_CACHED_ALN: 80443 N_COMPUTED_NOTALN: 5364 N_CACHED_NOTALN: 3469 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:25:\n",
      "\t Finished reads; N_TOT_READS: 131066 N_COMPUTED_ALN: 40973 N_CACHED_ALN: 81170 N_COMPUTED_NOTALN: 5415 N_CACHED_NOTALN: 3508 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:25:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:26:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:26:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 37126 N_CACHED_ALN: 83100 N_COMPUTED_NOTALN: 5994 N_CACHED_NOTALN: 3780 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:26:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:27:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:28:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 40024 N_CACHED_ALN: 80651 N_COMPUTED_NOTALN: 5793 N_CACHED_NOTALN: 3532 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:29:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 39612 N_CACHED_ALN: 90364 N_COMPUTED_NOTALN: 5870 N_CACHED_NOTALN: 4154 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:29:\n",
      "\t Finished reads; N_TOT_READS: 140130 N_COMPUTED_ALN: 39643 N_CACHED_ALN: 90454 N_COMPUTED_NOTALN: 5875 N_CACHED_NOTALN: 4158 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:29:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:30:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 39969 N_CACHED_ALN: 90009 N_COMPUTED_NOTALN: 5981 N_CACHED_NOTALN: 4041 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:30:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 40464 N_CACHED_ALN: 92068 N_COMPUTED_NOTALN: 4638 N_CACHED_NOTALN: 2830 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:32:\n",
      "\t Finished reads; N_TOT_READS: 139566 N_COMPUTED_ALN: 39465 N_CACHED_ALN: 89582 N_COMPUTED_NOTALN: 6414 N_CACHED_NOTALN: 4105 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:32:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:32:\n",
      "\t Finished reads; N_TOT_READS: 143680 N_COMPUTED_ALN: 41432 N_CACHED_ALN: 94578 N_COMPUTED_NOTALN: 4763 N_CACHED_NOTALN: 2907 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:32:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:32:\n",
      "\t Finished reads; N_TOT_READS: 144720 N_COMPUTED_ALN: 41171 N_CACHED_ALN: 93151 N_COMPUTED_NOTALN: 6194 N_CACHED_NOTALN: 4204 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:32:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:33:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 42738 N_CACHED_ALN: 87214 N_COMPUTED_NOTALN: 6192 N_CACHED_NOTALN: 3856 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:39:\n",
      "\t Processing reads; N_TOT_READS: 150000 N_COMPUTED_ALN: 45228 N_CACHED_ALN: 94022 N_COMPUTED_NOTALN: 6600 N_CACHED_NOTALN: 4150 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:39:\n",
      "\t Finished reads; N_TOT_READS: 151006 N_COMPUTED_ALN: 45509 N_CACHED_ALN: 94680 N_COMPUTED_NOTALN: 6635 N_CACHED_NOTALN: 4182 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:39:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:40:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:40:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:41:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:44:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:44:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:44:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:44:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:45:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:46:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:46:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:46:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:46:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:47:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:47:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:47:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:47:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:47:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:48:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:48:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:55:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:56:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:36:57:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:11:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:11:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:13:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:30:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:30:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:31:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:45:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:46:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:38:47:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:05:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:05:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:05:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:05:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:07:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:07:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:09:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:09:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:11:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:12:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:12:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:12:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:12:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:12:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:13:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:14:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:14:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:15:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:15:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:16:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:16:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:16:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:39:17:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:15:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle --name sample_ABE8e_PAX5sg_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_S139294.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_S139294.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle --name sample_ABE8e_PAX5sg_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_S139299.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_S139299.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle --name sample_ABE8e_PAX5sg_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_S139300.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_S139300.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle --name sample_ABE8e_PAX5sg_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S139295.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S139295.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle --name sample_ABE8e_PAX5sg_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S139296.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S139296.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle --name sample_ABE8e_PAX5sg_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S139301.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S139301.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle --name sample_ABE8e_PAX5sg_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_1_S139292.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_1_S139292.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle --name sample_ABE8e_PAX5sg_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_2_S139297.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_2_S139297.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:18:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle --name sample_ABE8e_PAX5sg_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_3_S139298.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_3_S139298.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:23:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:23:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:23:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:23:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:23:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:23:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:23:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:23:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:23:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:24:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:24:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:24:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:25:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:25:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:25:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:25:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:25:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:25:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 34 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:59:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:59:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:40:59:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_2/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S139296.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_2/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S139296.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 39 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:04:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:04:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:04:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_3/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S139301.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_3/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S139301.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:05:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:05:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 43 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:09:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:09:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:09:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_1_S139292.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_1_S139292.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 45 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:10:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:10:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:10:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:11:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:11:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_3/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_S139300.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_3/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_S139300.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:12:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1632 N_CACHED_ALN: 6538 N_COMPUTED_NOTALN: 1593 N_CACHED_NOTALN: 237 \n",
      "\n",
      "Completed in 49 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:15:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:15:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:15:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_2/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_2_S139297.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_2/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_2_S139297.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:15:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:15:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 52 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:17:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:18:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:18:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_2/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_S139299.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_2/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_S139299.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:18:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3058 N_CACHED_ALN: 13420 N_COMPUTED_NOTALN: 2951 N_CACHED_NOTALN: 571 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:18:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:18:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:18:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1485 N_CACHED_ALN: 6275 N_COMPUTED_NOTALN: 2013 N_CACHED_NOTALN: 227 \n",
      "\n",
      "Completed in 53 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:19:\n",
      "\t Estimating average read length... \n",
      "\n",
      "Completed in 53 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:19:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:19:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:19:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S139295.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S139295.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:19:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:19:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_S139294.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_S139294.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 54 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:20:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:21:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:21:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_3/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_3_S139298.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_3/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_3_S139298.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg219/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg219_unmerged_PE_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:22:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:22:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:23:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 4252 N_CACHED_ALN: 20622 N_COMPUTED_NOTALN: 4167 N_CACHED_NOTALN: 959 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:23:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1496 N_CACHED_ALN: 6348 N_COMPUTED_NOTALN: 1869 N_CACHED_NOTALN: 287 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:25:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1580 N_CACHED_ALN: 6590 N_COMPUTED_NOTALN: 1609 N_CACHED_NOTALN: 221 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:25:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:25:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:26:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2955 N_CACHED_ALN: 12788 N_COMPUTED_NOTALN: 3754 N_CACHED_NOTALN: 503 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:27:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:27:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:27:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:27:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:28:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 5483 N_CACHED_ALN: 27837 N_COMPUTED_NOTALN: 5385 N_CACHED_NOTALN: 1295 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:28:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1513 N_CACHED_ALN: 6902 N_COMPUTED_NOTALN: 1369 N_CACHED_NOTALN: 216 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:29:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:29:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:30:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2573 N_CACHED_ALN: 13045 N_COMPUTED_NOTALN: 3649 N_CACHED_NOTALN: 733 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:30:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2725 N_CACHED_ALN: 13688 N_COMPUTED_NOTALN: 3050 N_CACHED_NOTALN: 537 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:32:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 4038 N_CACHED_ALN: 19677 N_COMPUTED_NOTALN: 5422 N_CACHED_NOTALN: 863 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:33:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 6539 N_CACHED_ALN: 35150 N_COMPUTED_NOTALN: 6636 N_CACHED_NOTALN: 1675 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:34:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1588 N_CACHED_ALN: 5886 N_COMPUTED_NOTALN: 2234 N_CACHED_NOTALN: 292 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:34:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2866 N_CACHED_ALN: 13880 N_COMPUTED_NOTALN: 2716 N_CACHED_NOTALN: 538 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:35:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1440 N_CACHED_ALN: 6206 N_COMPUTED_NOTALN: 2065 N_CACHED_NOTALN: 289 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:35:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1531 N_CACHED_ALN: 6115 N_COMPUTED_NOTALN: 2052 N_CACHED_NOTALN: 302 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:36:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3853 N_CACHED_ALN: 20860 N_COMPUTED_NOTALN: 4428 N_CACHED_NOTALN: 859 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:36:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3767 N_CACHED_ALN: 19771 N_COMPUTED_NOTALN: 5240 N_CACHED_NOTALN: 1222 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:37:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1611 N_CACHED_ALN: 5985 N_COMPUTED_NOTALN: 2121 N_CACHED_NOTALN: 283 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:38:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 7501 N_CACHED_ALN: 42505 N_COMPUTED_NOTALN: 7933 N_CACHED_NOTALN: 2061 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:39:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 5355 N_CACHED_ALN: 26407 N_COMPUTED_NOTALN: 7042 N_CACHED_NOTALN: 1196 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:40:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 4037 N_CACHED_ALN: 21033 N_COMPUTED_NOTALN: 4041 N_CACHED_NOTALN: 889 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:42:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4962 N_CACHED_ALN: 27965 N_COMPUTED_NOTALN: 5850 N_CACHED_NOTALN: 1223 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:42:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2786 N_CACHED_ALN: 12179 N_COMPUTED_NOTALN: 4312 N_CACHED_NOTALN: 723 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:43:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4756 N_CACHED_ALN: 26546 N_COMPUTED_NOTALN: 6949 N_CACHED_NOTALN: 1749 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:43:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2501 N_CACHED_ALN: 12700 N_COMPUTED_NOTALN: 4081 N_CACHED_NOTALN: 718 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:43:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2667 N_CACHED_ALN: 12629 N_COMPUTED_NOTALN: 3959 N_CACHED_NOTALN: 745 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:44:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 9134 N_CACHED_ALN: 49450 N_COMPUTED_NOTALN: 9096 N_CACHED_NOTALN: 2320 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:44:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2986 N_CACHED_ALN: 12386 N_COMPUTED_NOTALN: 3972 N_CACHED_NOTALN: 656 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:45:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 5100 N_CACHED_ALN: 28382 N_COMPUTED_NOTALN: 5263 N_CACHED_NOTALN: 1255 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:46:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 6621 N_CACHED_ALN: 33115 N_COMPUTED_NOTALN: 8755 N_CACHED_NOTALN: 1509 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:47:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 6137 N_CACHED_ALN: 35041 N_COMPUTED_NOTALN: 7251 N_CACHED_NOTALN: 1571 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:49:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5790 N_CACHED_ALN: 33505 N_COMPUTED_NOTALN: 8525 N_CACHED_NOTALN: 2180 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:49:\n",
      "\t Finished reads; N_TOT_READS: 79392 N_COMPUTED_ALN: 10703 N_CACHED_ALN: 56066 N_COMPUTED_NOTALN: 10063 N_CACHED_NOTALN: 2560 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:49:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:49:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3789 N_CACHED_ALN: 18596 N_COMPUTED_NOTALN: 6377 N_CACHED_NOTALN: 1238 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:49:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3699 N_CACHED_ALN: 19309 N_COMPUTED_NOTALN: 5836 N_CACHED_NOTALN: 1156 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:50:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3608 N_CACHED_ALN: 19227 N_COMPUTED_NOTALN: 5915 N_CACHED_NOTALN: 1250 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:51:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 6127 N_CACHED_ALN: 35750 N_COMPUTED_NOTALN: 6498 N_CACHED_NOTALN: 1625 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:52:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 4090 N_CACHED_ALN: 18843 N_COMPUTED_NOTALN: 5919 N_CACHED_NOTALN: 1148 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:52:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 7670 N_CACHED_ALN: 40031 N_COMPUTED_NOTALN: 10407 N_CACHED_NOTALN: 1892 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:53:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:53:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:53:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:53:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 7175 N_CACHED_ALN: 42180 N_COMPUTED_NOTALN: 8699 N_CACHED_NOTALN: 1946 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:55:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 6920 N_CACHED_ALN: 40330 N_COMPUTED_NOTALN: 10077 N_CACHED_NOTALN: 2673 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:56:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 7300 N_CACHED_ALN: 43055 N_COMPUTED_NOTALN: 7650 N_CACHED_NOTALN: 1995 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:56:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4881 N_CACHED_ALN: 25177 N_COMPUTED_NOTALN: 8225 N_CACHED_NOTALN: 1717 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:56:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4772 N_CACHED_ALN: 25937 N_COMPUTED_NOTALN: 7662 N_CACHED_NOTALN: 1629 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:57:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4568 N_CACHED_ALN: 25856 N_COMPUTED_NOTALN: 7814 N_CACHED_NOTALN: 1762 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:59:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 8504 N_CACHED_ALN: 49147 N_COMPUTED_NOTALN: 9998 N_CACHED_NOTALN: 2351 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:59:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 5019 N_CACHED_ALN: 25438 N_COMPUTED_NOTALN: 7869 N_CACHED_NOTALN: 1674 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:41:59:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 8835 N_CACHED_ALN: 47003 N_COMPUTED_NOTALN: 11923 N_CACHED_NOTALN: 2239 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:02:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 8708 N_CACHED_ALN: 50180 N_COMPUTED_NOTALN: 8773 N_CACHED_NOTALN: 2339 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:02:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 7866 N_CACHED_ALN: 47171 N_COMPUTED_NOTALN: 11799 N_CACHED_NOTALN: 3164 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:03:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5807 N_CACHED_ALN: 31880 N_COMPUTED_NOTALN: 10047 N_CACHED_NOTALN: 2266 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:03:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5824 N_CACHED_ALN: 32591 N_COMPUTED_NOTALN: 9487 N_CACHED_NOTALN: 2098 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:04:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5633 N_CACHED_ALN: 32427 N_COMPUTED_NOTALN: 9642 N_CACHED_NOTALN: 2298 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:04:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 9643 N_CACHED_ALN: 56341 N_COMPUTED_NOTALN: 11304 N_CACHED_NOTALN: 2712 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:06:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5909 N_CACHED_ALN: 32120 N_COMPUTED_NOTALN: 9709 N_CACHED_NOTALN: 2262 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:06:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 9942 N_CACHED_ALN: 53899 N_COMPUTED_NOTALN: 13539 N_CACHED_NOTALN: 2620 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:07:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 9870 N_CACHED_ALN: 57455 N_COMPUTED_NOTALN: 9961 N_CACHED_NOTALN: 2714 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:08:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 8895 N_CACHED_ALN: 54103 N_COMPUTED_NOTALN: 13325 N_CACHED_NOTALN: 3677 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:10:\n",
      "\t Finished reads; N_TOT_READS: 85846 N_COMPUTED_ALN: 10811 N_CACHED_ALN: 57836 N_COMPUTED_NOTALN: 14388 N_CACHED_NOTALN: 2811 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:10:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:10:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 11104 N_CACHED_ALN: 63393 N_COMPUTED_NOTALN: 12477 N_CACHED_NOTALN: 3026 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:10:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 6975 N_CACHED_ALN: 39382 N_COMPUTED_NOTALN: 11112 N_CACHED_NOTALN: 2531 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:10:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 6876 N_CACHED_ALN: 38415 N_COMPUTED_NOTALN: 11873 N_CACHED_NOTALN: 2836 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:11:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 6615 N_CACHED_ALN: 39119 N_COMPUTED_NOTALN: 11429 N_CACHED_NOTALN: 2837 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:12:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 6781 N_CACHED_ALN: 38955 N_COMPUTED_NOTALN: 11444 N_CACHED_NOTALN: 2820 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:12:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 11004 N_CACHED_ALN: 64756 N_COMPUTED_NOTALN: 11149 N_CACHED_NOTALN: 3091 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:14:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:14:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:14:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 10088 N_CACHED_ALN: 60889 N_COMPUTED_NOTALN: 14919 N_CACHED_NOTALN: 4104 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:14:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:16:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 12409 N_CACHED_ALN: 70555 N_COMPUTED_NOTALN: 13707 N_CACHED_NOTALN: 3329 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:17:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 7920 N_CACHED_ALN: 46133 N_COMPUTED_NOTALN: 12891 N_CACHED_NOTALN: 3056 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:17:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 7390 N_CACHED_ALN: 45955 N_COMPUTED_NOTALN: 13190 N_CACHED_NOTALN: 3465 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:17:\n",
      "\t Finished reads; N_TOT_READS: 103215 N_COMPUTED_ALN: 12769 N_CACHED_ALN: 72942 N_COMPUTED_NOTALN: 14074 N_CACHED_NOTALN: 3430 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:17:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:17:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 7785 N_CACHED_ALN: 44972 N_COMPUTED_NOTALN: 13833 N_CACHED_NOTALN: 3410 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:18:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 12201 N_CACHED_ALN: 72069 N_COMPUTED_NOTALN: 12273 N_CACHED_NOTALN: 3457 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:19:\n",
      "\t Finished reads; N_TOT_READS: 96396 N_COMPUTED_ALN: 11051 N_CACHED_ALN: 65171 N_COMPUTED_NOTALN: 15825 N_CACHED_NOTALN: 4349 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:19:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:19:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 7592 N_CACHED_ALN: 45733 N_COMPUTED_NOTALN: 13282 N_CACHED_NOTALN: 3393 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:22:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:22:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:22:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:23:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:23:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:23:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:23:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:23:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 13497 N_CACHED_ALN: 79408 N_COMPUTED_NOTALN: 13342 N_CACHED_NOTALN: 3753 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:23:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 9082 N_CACHED_ALN: 52825 N_COMPUTED_NOTALN: 14603 N_CACHED_NOTALN: 3490 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:23:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:24:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 8444 N_CACHED_ALN: 52726 N_COMPUTED_NOTALN: 14839 N_CACHED_NOTALN: 3991 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:24:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 8646 N_CACHED_ALN: 51742 N_COMPUTED_NOTALN: 15667 N_CACHED_NOTALN: 3945 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:24:\n",
      "\t Finished reads; N_TOT_READS: 112386 N_COMPUTED_ALN: 13844 N_CACHED_ALN: 81148 N_COMPUTED_NOTALN: 13556 N_CACHED_NOTALN: 3838 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:24:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:26:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 8940 N_CACHED_ALN: 52194 N_COMPUTED_NOTALN: 15014 N_CACHED_NOTALN: 3852 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:29:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:29:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:29:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:30:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 10005 N_CACHED_ALN: 59690 N_COMPUTED_NOTALN: 16289 N_CACHED_NOTALN: 4016 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:30:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 9546 N_CACHED_ALN: 59446 N_COMPUTED_NOTALN: 16479 N_CACHED_NOTALN: 4529 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:31:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 9859 N_CACHED_ALN: 58257 N_COMPUTED_NOTALN: 17460 N_CACHED_NOTALN: 4424 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:33:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 10077 N_CACHED_ALN: 58879 N_COMPUTED_NOTALN: 16731 N_CACHED_NOTALN: 4313 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:36:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 10977 N_CACHED_ALN: 66606 N_COMPUTED_NOTALN: 17878 N_CACHED_NOTALN: 4539 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:37:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 10739 N_CACHED_ALN: 66022 N_COMPUTED_NOTALN: 18178 N_CACHED_NOTALN: 5061 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:38:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 10889 N_CACHED_ALN: 64875 N_COMPUTED_NOTALN: 19243 N_CACHED_NOTALN: 4993 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:40:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 11480 N_CACHED_ALN: 65257 N_COMPUTED_NOTALN: 18560 N_CACHED_NOTALN: 4703 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:42:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 12083 N_CACHED_ALN: 73445 N_COMPUTED_NOTALN: 19488 N_CACHED_NOTALN: 4984 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:44:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 12286 N_CACHED_ALN: 72452 N_COMPUTED_NOTALN: 19830 N_CACHED_NOTALN: 5432 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:45:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 12181 N_CACHED_ALN: 71536 N_COMPUTED_NOTALN: 20980 N_CACHED_NOTALN: 5303 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:47:\n",
      "\t Finished reads; N_TOT_READS: 117433 N_COMPUTED_ALN: 13228 N_CACHED_ALN: 78382 N_COMPUTED_NOTALN: 20567 N_CACHED_NOTALN: 5256 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:47:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:47:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 12421 N_CACHED_ALN: 72066 N_COMPUTED_NOTALN: 20307 N_CACHED_NOTALN: 5206 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:50:\n",
      "\t Finished reads; N_TOT_READS: 117501 N_COMPUTED_ALN: 13263 N_CACHED_ALN: 76417 N_COMPUTED_NOTALN: 22171 N_CACHED_NOTALN: 5650 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:50:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:51:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 13649 N_CACHED_ALN: 79125 N_COMPUTED_NOTALN: 21394 N_CACHED_NOTALN: 5832 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:51:\n",
      "\t Finished reads; N_TOT_READS: 120199 N_COMPUTED_ALN: 13679 N_CACHED_ALN: 79254 N_COMPUTED_NOTALN: 21428 N_CACHED_NOTALN: 5838 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:51:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:51:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:51:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:51:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:51:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:51:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:52:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:53:\n",
      "\t Finished reads; N_TOT_READS: 119501 N_COMPUTED_ALN: 13432 N_CACHED_ALN: 78504 N_COMPUTED_NOTALN: 21920 N_CACHED_NOTALN: 5645 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:53:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:55:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:55:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:56:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:56:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:56:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:56:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:58:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:58:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:42:58:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:43:44:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:43:44:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:43:45:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:43:46:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:20:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:20:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:21:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:33:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:34:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:35:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:40:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:40:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:42:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:45:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:45:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:44:47:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:02:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:03:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:04:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:07:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:07:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:08:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:08:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:08:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:08:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:09:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:09:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:10:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:10:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:10:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:45:10:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:07:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 15:46:10:\n",
      "\t \n",
      "\n",
      "ERROR: Offset around cut would be greater than reference sequence length. Please decrease plot_window_size parameter. Cut point: 234 window: 20 reference: 250 \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle --name sample_ABE8e_PAX5sg_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_S139294.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle --name sample_ABE8e_PAX5sg_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_S139300.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle --name sample_ABE8e_PAX5sg_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_S139299.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle --name sample_ABE8e_PAX5sg_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S139295.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle --name sample_ABE8e_PAX5sg_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S139296.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle --name sample_ABE8e_PAX5sg_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S139301.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle --name sample_ABE8e_PAX5sg_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_1_S139292.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle --name sample_ABE8e_PAX5sg_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_2_S139297.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:13:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle --name sample_ABE8e_PAX5sg_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_3_S139298.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ACAGGAAGTCACAGCCTGGT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle/CRISPResso_on_sample_ABE8e_PAX5sg_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle/CRISPResso_on_sample_ABE8e_PAX5sg_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_ABE8e_sg219_unmerged_R2.settings/CRISPRessoBatch_on_ABE8e_sg219_unmerged_R2_middle/CRISPResso_on_sample_ABE8e_PAX5sg_presort_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:20:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:20:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:20:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:20:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:34:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:34:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 16 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:37:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:37:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 18 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:38:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:38:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 19 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:40:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:40:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:41:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3242 N_CACHED_ALN: 4402 N_COMPUTED_NOTALN: 1846 N_CACHED_NOTALN: 510 \n",
      "\n",
      "Completed in 22 seconds\n",
      "\n",
      "Completed in 22 seconds\n",
      "\n",
      "Completed in 22 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:42:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:42:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:42:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:42:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:42:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:42:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 23 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:43:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:43:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 23 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:44:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3423 N_CACHED_ALN: 3866 N_COMPUTED_NOTALN: 2175 N_CACHED_NOTALN: 536 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:44:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:44:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:46:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3741 N_CACHED_ALN: 3518 N_COMPUTED_NOTALN: 2255 N_CACHED_NOTALN: 486 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:46:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3147 N_CACHED_ALN: 4456 N_COMPUTED_NOTALN: 1908 N_CACHED_NOTALN: 489 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:46:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5728 N_CACHED_ALN: 9637 N_COMPUTED_NOTALN: 3474 N_CACHED_NOTALN: 1161 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:49:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3475 N_CACHED_ALN: 4361 N_COMPUTED_NOTALN: 1652 N_CACHED_NOTALN: 512 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:49:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3134 N_CACHED_ALN: 3855 N_COMPUTED_NOTALN: 2460 N_CACHED_NOTALN: 551 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:50:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3426 N_CACHED_ALN: 3667 N_COMPUTED_NOTALN: 2375 N_CACHED_NOTALN: 532 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:50:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6016 N_CACHED_ALN: 8592 N_COMPUTED_NOTALN: 4246 N_CACHED_NOTALN: 1146 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:50:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3159 N_CACHED_ALN: 3864 N_COMPUTED_NOTALN: 2369 N_CACHED_NOTALN: 608 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:51:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7973 N_CACHED_ALN: 15084 N_COMPUTED_NOTALN: 5080 N_CACHED_NOTALN: 1863 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:52:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5619 N_CACHED_ALN: 9604 N_COMPUTED_NOTALN: 3702 N_CACHED_NOTALN: 1075 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:52:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3198 N_CACHED_ALN: 3820 N_COMPUTED_NOTALN: 2444 N_CACHED_NOTALN: 538 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:53:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6728 N_CACHED_ALN: 7720 N_COMPUTED_NOTALN: 4391 N_CACHED_NOTALN: 1161 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:55:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6171 N_CACHED_ALN: 9504 N_COMPUTED_NOTALN: 3221 N_CACHED_NOTALN: 1104 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:56:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 9990 N_CACHED_ALN: 20767 N_COMPUTED_NOTALN: 6568 N_CACHED_NOTALN: 2675 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:56:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5653 N_CACHED_ALN: 8349 N_COMPUTED_NOTALN: 4795 N_CACHED_NOTALN: 1203 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:56:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 8456 N_CACHED_ALN: 13449 N_COMPUTED_NOTALN: 6264 N_CACHED_NOTALN: 1831 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:57:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5596 N_CACHED_ALN: 8382 N_COMPUTED_NOTALN: 4626 N_CACHED_NOTALN: 1396 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:57:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7830 N_CACHED_ALN: 15067 N_COMPUTED_NOTALN: 5406 N_CACHED_NOTALN: 1697 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:57:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6138 N_CACHED_ALN: 7950 N_COMPUTED_NOTALN: 4658 N_CACHED_NOTALN: 1254 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:58:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5667 N_CACHED_ALN: 8386 N_COMPUTED_NOTALN: 4713 N_CACHED_NOTALN: 1234 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:46:59:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9558 N_CACHED_ALN: 12151 N_COMPUTED_NOTALN: 6380 N_CACHED_NOTALN: 1911 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:00:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 8879 N_CACHED_ALN: 14649 N_COMPUTED_NOTALN: 4726 N_CACHED_NOTALN: 1746 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:02:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 9903 N_CACHED_ALN: 20626 N_COMPUTED_NOTALN: 7079 N_CACHED_NOTALN: 2392 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:02:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 10615 N_CACHED_ALN: 18628 N_COMPUTED_NOTALN: 8225 N_CACHED_NOTALN: 2532 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:02:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 8010 N_CACHED_ALN: 12972 N_COMPUTED_NOTALN: 7009 N_CACHED_NOTALN: 2009 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:03:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 13729 N_CACHED_ALN: 24894 N_COMPUTED_NOTALN: 8024 N_CACHED_NOTALN: 3353 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:03:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7835 N_CACHED_ALN: 13191 N_COMPUTED_NOTALN: 6777 N_CACHED_NOTALN: 2197 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:03:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 8648 N_CACHED_ALN: 12639 N_COMPUTED_NOTALN: 6745 N_CACHED_NOTALN: 1968 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:05:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7925 N_CACHED_ALN: 13094 N_COMPUTED_NOTALN: 6997 N_CACHED_NOTALN: 1984 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:05:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 11135 N_CACHED_ALN: 20247 N_COMPUTED_NOTALN: 6190 N_CACHED_NOTALN: 2428 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:05:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 12004 N_CACHED_ALN: 16942 N_COMPUTED_NOTALN: 8312 N_CACHED_NOTALN: 2742 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:07:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 11704 N_CACHED_ALN: 26518 N_COMPUTED_NOTALN: 8677 N_CACHED_NOTALN: 3101 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:07:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 12506 N_CACHED_ALN: 24130 N_COMPUTED_NOTALN: 10098 N_CACHED_NOTALN: 3266 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:08:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 10289 N_CACHED_ALN: 17714 N_COMPUTED_NOTALN: 9213 N_CACHED_NOTALN: 2784 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:09:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 10051 N_CACHED_ALN: 18051 N_COMPUTED_NOTALN: 8858 N_CACHED_NOTALN: 3040 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:10:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 11052 N_CACHED_ALN: 17316 N_COMPUTED_NOTALN: 8882 N_CACHED_NOTALN: 2750 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:10:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 13254 N_CACHED_ALN: 25979 N_COMPUTED_NOTALN: 7609 N_CACHED_NOTALN: 3158 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:11:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 10073 N_CACHED_ALN: 17956 N_COMPUTED_NOTALN: 9158 N_CACHED_NOTALN: 2813 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:11:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 14413 N_CACHED_ALN: 21855 N_COMPUTED_NOTALN: 10194 N_CACHED_NOTALN: 3538 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:12:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 19296 N_CACHED_ALN: 27261 N_COMPUTED_NOTALN: 9635 N_CACHED_NOTALN: 3808 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:12:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 14459 N_CACHED_ALN: 31507 N_COMPUTED_NOTALN: 10210 N_CACHED_NOTALN: 3824 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:14:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 12189 N_CACHED_ALN: 22833 N_COMPUTED_NOTALN: 11345 N_CACHED_NOTALN: 3633 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:14:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 11977 N_CACHED_ALN: 23166 N_COMPUTED_NOTALN: 10874 N_CACHED_NOTALN: 3983 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:15:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 15161 N_CACHED_ALN: 31925 N_COMPUTED_NOTALN: 9070 N_CACHED_NOTALN: 3844 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:15:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 13084 N_CACHED_ALN: 22352 N_COMPUTED_NOTALN: 10988 N_CACHED_NOTALN: 3576 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:17:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 17709 N_CACHED_ALN: 26540 N_COMPUTED_NOTALN: 11975 N_CACHED_NOTALN: 3776 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:17:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 16531 N_CACHED_ALN: 27078 N_COMPUTED_NOTALN: 12023 N_CACHED_NOTALN: 4368 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:17:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 11933 N_CACHED_ALN: 23198 N_COMPUTED_NOTALN: 11220 N_CACHED_NOTALN: 3649 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:19:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 13984 N_CACHED_ALN: 28099 N_COMPUTED_NOTALN: 13406 N_CACHED_NOTALN: 4511 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:19:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 13789 N_CACHED_ALN: 28427 N_COMPUTED_NOTALN: 12867 N_CACHED_NOTALN: 4917 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:21:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 15057 N_CACHED_ALN: 27534 N_COMPUTED_NOTALN: 12966 N_CACHED_NOTALN: 4443 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:21:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 19484 N_CACHED_ALN: 34361 N_COMPUTED_NOTALN: 11834 N_CACHED_NOTALN: 4321 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:21:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 25162 N_CACHED_ALN: 29439 N_COMPUTED_NOTALN: 11200 N_CACHED_NOTALN: 4199 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:22:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 19193 N_CACHED_ALN: 35877 N_COMPUTED_NOTALN: 10496 N_CACHED_NOTALN: 4434 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:22:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 13727 N_CACHED_ALN: 28535 N_COMPUTED_NOTALN: 13236 N_CACHED_NOTALN: 4502 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:25:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 15579 N_CACHED_ALN: 33729 N_COMPUTED_NOTALN: 14821 N_CACHED_NOTALN: 5871 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:26:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 16815 N_CACHED_ALN: 32385 N_COMPUTED_NOTALN: 15503 N_CACHED_NOTALN: 5297 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:27:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 16748 N_CACHED_ALN: 32899 N_COMPUTED_NOTALN: 15005 N_CACHED_NOTALN: 5348 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:27:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 23620 N_CACHED_ALN: 28330 N_COMPUTED_NOTALN: 13886 N_CACHED_NOTALN: 4164 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:27:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 22103 N_CACHED_ALN: 28930 N_COMPUTED_NOTALN: 14062 N_CACHED_NOTALN: 4905 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:29:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 15940 N_CACHED_ALN: 33512 N_COMPUTED_NOTALN: 15205 N_CACHED_NOTALN: 5343 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:30:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 30032 N_CACHED_ALN: 32593 N_COMPUTED_NOTALN: 12680 N_CACHED_NOTALN: 4695 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:30:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 24962 N_CACHED_ALN: 36842 N_COMPUTED_NOTALN: 13448 N_CACHED_NOTALN: 4748 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:31:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 24722 N_CACHED_ALN: 38362 N_COMPUTED_NOTALN: 12004 N_CACHED_NOTALN: 4912 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:34:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 20326 N_CACHED_ALN: 36184 N_COMPUTED_NOTALN: 16912 N_CACHED_NOTALN: 6578 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:36:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 21554 N_CACHED_ALN: 34889 N_COMPUTED_NOTALN: 17669 N_CACHED_NOTALN: 5888 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:36:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 21555 N_CACHED_ALN: 35470 N_COMPUTED_NOTALN: 17080 N_CACHED_NOTALN: 5895 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:36:\n",
      "\t Finished reads; N_TOT_READS: 87012 N_COMPUTED_ALN: 33791 N_CACHED_ALN: 34527 N_COMPUTED_NOTALN: 13652 N_CACHED_NOTALN: 5042 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:36:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:37:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 29146 N_CACHED_ALN: 30568 N_COMPUTED_NOTALN: 15697 N_CACHED_NOTALN: 4589 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:38:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 20748 N_CACHED_ALN: 35971 N_COMPUTED_NOTALN: 17304 N_CACHED_NOTALN: 5977 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:39:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 28271 N_CACHED_ALN: 30247 N_COMPUTED_NOTALN: 16213 N_CACHED_NOTALN: 5269 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:40:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 30773 N_CACHED_ALN: 39051 N_COMPUTED_NOTALN: 15074 N_CACHED_NOTALN: 5102 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:42:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 30900 N_CACHED_ALN: 40238 N_COMPUTED_NOTALN: 13570 N_CACHED_NOTALN: 5292 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:44:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 25517 N_CACHED_ALN: 38199 N_COMPUTED_NOTALN: 19104 N_CACHED_NOTALN: 7180 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:46:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 26645 N_CACHED_ALN: 37110 N_COMPUTED_NOTALN: 19857 N_CACHED_NOTALN: 6388 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:46:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:46:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:46:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:46:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:47:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:47:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 34504 N_CACHED_ALN: 32966 N_COMPUTED_NOTALN: 17402 N_CACHED_NOTALN: 5128 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:47:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 27235 N_CACHED_ALN: 37259 N_COMPUTED_NOTALN: 19208 N_CACHED_NOTALN: 6298 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:48:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 25836 N_CACHED_ALN: 38251 N_COMPUTED_NOTALN: 19407 N_CACHED_NOTALN: 6506 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:48:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 35770 N_CACHED_ALN: 42034 N_COMPUTED_NOTALN: 16652 N_CACHED_NOTALN: 5544 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:49:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 34132 N_CACHED_ALN: 31975 N_COMPUTED_NOTALN: 18221 N_CACHED_NOTALN: 5672 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:50:\n",
      "\t Finished reads; N_TOT_READS: 93769 N_COMPUTED_ALN: 36068 N_CACHED_ALN: 34293 N_COMPUTED_NOTALN: 18094 N_CACHED_NOTALN: 5314 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:50:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:51:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 36561 N_CACHED_ALN: 42701 N_COMPUTED_NOTALN: 15033 N_CACHED_NOTALN: 5705 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:55:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 31176 N_CACHED_ALN: 39956 N_COMPUTED_NOTALN: 21230 N_CACHED_NOTALN: 7638 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:56:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 32470 N_CACHED_ALN: 38798 N_COMPUTED_NOTALN: 21997 N_CACHED_NOTALN: 6735 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:58:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 40702 N_CACHED_ALN: 45175 N_COMPUTED_NOTALN: 18096 N_CACHED_NOTALN: 6027 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:59:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 33234 N_CACHED_ALN: 38719 N_COMPUTED_NOTALN: 21428 N_CACHED_NOTALN: 6619 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:59:\n",
      "\t Finished reads; N_TOT_READS: 112161 N_COMPUTED_ALN: 41549 N_CACHED_ALN: 46066 N_COMPUTED_NOTALN: 18385 N_CACHED_NOTALN: 6161 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:59:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:59:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 31679 N_CACHED_ALN: 39818 N_COMPUTED_NOTALN: 21600 N_CACHED_NOTALN: 6903 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:47:59:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 41648 N_CACHED_ALN: 45739 N_COMPUTED_NOTALN: 16422 N_CACHED_NOTALN: 6191 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:00:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:00:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 39742 N_CACHED_ALN: 33995 N_COMPUTED_NOTALN: 20089 N_CACHED_NOTALN: 6174 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:00:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:01:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:03:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 36032 N_CACHED_ALN: 42510 N_COMPUTED_NOTALN: 23212 N_CACHED_NOTALN: 8246 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:04:\n",
      "\t Finished reads; N_TOT_READS: 104891 N_COMPUTED_ALN: 41993 N_CACHED_ALN: 35455 N_COMPUTED_NOTALN: 20984 N_CACHED_NOTALN: 6459 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:04:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:05:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 37275 N_CACHED_ALN: 41315 N_COMPUTED_NOTALN: 24199 N_CACHED_NOTALN: 7211 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:08:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 36283 N_CACHED_ALN: 42629 N_COMPUTED_NOTALN: 23638 N_CACHED_NOTALN: 7450 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:08:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 38351 N_CACHED_ALN: 41125 N_COMPUTED_NOTALN: 23495 N_CACHED_NOTALN: 7029 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:09:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 46964 N_CACHED_ALN: 48590 N_COMPUTED_NOTALN: 17798 N_CACHED_NOTALN: 6648 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:10:\n",
      "\t Finished reads; N_TOT_READS: 121527 N_COMPUTED_ALN: 47628 N_CACHED_ALN: 49164 N_COMPUTED_NOTALN: 18002 N_CACHED_NOTALN: 6733 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:10:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:11:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:11:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:12:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:12:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 40538 N_CACHED_ALN: 45360 N_COMPUTED_NOTALN: 25213 N_CACHED_NOTALN: 8889 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:15:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 41959 N_CACHED_ALN: 44052 N_COMPUTED_NOTALN: 26207 N_CACHED_NOTALN: 7782 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:16:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:17:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:17:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 40585 N_CACHED_ALN: 45797 N_COMPUTED_NOTALN: 25569 N_CACHED_NOTALN: 8049 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:17:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:18:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 43388 N_CACHED_ALN: 43761 N_COMPUTED_NOTALN: 25296 N_CACHED_NOTALN: 7555 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:22:\n",
      "\t Finished reads; N_TOT_READS: 127778 N_COMPUTED_ALN: 45344 N_CACHED_ALN: 46473 N_COMPUTED_NOTALN: 27718 N_CACHED_NOTALN: 8243 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:22:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:22:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 45444 N_CACHED_ALN: 47903 N_COMPUTED_NOTALN: 27152 N_CACHED_NOTALN: 9501 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:24:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:24:\n",
      "\t Finished reads; N_TOT_READS: 132181 N_COMPUTED_ALN: 46357 N_CACHED_ALN: 48634 N_COMPUTED_NOTALN: 27552 N_CACHED_NOTALN: 9638 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:24:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:24:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:25:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:26:\n",
      "\t Finished reads; N_TOT_READS: 128081 N_COMPUTED_ALN: 47461 N_CACHED_ALN: 45927 N_COMPUTED_NOTALN: 26807 N_CACHED_NOTALN: 7886 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:26:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:27:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 45516 N_CACHED_ALN: 48456 N_COMPUTED_NOTALN: 27458 N_CACHED_NOTALN: 8570 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:27:\n",
      "\t Finished reads; N_TOT_READS: 130553 N_COMPUTED_ALN: 45725 N_CACHED_ALN: 48671 N_COMPUTED_NOTALN: 27552 N_CACHED_NOTALN: 8605 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:27:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:35:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:35:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:36:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:37:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:37:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:37:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:37:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:38:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:40:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:40:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:40:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:41:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:41:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:48:42:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:04:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:04:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:06:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:22:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:22:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:23:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:33:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:33:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:34:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:42:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:42:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:42:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:44:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:48:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:48:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:48:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:48:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:49:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:49:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:55:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:55:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:57:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:57:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:57:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:58:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:58:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:58:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:50:58:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:45:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "Processing EvoCDA_sg802\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:48:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:48:\n",
      "\t Running CRISPResso with 20 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:49:\r\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle --name sample_EvoCDA_802_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_1_S139242.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:49:\r\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle --name sample_EvoCDA_802_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_2_S139243.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:49:\r\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle --name sample_EvoCDA_802_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_3_S139248.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:49:\r\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle --name sample_EvoCDA_802_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_1_S139239.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:49:\r\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle --name sample_EvoCDA_802_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_2_S139244.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:49:\r\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle --name sample_EvoCDA_802_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_3_S139245.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:49:\r\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle --name sample_EvoCDA_802_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_1_S139240.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:49:\r\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle --name sample_EvoCDA_802_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_2_S139241.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:49:\r\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle --name sample_EvoCDA_802_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_3_S139246.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:53:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle/CRISPResso_on_sample_EvoCDA_802_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:53:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle/CRISPResso_on_sample_EvoCDA_802_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:53:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle/CRISPResso_on_sample_EvoCDA_802_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:53:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle/CRISPResso_on_sample_EvoCDA_802_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:53:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle/CRISPResso_on_sample_EvoCDA_802_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:53:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle/CRISPResso_on_sample_EvoCDA_802_presort_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:53:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:53:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle/CRISPResso_on_sample_EvoCDA_802_low_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:53:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle/CRISPResso_on_sample_EvoCDA_802_presort_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:53:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/premerged/CRISPRessoBatch_on_EvoCDA_sg802_premerged_middle/CRISPResso_on_sample_EvoCDA_802_high_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:54:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:54:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:54:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:54:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:54:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:54:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:54:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:59:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:51:59:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:01:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:01:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:03:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:03:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:04:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:04:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:06:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:06:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:06:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:06:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:06:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4847 N_CACHED_ALN: 4591 N_COMPUTED_NOTALN: 380 N_CACHED_NOTALN: 182 \n",
      "\n",
      "Completed in 13 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:08:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4816 N_CACHED_ALN: 4970 N_COMPUTED_NOTALN: 166 N_CACHED_NOTALN: 48 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:08:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:08:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:09:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4925 N_CACHED_ALN: 4881 N_COMPUTED_NOTALN: 146 N_CACHED_NOTALN: 48 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:10:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5163 N_CACHED_ALN: 4641 N_COMPUTED_NOTALN: 154 N_CACHED_NOTALN: 42 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:10:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5135 N_CACHED_ALN: 4166 N_COMPUTED_NOTALN: 523 N_CACHED_NOTALN: 176 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:11:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4688 N_CACHED_ALN: 5070 N_COMPUTED_NOTALN: 177 N_CACHED_NOTALN: 65 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:13:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4701 N_CACHED_ALN: 5083 N_COMPUTED_NOTALN: 166 N_CACHED_NOTALN: 50 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:13:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4980 N_CACHED_ALN: 4839 N_COMPUTED_NOTALN: 121 N_CACHED_NOTALN: 60 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:14:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10078 N_CACHED_ALN: 8887 N_COMPUTED_NOTALN: 713 N_CACHED_NOTALN: 322 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:14:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9543 N_CACHED_ALN: 10029 N_COMPUTED_NOTALN: 317 N_CACHED_NOTALN: 111 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:15:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5817 N_CACHED_ALN: 3992 N_COMPUTED_NOTALN: 157 N_CACHED_NOTALN: 34 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:16:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10409 N_CACHED_ALN: 9204 N_COMPUTED_NOTALN: 272 N_CACHED_NOTALN: 115 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:16:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9833 N_CACHED_ALN: 9731 N_COMPUTED_NOTALN: 327 N_CACHED_NOTALN: 109 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:17:\n",
      "\t Finished reads; N_TOT_READS: 22090 N_COMPUTED_ALN: 11273 N_CACHED_ALN: 9681 N_COMPUTED_NOTALN: 794 N_CACHED_NOTALN: 342 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:17:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:17:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9017 N_CACHED_ALN: 10507 N_COMPUTED_NOTALN: 322 N_CACHED_NOTALN: 154 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:18:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9909 N_CACHED_ALN: 8692 N_COMPUTED_NOTALN: 995 N_CACHED_NOTALN: 404 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:18:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8412 N_CACHED_ALN: 11123 N_COMPUTED_NOTALN: 316 N_CACHED_NOTALN: 149 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:19:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9611 N_CACHED_ALN: 10044 N_COMPUTED_NOTALN: 211 N_CACHED_NOTALN: 134 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:21:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:21:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:21:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:22:\n",
      "\t Finished reads; N_TOT_READS: 29243 N_COMPUTED_ALN: 14390 N_CACHED_ALN: 14237 N_COMPUTED_NOTALN: 463 N_CACHED_NOTALN: 153 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:22:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:23:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15779 N_CACHED_ALN: 13672 N_COMPUTED_NOTALN: 388 N_CACHED_NOTALN: 161 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:23:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 11037 N_CACHED_ALN: 8569 N_COMPUTED_NOTALN: 317 N_CACHED_NOTALN: 77 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:23:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 14721 N_CACHED_ALN: 14633 N_COMPUTED_NOTALN: 493 N_CACHED_NOTALN: 153 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:23:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 13742 N_CACHED_ALN: 15555 N_COMPUTED_NOTALN: 476 N_CACHED_NOTALN: 227 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:25:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 12850 N_CACHED_ALN: 16484 N_COMPUTED_NOTALN: 459 N_CACHED_NOTALN: 207 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:25:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 14127 N_CACHED_ALN: 15349 N_COMPUTED_NOTALN: 313 N_CACHED_NOTALN: 211 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:26:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 14890 N_CACHED_ALN: 13055 N_COMPUTED_NOTALN: 1463 N_CACHED_NOTALN: 592 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:27:\n",
      "\t Finished reads; N_TOT_READS: 31008 N_COMPUTED_ALN: 15414 N_CACHED_ALN: 13478 N_COMPUTED_NOTALN: 1506 N_CACHED_NOTALN: 610 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:27:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:27:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:27:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:28:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:28:\n",
      "\t Finished reads; N_TOT_READS: 35228 N_COMPUTED_ALN: 18852 N_CACHED_ALN: 15737 N_COMPUTED_NOTALN: 455 N_CACHED_NOTALN: 184 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:28:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:29:\n",
      "\t Finished reads; N_TOT_READS: 36026 N_COMPUTED_ALN: 18067 N_CACHED_ALN: 17207 N_COMPUTED_NOTALN: 586 N_CACHED_NOTALN: 166 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:29:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:30:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 16118 N_CACHED_ALN: 13294 N_COMPUTED_NOTALN: 444 N_CACHED_NOTALN: 144 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:31:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 18519 N_CACHED_ALN: 20545 N_COMPUTED_NOTALN: 641 N_CACHED_NOTALN: 295 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:32:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 17746 N_CACHED_ALN: 21377 N_COMPUTED_NOTALN: 615 N_CACHED_NOTALN: 262 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:32:\n",
      "\t Finished reads; N_TOT_READS: 41611 N_COMPUTED_ALN: 19392 N_CACHED_ALN: 21242 N_COMPUTED_NOTALN: 674 N_CACHED_NOTALN: 303 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:32:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:33:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:33:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:33:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 19620 N_CACHED_ALN: 19681 N_COMPUTED_NOTALN: 429 N_CACHED_NOTALN: 270 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:33:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:35:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:35:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:36:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:36:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:36:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:36:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 21318 N_CACHED_ALN: 17894 N_COMPUTED_NOTALN: 579 N_CACHED_NOTALN: 209 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:37:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:38:\n",
      "\t Finished reads; N_TOT_READS: 47100 N_COMPUTED_ALN: 21103 N_CACHED_ALN: 24971 N_COMPUTED_NOTALN: 724 N_CACHED_NOTALN: 302 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:38:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:39:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:39:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:40:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:40:\n",
      "\t Finished reads; N_TOT_READS: 49706 N_COMPUTED_ALN: 24657 N_CACHED_ALN: 24183 N_COMPUTED_NOTALN: 542 N_CACHED_NOTALN: 324 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:40:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:43:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 25734 N_CACHED_ALN: 23277 N_COMPUTED_NOTALN: 712 N_CACHED_NOTALN: 277 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:46:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:46:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:46:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:46:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:46:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:49:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:49:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:50:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:50:\n",
      "\t Finished reads; N_TOT_READS: 57899 N_COMPUTED_ALN: 30289 N_CACHED_ALN: 26460 N_COMPUTED_NOTALN: 839 N_CACHED_NOTALN: 311 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:52:50:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:53:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:53:01:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:53:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:53:02:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:53:02:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:54:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:54:40:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:54:40:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:54:41:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:54:52:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:54:52:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:54:52:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:54:54:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:00:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:00:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:01:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:01:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:02:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:02:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:09:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:09:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:11:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:11:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:11:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:12:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:18:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:18:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:18:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:18:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:19:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:20:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:26:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:26:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:27:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:27:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:28:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:55:28:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:41:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:44:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:44:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:45:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle --name sample_EvoCDA_802_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_1_S139242.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_1_S139242.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:45:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle --name sample_EvoCDA_802_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_2_S139243.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_2_S139243.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:45:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle --name sample_EvoCDA_802_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_3_S139248.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_3_S139248.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:45:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle --name sample_EvoCDA_802_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_1_S139239.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_1_S139239.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:45:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle --name sample_EvoCDA_802_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_2_S139244.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_2_S139244.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:45:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle --name sample_EvoCDA_802_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_3_S139245.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_3_S139245.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:45:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle --name sample_EvoCDA_802_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_1_S139240.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_1_S139240.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:45:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle --name sample_EvoCDA_802_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_2_S139241.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_2_S139241.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:45:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle --name sample_EvoCDA_802_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_3_S139246.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_3_S139246.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:49:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:49:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:49:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:50:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:56:51:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:02:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:02:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:02:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_3_S139248.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_3_S139248.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:02:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:03:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:03:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_1_S139239.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_1_S139239.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:03:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:03:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:03:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_1_S139240.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_1_S139240.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 13 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:04:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:04:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:04:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_3_S139246.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_3_S139246.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 13 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:04:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_2_S139243.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_2_S139243.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Done! \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:05:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_2_S139244.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_2_S139244.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:06:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:06:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 16 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:06:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:06:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:06:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_3_S139245.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_3_S139245.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_low_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 17 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:08:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:08:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:08:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:08:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:08:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_2_S139241.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_2_S139241.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_presort_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:09:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:09:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:10:\n",
      "\t Done! \n",
      "\n",
      "Completed in 19 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:11:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:11:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:11:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:11:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:11:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_1_S139242.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_1_S139242.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg802/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_802_high_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:14:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:14:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:14:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3477 N_CACHED_ALN: 5477 N_COMPUTED_NOTALN: 938 N_CACHED_NOTALN: 108 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:14:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3203 N_CACHED_ALN: 4838 N_COMPUTED_NOTALN: 1868 N_CACHED_NOTALN: 91 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:15:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2757 N_CACHED_ALN: 5011 N_COMPUTED_NOTALN: 2135 N_CACHED_NOTALN: 97 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:16:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2926 N_CACHED_ALN: 4870 N_COMPUTED_NOTALN: 2113 N_CACHED_NOTALN: 91 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:17:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2956 N_CACHED_ALN: 5548 N_COMPUTED_NOTALN: 1376 N_CACHED_NOTALN: 120 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:17:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2843 N_CACHED_ALN: 5019 N_COMPUTED_NOTALN: 2011 N_CACHED_NOTALN: 127 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:19:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5644 N_CACHED_ALN: 12309 N_COMPUTED_NOTALN: 1785 N_CACHED_NOTALN: 262 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:20:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2870 N_CACHED_ALN: 5027 N_COMPUTED_NOTALN: 1944 N_CACHED_NOTALN: 159 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:21:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2291 N_CACHED_ALN: 3476 N_COMPUTED_NOTALN: 4041 N_CACHED_NOTALN: 192 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:23:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5825 N_CACHED_ALN: 10425 N_COMPUTED_NOTALN: 3523 N_CACHED_NOTALN: 227 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:23:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5227 N_CACHED_ALN: 11906 N_COMPUTED_NOTALN: 2572 N_CACHED_NOTALN: 295 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:24:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5338 N_CACHED_ALN: 10414 N_COMPUTED_NOTALN: 4009 N_CACHED_NOTALN: 239 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:25:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5028 N_CACHED_ALN: 10680 N_COMPUTED_NOTALN: 4066 N_CACHED_NOTALN: 226 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:25:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 8732 N_CACHED_ALN: 18304 N_COMPUTED_NOTALN: 2588 N_CACHED_NOTALN: 376 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:26:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4881 N_CACHED_ALN: 10829 N_COMPUTED_NOTALN: 3940 N_CACHED_NOTALN: 350 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:27:\n",
      "\t Finished reads; N_TOT_READS: 23457 N_COMPUTED_ALN: 6615 N_CACHED_ALN: 12091 N_COMPUTED_NOTALN: 4477 N_CACHED_NOTALN: 274 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:27:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:27:\n",
      "\t Finished reads; N_TOT_READS: 24691 N_COMPUTED_ALN: 7745 N_CACHED_ALN: 12536 N_COMPUTED_NOTALN: 4153 N_CACHED_NOTALN: 257 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:27:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:28:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4840 N_CACHED_ALN: 10900 N_COMPUTED_NOTALN: 3877 N_CACHED_NOTALN: 383 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:28:\n",
      "\t Finished reads; N_TOT_READS: 33692 N_COMPUTED_ALN: 10314 N_CACHED_ALN: 20152 N_COMPUTED_NOTALN: 2821 N_CACHED_NOTALN: 405 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:28:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:29:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:29:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:30:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:30:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2054 N_CACHED_ALN: 3175 N_COMPUTED_NOTALN: 4523 N_CACHED_NOTALN: 248 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:30:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:30:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:30:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7593 N_CACHED_ALN: 18328 N_COMPUTED_NOTALN: 3632 N_CACHED_NOTALN: 447 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:30:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:32:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:32:\n",
      "\t Finished reads; N_TOT_READS: 27282 N_COMPUTED_ALN: 7715 N_CACHED_ALN: 14150 N_COMPUTED_NOTALN: 5141 N_CACHED_NOTALN: 276 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:32:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:32:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:33:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:34:\n",
      "\t Finished reads; N_TOT_READS: 35945 N_COMPUTED_ALN: 9725 N_CACHED_ALN: 21518 N_COMPUTED_NOTALN: 4195 N_CACHED_NOTALN: 507 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:34:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:34:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7320 N_CACHED_ALN: 16472 N_COMPUTED_NOTALN: 5712 N_CACHED_NOTALN: 496 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:34:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4720 N_CACHED_ALN: 7469 N_COMPUTED_NOTALN: 7368 N_CACHED_NOTALN: 443 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:35:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:35:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:35:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:36:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7544 N_CACHED_ALN: 16466 N_COMPUTED_NOTALN: 5420 N_CACHED_NOTALN: 570 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:36:\n",
      "\t Finished reads; N_TOT_READS: 31975 N_COMPUTED_ALN: 7935 N_CACHED_ALN: 17511 N_COMPUTED_NOTALN: 6006 N_CACHED_NOTALN: 523 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:36:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:36:\n",
      "\t Finished reads; N_TOT_READS: 21686 N_COMPUTED_ALN: 5257 N_CACHED_ALN: 8160 N_COMPUTED_NOTALN: 7802 N_CACHED_NOTALN: 467 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:36:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:38:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:38:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:38:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:38:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:39:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:39:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:39:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:39:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:39:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:41:\n",
      "\t Finished reads; N_TOT_READS: 35642 N_COMPUTED_ALN: 9446 N_CACHED_ALN: 19301 N_COMPUTED_NOTALN: 6237 N_CACHED_NOTALN: 658 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:41:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:45:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:45:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:45:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3692 N_CACHED_ALN: 6886 N_COMPUTED_NOTALN: 8810 N_CACHED_NOTALN: 612 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:45:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:57:59:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 6099 N_CACHED_ALN: 10668 N_COMPUTED_NOTALN: 12352 N_CACHED_NOTALN: 881 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:58:00:\n",
      "\t Finished reads; N_TOT_READS: 30840 N_COMPUTED_ALN: 6323 N_CACHED_ALN: 10993 N_COMPUTED_NOTALN: 12609 N_CACHED_NOTALN: 915 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:58:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:58:00:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:58:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:58:02:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:58:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:58:02:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:58:02:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:59:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:59:49:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:59:49:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:59:51:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:59:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:59:53:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:59:53:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 15:59:55:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:04:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:04:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:06:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:06:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:06:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:07:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:07:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:08:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:08:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:08:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:08:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:09:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:09:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:10:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:10:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:24:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:24:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:24:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:24:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:25:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:00:25:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:33:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:01:36:\n",
      "\t \n",
      "\n",
      "ERROR: Offset around cut would be greater than reference sequence length. Please decrease plot_window_size parameter. Cut point: 239 window: 20 reference: 250 \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle --name sample_EvoCDA_802_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_1_S139242.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle --name sample_EvoCDA_802_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_2_S139243.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle --name sample_EvoCDA_802_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_high_3_S139248.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle --name sample_EvoCDA_802_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_1_S139239.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle --name sample_EvoCDA_802_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_2_S139244.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle --name sample_EvoCDA_802_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_low_3_S139245.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle --name sample_EvoCDA_802_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_1_S139240.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle --name sample_EvoCDA_802_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_2_S139241.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:39:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle --name sample_EvoCDA_802_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_802_presort_3_S139246.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGTCACAGCCTGGTGAGAT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:44:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_802_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:44:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_802_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:44:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_802_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by tINFO  @ Thu, 05 Oct 2023 16:01:44:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_802_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "he quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:44:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_802_high_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:44:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_802_low_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:44:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_802_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:44:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_802_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:44:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg802_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg802_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_802_presort_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:45:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:45:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:45:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:45:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:45:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:45:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:45:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:45:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:45:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:50:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:50:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:50:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:50:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:51:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:51:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:51:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:51:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:51:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:51:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:51:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:51:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:52:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:52:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:53:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:53:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:54:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:54:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:58:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4440 N_CACHED_ALN: 3191 N_COMPUTED_NOTALN: 2266 N_CACHED_NOTALN: 103 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:58:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5398 N_CACHED_ALN: 2579 N_COMPUTED_NOTALN: 1948 N_CACHED_NOTALN: 75 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:59:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4876 N_CACHED_ALN: 3624 N_COMPUTED_NOTALN: 1394 N_CACHED_NOTALN: 106 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:59:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6409 N_CACHED_ALN: 2456 N_COMPUTED_NOTALN: 1054 N_CACHED_NOTALN: 81 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:01:59:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5137 N_CACHED_ALN: 2599 N_COMPUTED_NOTALN: 2169 N_CACHED_NOTALN: 95 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:00:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4684 N_CACHED_ALN: 3023 N_COMPUTED_NOTALN: 2182 N_CACHED_NOTALN: 111 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:01:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3479 N_CACHED_ALN: 2135 N_COMPUTED_NOTALN: 4206 N_CACHED_NOTALN: 180 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:01:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5336 N_CACHED_ALN: 2526 N_COMPUTED_NOTALN: 2009 N_CACHED_NOTALN: 129 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:05:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9008 N_CACHED_ALN: 7988 N_COMPUTED_NOTALN: 2737 N_CACHED_NOTALN: 267 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:06:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3412 N_CACHED_ALN: 1631 N_COMPUTED_NOTALN: 4720 N_CACHED_NOTALN: 237 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:06:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 11630 N_CACHED_ALN: 6176 N_COMPUTED_NOTALN: 1968 N_CACHED_NOTALN: 226 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:08:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10318 N_CACHED_ALN: 5333 N_COMPUTED_NOTALN: 4156 N_CACHED_NOTALN: 193 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:08:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8888 N_CACHED_ALN: 6581 N_COMPUTED_NOTALN: 4236 N_CACHED_NOTALN: 295 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:08:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10476 N_CACHED_ALN: 5149 N_COMPUTED_NOTALN: 4175 N_CACHED_NOTALN: 200 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:08:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 11454 N_CACHED_ALN: 4695 N_COMPUTED_NOTALN: 3696 N_CACHED_NOTALN: 155 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:09:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9750 N_CACHED_ALN: 5931 N_COMPUTED_NOTALN: 3988 N_CACHED_NOTALN: 331 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:12:\n",
      "\t Finished reads; N_TOT_READS: 24015 N_COMPUTED_ALN: 13018 N_CACHED_ALN: 6012 N_COMPUTED_NOTALN: 4762 N_CACHED_NOTALN: 223 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:12:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:12:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:12:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8427 N_CACHED_ALN: 3602 N_COMPUTED_NOTALN: 7610 N_CACHED_NOTALN: 361 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:14:\n",
      "\t Finished reads; N_TOT_READS: 25180 N_COMPUTED_ALN: 15204 N_CACHED_ALN: 5379 N_COMPUTED_NOTALN: 4429 N_CACHED_NOTALN: 168 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:14:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:15:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 16129 N_CACHED_ALN: 9798 N_COMPUTED_NOTALN: 3750 N_CACHED_NOTALN: 323 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:15:\n",
      "\t Finished reads; N_TOT_READS: 22420 N_COMPUTED_ALN: 9825 N_CACHED_ALN: 3913 N_COMPUTED_NOTALN: 8294 N_CACHED_NOTALN: 388 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:15:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:16:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:16:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:16:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:16:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 19411 N_CACHED_ALN: 7417 N_COMPUTED_NOTALN: 2877 N_CACHED_NOTALN: 295 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:16:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6374 N_CACHED_ALN: 3908 N_COMPUTED_NOTALN: 9174 N_CACHED_NOTALN: 544 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:17:\n",
      "\t Finished reads; N_TOT_READS: 27740 N_COMPUTED_ALN: 16127 N_CACHED_ALN: 6029 N_COMPUTED_NOTALN: 5347 N_CACHED_NOTALN: 237 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:17:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:18:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:18:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:18:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:18:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:18:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:19:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:19:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15766 N_CACHED_ALN: 7950 N_COMPUTED_NOTALN: 5911 N_CACHED_NOTALN: 373 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:19:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 16557 N_CACHED_ALN: 7385 N_COMPUTED_NOTALN: 5626 N_CACHED_NOTALN: 432 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:21:\n",
      "\t Finished reads; N_TOT_READS: 36683 N_COMPUTED_ALN: 20761 N_CACHED_ALN: 11141 N_COMPUTED_NOTALN: 4427 N_CACHED_NOTALN: 354 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:21:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:21:\n",
      "\t Finished reads; N_TOT_READS: 34388 N_COMPUTED_ALN: 22904 N_CACHED_ALN: 7961 N_COMPUTED_NOTALN: 3208 N_CACHED_NOTALN: 315 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:21:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:21:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:21:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:22:\n",
      "\t Finished reads; N_TOT_READS: 32597 N_COMPUTED_ALN: 17548 N_CACHED_ALN: 8342 N_COMPUTED_NOTALN: 6318 N_CACHED_NOTALN: 389 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:22:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:22:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:26:\n",
      "\t Finished reads; N_TOT_READS: 36295 N_COMPUTED_ALN: 21070 N_CACHED_ALN: 8239 N_COMPUTED_NOTALN: 6522 N_CACHED_NOTALN: 464 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:26:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:27:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:27:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:27:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:27:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:27:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:28:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:28:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:28:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:28:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:29:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 11825 N_CACHED_ALN: 4623 N_COMPUTED_NOTALN: 12855 N_CACHED_NOTALN: 697 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:31:\n",
      "\t Finished reads; N_TOT_READS: 31840 N_COMPUTED_ALN: 12827 N_CACHED_ALN: 4802 N_COMPUTED_NOTALN: 13485 N_CACHED_NOTALN: 726 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:31:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:32:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:32:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:33:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:35:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:35:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:02:35:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:44:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:44:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:44:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:45:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:47:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:47:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:48:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:52:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:52:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:52:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:52:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:52:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:52:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:53:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:54:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:54:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:54:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:04:56:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:00:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:01:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:02:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:02:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:02:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:02:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:02:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:02:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:03:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:03:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:04:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:04:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:04:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:04:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:05:05:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:05:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "Processing EvoCDA_sg790\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Running CRISPResso with 20 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle --name sample_EvoCDA_790_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_1_S139267.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle --name sample_EvoCDA_790_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_2_S139272.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle --name sample_EvoCDA_790_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_3_S139273.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle --name sample_EvoCDA_790_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_1_S139268.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle --name sample_EvoCDA_790_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_2_S139269.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle --name sample_EvoCDA_790_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_3_S139274.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle --name sample_EvoCDA_790_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_1_S139265.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle --name sample_EvoCDA_790_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_2_S139270.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:08:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle --name sample_EvoCDA_790_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_3_S139271.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle/CRISPResso_on_sample_EvoCDA_790_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle/CRISPResso_on_sample_EvoCDA_790_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle/CRISPResso_on_sample_EvoCDA_790_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle/CRISPResso_on_sample_EvoCDA_790_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle/CRISPResso_on_sample_EvoCDA_790_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle/CRISPResso_on_sample_EvoCDA_790_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle/CRISPResso_on_sample_EvoCDA_790_high_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:13:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:13:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:14:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle/CRISPResso_on_sample_EvoCDA_790_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:14:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:14:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/premerged/CRISPRessoBatch_on_EvoCDA_sg790_premerged_middle/CRISPResso_on_sample_EvoCDA_790_low_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:14:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:14:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:14:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:14:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:14:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:14:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 3 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:17:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:17:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 3 seconds\n",
      "\n",
      "Completed in 3 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:17:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:17:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:17:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:17:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 4 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:18:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:18:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 4 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:18:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:18:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 4 seconds\n",
      "\n",
      "Completed in 4 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:18:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:18:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:19:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:19:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:20:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:20:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:25:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3943 N_CACHED_ALN: 5151 N_COMPUTED_NOTALN: 684 N_CACHED_NOTALN: 222 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:25:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4221 N_CACHED_ALN: 4946 N_COMPUTED_NOTALN: 593 N_CACHED_NOTALN: 240 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:26:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3928 N_CACHED_ALN: 5285 N_COMPUTED_NOTALN: 539 N_CACHED_NOTALN: 248 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:26:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3874 N_CACHED_ALN: 5472 N_COMPUTED_NOTALN: 444 N_CACHED_NOTALN: 210 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:26:\n",
      "\t Finished reads; N_TOT_READS: 11475 N_COMPUTED_ALN: 4569 N_CACHED_ALN: 5886 N_COMPUTED_NOTALN: 778 N_CACHED_NOTALN: 242 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:26:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:26:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5060 N_CACHED_ALN: 4185 N_COMPUTED_NOTALN: 574 N_CACHED_NOTALN: 181 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:27:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4664 N_CACHED_ALN: 4453 N_COMPUTED_NOTALN: 681 N_CACHED_NOTALN: 202 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:27:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3613 N_CACHED_ALN: 5474 N_COMPUTED_NOTALN: 580 N_CACHED_NOTALN: 333 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:28:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5391 N_CACHED_ALN: 3816 N_COMPUTED_NOTALN: 565 N_CACHED_NOTALN: 228 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:28:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:28:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:28:\n",
      "\t Finished reads; N_TOT_READS: 13001 N_COMPUTED_ALN: 5496 N_CACHED_ALN: 6423 N_COMPUTED_NOTALN: 776 N_CACHED_NOTALN: 306 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:28:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:28:\n",
      "\t Making Plots... \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:29:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:29:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:30:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:30:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:30:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:31:\n",
      "\t Finished reads; N_TOT_READS: 15928 N_COMPUTED_ALN: 6400 N_CACHED_ALN: 8346 N_COMPUTED_NOTALN: 827 N_CACHED_NOTALN: 355 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:31:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:31:\n",
      "\t Finished reads; N_TOT_READS: 14814 N_COMPUTED_ALN: 7324 N_CACHED_ALN: 6374 N_COMPUTED_NOTALN: 811 N_CACHED_NOTALN: 305 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:31:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:33:\n",
      "\t Finished reads; N_TOT_READS: 17421 N_COMPUTED_ALN: 7166 N_CACHED_ALN: 9175 N_COMPUTED_NOTALN: 756 N_CACHED_NOTALN: 324 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:33:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:33:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:33:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:34:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:34:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:34:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:34:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:34:\n",
      "\t Finished reads; N_TOT_READS: 16451 N_COMPUTED_ALN: 7971 N_CACHED_ALN: 7029 N_COMPUTED_NOTALN: 1156 N_CACHED_NOTALN: 295 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:34:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:34:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7233 N_CACHED_ALN: 10983 N_COMPUTED_NOTALN: 1116 N_CACHED_NOTALN: 668 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:35:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:35:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:35:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:36:\n",
      "\t Finished reads; N_TOT_READS: 22460 N_COMPUTED_ALN: 8218 N_CACHED_ALN: 12244 N_COMPUTED_NOTALN: 1273 N_CACHED_NOTALN: 725 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:36:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:37:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:37:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:37:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:37:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:37:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:38:\n",
      "\t Finished reads; N_TOT_READS: 19903 N_COMPUTED_ALN: 10903 N_CACHED_ALN: 7474 N_COMPUTED_NOTALN: 1086 N_CACHED_NOTALN: 440 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:38:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:38:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4571 N_CACHED_ALN: 4623 N_COMPUTED_NOTALN: 632 N_CACHED_NOTALN: 174 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:39:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:39:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:40:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:42:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:42:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:42:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:42:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:42:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:46:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8074 N_CACHED_ALN: 10332 N_COMPUTED_NOTALN: 1070 N_CACHED_NOTALN: 524 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:53:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 11238 N_CACHED_ALN: 16368 N_COMPUTED_NOTALN: 1496 N_CACHED_NOTALN: 898 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:06:59:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 14306 N_CACHED_ALN: 22469 N_COMPUTED_NOTALN: 1914 N_CACHED_NOTALN: 1311 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:07:06:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 17350 N_CACHED_ALN: 28661 N_COMPUTED_NOTALN: 2307 N_CACHED_NOTALN: 1682 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:07:10:\n",
      "\t Finished reads; N_TOT_READS: 54599 N_COMPUTED_ALN: 18977 N_CACHED_ALN: 31322 N_COMPUTED_NOTALN: 2498 N_CACHED_NOTALN: 1802 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:07:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:07:10:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:07:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:07:17:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:07:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:07:18:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:07:18:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:08:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:08:55:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:08:56:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:08:57:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:02:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:03:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:04:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:05:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:05:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:06:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:06:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:07:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:07:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:07:\n",
      "\t Analysis Complete! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:07:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:08:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:08:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:09:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:09:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:10:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:10:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:12:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:16:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:16:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:17:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:38:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:38:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:40:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:40:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:40:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:09:41:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:47:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle --name sample_EvoCDA_790_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_1_S139267.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_1_S139267.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle --name sample_EvoCDA_790_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_2_S139272.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_2_S139272.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle --name sample_EvoCDA_790_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_3_S139273.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_3_S139273.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle --name sample_EvoCDA_790_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_1_S139268.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_1_S139268.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle --name sample_EvoCDA_790_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_2_S139269.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_2_S139269.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle --name sample_EvoCDA_790_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_3_S139274.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_3_S139274.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\r\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle --name sample_EvoCDA_790_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_1_S139265.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_1_S139265.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\r\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle --name sample_EvoCDA_790_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_2_S139270.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_2_S139270.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:51:\r\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle --name sample_EvoCDA_790_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_3_S139271.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_3_S139271.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:10:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:06:\n",
      "\t Estimating average read length... \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:07:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:07:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:07:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_3_S139273.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_3_S139273.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:07:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:07:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_2_S139270.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_2_S139270.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:08:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:08:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_2_S139269.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_2_S139269.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_3_S139271.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_3_S139271.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:09:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_3_S139274.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_3_S139274.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 13 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:10:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_1_S139267.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_1_S139267.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:11:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 15 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:13:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:13:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:13:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_1_S139265.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_1_S139265.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_presort_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:13:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:13:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:15:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:15:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 19 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:17:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:17:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:17:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_1_S139268.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_1_S139268.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_low_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:20:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:20:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:24:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2338 N_CACHED_ALN: 4047 N_COMPUTED_NOTALN: 3362 N_CACHED_NOTALN: 253 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:26:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1294 N_CACHED_ALN: 3456 N_COMPUTED_NOTALN: 4969 N_CACHED_NOTALN: 281 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:29:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1410 N_CACHED_ALN: 3165 N_COMPUTED_NOTALN: 5127 N_CACHED_NOTALN: 298 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:29:\n",
      "\t Finished reads; N_TOT_READS: 13676 N_COMPUTED_ALN: 3359 N_CACHED_ALN: 5542 N_COMPUTED_NOTALN: 4437 N_CACHED_NOTALN: 338 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:29:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "Completed in 31 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:29:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:29:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:29:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_2_S139272.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_2_S139272.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_790_high_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:29:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1942 N_CACHED_ALN: 3178 N_COMPUTED_NOTALN: 4600 N_CACHED_NOTALN: 280 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:30:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1535 N_CACHED_ALN: 3626 N_COMPUTED_NOTALN: 4628 N_CACHED_NOTALN: 211 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:30:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:30:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:30:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1754 N_CACHED_ALN: 2987 N_COMPUTED_NOTALN: 5036 N_CACHED_NOTALN: 223 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:30:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:34:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:34:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:34:\n",
      "\t Finished reads; N_TOT_READS: 15286 N_COMPUTED_ALN: 2336 N_CACHED_ALN: 5446 N_COMPUTED_NOTALN: 7058 N_CACHED_NOTALN: 446 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:34:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:35:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:35:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:36:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:36:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1263 N_CACHED_ALN: 2738 N_COMPUTED_NOTALN: 5744 N_CACHED_NOTALN: 255 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:38:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1283 N_CACHED_ALN: 3275 N_COMPUTED_NOTALN: 5080 N_CACHED_NOTALN: 362 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:38:\n",
      "\t Finished reads; N_TOT_READS: 15537 N_COMPUTED_ALN: 3094 N_CACHED_ALN: 5123 N_COMPUTED_NOTALN: 6844 N_CACHED_NOTALN: 476 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:38:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:39:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:39:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:40:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:40:\n",
      "\t Finished reads; N_TOT_READS: 17223 N_COMPUTED_ALN: 2728 N_CACHED_ALN: 5838 N_COMPUTED_NOTALN: 8138 N_CACHED_NOTALN: 519 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:40:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:41:\n",
      "\t Finished reads; N_TOT_READS: 15968 N_COMPUTED_ALN: 2850 N_CACHED_ALN: 4929 N_COMPUTED_NOTALN: 7799 N_CACHED_NOTALN: 390 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:41:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:41:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:41:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:41:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:42:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:42:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:42:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:42:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:42:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:45:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3245 N_CACHED_ALN: 7593 N_COMPUTED_NOTALN: 8669 N_CACHED_NOTALN: 493 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:47:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1614 N_CACHED_ALN: 4465 N_COMPUTED_NOTALN: 3563 N_CACHED_NOTALN: 358 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:48:\n",
      "\t Finished reads; N_TOT_READS: 22035 N_COMPUTED_ALN: 3733 N_CACHED_ALN: 8446 N_COMPUTED_NOTALN: 9326 N_CACHED_NOTALN: 530 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:48:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:50:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:50:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:50:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:55:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2317 N_CACHED_ALN: 6669 N_COMPUTED_NOTALN: 10121 N_CACHED_NOTALN: 893 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:56:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2367 N_CACHED_ALN: 5700 N_COMPUTED_NOTALN: 11330 N_CACHED_NOTALN: 603 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:11:59:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2926 N_CACHED_ALN: 9444 N_COMPUTED_NOTALN: 6804 N_CACHED_NOTALN: 826 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:06:\n",
      "\t Finished reads; N_TOT_READS: 26110 N_COMPUTED_ALN: 3529 N_CACHED_ALN: 7756 N_COMPUTED_NOTALN: 14059 N_CACHED_NOTALN: 766 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:06:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:07:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:07:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:07:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:10:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 4299 N_CACHED_ALN: 14666 N_COMPUTED_NOTALN: 9710 N_CACHED_NOTALN: 1325 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:12:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3656 N_CACHED_ALN: 10078 N_COMPUTED_NOTALN: 14938 N_CACHED_NOTALN: 1328 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:17:\n",
      "\t Finished reads; N_TOT_READS: 33282 N_COMPUTED_ALN: 4272 N_CACHED_ALN: 11326 N_COMPUTED_NOTALN: 16278 N_CACHED_NOTALN: 1406 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:17:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:19:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:19:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:19:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:22:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 5574 N_CACHED_ALN: 19877 N_COMPUTED_NOTALN: 12714 N_CACHED_NOTALN: 1835 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:34:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 7239 N_CACHED_ALN: 24928 N_COMPUTED_NOTALN: 15584 N_CACHED_NOTALN: 2249 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:42:\n",
      "\t Finished reads; N_TOT_READS: 56712 N_COMPUTED_ALN: 8589 N_CACHED_ALN: 28432 N_COMPUTED_NOTALN: 17177 N_CACHED_NOTALN: 2514 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:42:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:42:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:45:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:45:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:12:46:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:29:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:29:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:30:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:51:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:51:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:51:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:51:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:51:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:52:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:52:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:53:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:58:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:58:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:59:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:59:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:13:59:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:00:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:10:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:11:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:12:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:26:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:26:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:27:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:28:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:28:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:29:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:45:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:45:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:46:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:46:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:46:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:14:47:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:15:57:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle --name sample_EvoCDA_790_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_1_S139267.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle --name sample_EvoCDA_790_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_2_S139272.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle --name sample_EvoCDA_790_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_3_S139273.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle --name sample_EvoCDA_790_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_1_S139268.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle --name sample_EvoCDA_790_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_2_S139269.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle --name sample_EvoCDA_790_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_3_S139274.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle --name sample_EvoCDA_790_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_1_S139265.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle --name sample_EvoCDA_790_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_2_S139270.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:00:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle --name sample_EvoCDA_790_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_3_S139271.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:05:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_790_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:05:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_790_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:05:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_790_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:05:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_790_high_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:05:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_790_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by t\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "he quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:05:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_790_low_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:05:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_790_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:05:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_790_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:05:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg790/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_790_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:06:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:06:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:06:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:06:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:06:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:06:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:06:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:06:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:07:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:13:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:13:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:15:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:15:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:15:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:15:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:16:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:16:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:17:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:17:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:17:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:17:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:19:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:19:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 13 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:20:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:20:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:25:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2100 N_CACHED_ALN: 2007 N_COMPUTED_NOTALN: 5329 N_CACHED_NOTALN: 564 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:25:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2434 N_CACHED_ALN: 2391 N_COMPUTED_NOTALN: 4403 N_CACHED_NOTALN: 772 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:27:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2178 N_CACHED_ALN: 1682 N_COMPUTED_NOTALN: 5612 N_CACHED_NOTALN: 528 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:28:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2278 N_CACHED_ALN: 2088 N_COMPUTED_NOTALN: 5153 N_CACHED_NOTALN: 481 \n",
      "\n",
      "Completed in 21 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:29:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:29:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:29:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2085 N_CACHED_ALN: 1741 N_COMPUTED_NOTALN: 5639 N_CACHED_NOTALN: 535 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:29:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2091 N_CACHED_ALN: 1773 N_COMPUTED_NOTALN: 5447 N_CACHED_NOTALN: 689 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:31:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2025 N_CACHED_ALN: 1983 N_COMPUTED_NOTALN: 5303 N_CACHED_NOTALN: 689 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:31:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1975 N_CACHED_ALN: 2013 N_COMPUTED_NOTALN: 5293 N_CACHED_NOTALN: 719 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:35:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3794 N_CACHED_ALN: 4544 N_COMPUTED_NOTALN: 10330 N_CACHED_NOTALN: 1332 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:35:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4483 N_CACHED_ALN: 5232 N_COMPUTED_NOTALN: 8559 N_CACHED_NOTALN: 1726 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:38:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2390 N_CACHED_ALN: 3042 N_COMPUTED_NOTALN: 4035 N_CACHED_NOTALN: 533 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:39:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4083 N_CACHED_ALN: 3660 N_COMPUTED_NOTALN: 11046 N_CACHED_NOTALN: 1211 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:39:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4355 N_CACHED_ALN: 4587 N_COMPUTED_NOTALN: 9940 N_CACHED_NOTALN: 1118 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:40:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4091 N_CACHED_ALN: 3848 N_COMPUTED_NOTALN: 10540 N_CACHED_NOTALN: 1521 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:40:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3958 N_CACHED_ALN: 3761 N_COMPUTED_NOTALN: 11074 N_CACHED_NOTALN: 1207 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:41:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3926 N_CACHED_ALN: 4300 N_COMPUTED_NOTALN: 10248 N_CACHED_NOTALN: 1526 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:42:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3830 N_CACHED_ALN: 4240 N_COMPUTED_NOTALN: 10348 N_CACHED_NOTALN: 1582 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:43:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 6187 N_CACHED_ALN: 8846 N_COMPUTED_NOTALN: 12449 N_CACHED_NOTALN: 2518 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:43:\n",
      "\t Finished reads; N_TOT_READS: 28946 N_COMPUTED_ALN: 5134 N_CACHED_ALN: 7225 N_COMPUTED_NOTALN: 14532 N_CACHED_NOTALN: 2055 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:43:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:43:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:45:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:45:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:45:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:47:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4389 N_CACHED_ALN: 6340 N_COMPUTED_NOTALN: 7907 N_CACHED_NOTALN: 1364 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:48:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5990 N_CACHED_ALN: 7896 N_COMPUTED_NOTALN: 14248 N_CACHED_NOTALN: 1866 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:48:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5454 N_CACHED_ALN: 6436 N_COMPUTED_NOTALN: 16029 N_CACHED_NOTALN: 2081 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:49:\n",
      "\t Finished reads; N_TOT_READS: 36792 N_COMPUTED_ALN: 7420 N_CACHED_ALN: 11398 N_COMPUTED_NOTALN: 14955 N_CACHED_NOTALN: 3019 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:49:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:50:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5667 N_CACHED_ALN: 6550 N_COMPUTED_NOTALN: 15433 N_CACHED_NOTALN: 2350 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:51:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5358 N_CACHED_ALN: 6322 N_COMPUTED_NOTALN: 16313 N_CACHED_NOTALN: 2007 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:51:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:51:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:51:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:51:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:51:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:52:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5572 N_CACHED_ALN: 6926 N_COMPUTED_NOTALN: 14989 N_CACHED_NOTALN: 2513 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:53:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5571 N_CACHED_ALN: 6616 N_COMPUTED_NOTALN: 15250 N_CACHED_NOTALN: 2563 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:54:\n",
      "\t Finished reads; N_TOT_READS: 36020 N_COMPUTED_ALN: 6334 N_CACHED_ALN: 8219 N_COMPUTED_NOTALN: 18895 N_CACHED_NOTALN: 2572 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:54:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:56:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:56:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:56:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:56:\n",
      "\t Finished reads; N_TOT_READS: 39583 N_COMPUTED_ALN: 7664 N_CACHED_ALN: 11170 N_COMPUTED_NOTALN: 18247 N_CACHED_NOTALN: 2502 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:56:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:57:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 6315 N_CACHED_ALN: 9724 N_COMPUTED_NOTALN: 11765 N_CACHED_NOTALN: 2196 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:59:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:59:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:16:59:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:00:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 7066 N_CACHED_ALN: 9527 N_COMPUTED_NOTALN: 20187 N_CACHED_NOTALN: 3220 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:00:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 6700 N_CACHED_ALN: 9075 N_COMPUTED_NOTALN: 21384 N_CACHED_NOTALN: 2841 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:00:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 6978 N_CACHED_ALN: 10135 N_COMPUTED_NOTALN: 19429 N_CACHED_NOTALN: 3458 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:02:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 6858 N_CACHED_ALN: 9709 N_COMPUTED_NOTALN: 19835 N_CACHED_NOTALN: 3598 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:03:\n",
      "\t Finished reads; N_TOT_READS: 43419 N_COMPUTED_ALN: 7668 N_CACHED_ALN: 10563 N_COMPUTED_NOTALN: 21740 N_CACHED_NOTALN: 3448 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:03:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:04:\n",
      "\t Finished reads; N_TOT_READS: 43470 N_COMPUTED_ALN: 7234 N_CACHED_ALN: 10107 N_COMPUTED_NOTALN: 23049 N_CACHED_NOTALN: 3080 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:04:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:06:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:06:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:06:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 8225 N_CACHED_ALN: 13224 N_COMPUTED_NOTALN: 15563 N_CACHED_NOTALN: 2988 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:06:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:06:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:06:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:07:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:10:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 8475 N_CACHED_ALN: 13365 N_COMPUTED_NOTALN: 23779 N_CACHED_NOTALN: 4381 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:10:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 8218 N_CACHED_ALN: 12805 N_COMPUTED_NOTALN: 24353 N_CACHED_NOTALN: 4624 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:14:\n",
      "\t Finished reads; N_TOT_READS: 54565 N_COMPUTED_ALN: 9240 N_CACHED_ALN: 14857 N_COMPUTED_NOTALN: 25763 N_CACHED_NOTALN: 4705 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:14:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:16:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 9960 N_CACHED_ALN: 16841 N_COMPUTED_NOTALN: 19377 N_CACHED_NOTALN: 3822 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:16:\n",
      "\t Finished reads; N_TOT_READS: 56390 N_COMPUTED_ALN: 9129 N_CACHED_ALN: 14858 N_COMPUTED_NOTALN: 27198 N_CACHED_NOTALN: 5205 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:16:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:18:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:18:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:18:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:19:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:19:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:20:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:23:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 11477 N_CACHED_ALN: 21089 N_COMPUTED_NOTALN: 22701 N_CACHED_NOTALN: 4733 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:30:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 12932 N_CACHED_ALN: 25393 N_COMPUTED_NOTALN: 26022 N_CACHED_NOTALN: 5653 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:38:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 14643 N_CACHED_ALN: 29507 N_COMPUTED_NOTALN: 29346 N_CACHED_NOTALN: 6504 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:44:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 16093 N_CACHED_ALN: 34045 N_COMPUTED_NOTALN: 32495 N_CACHED_NOTALN: 7367 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:50:\n",
      "\t Finished reads; N_TOT_READS: 97480 N_COMPUTED_ALN: 17498 N_CACHED_ALN: 37359 N_COMPUTED_NOTALN: 34773 N_CACHED_NOTALN: 7850 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:50:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:55:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:55:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:17:56:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:18:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:18:57:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:18:57:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:18:59:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:18:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:18:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:19:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:24:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:24:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:25:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:31:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:31:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:31:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:31:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:32:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:32:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:37:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:37:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:37:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:38:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:38:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:38:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:39:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:39:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:40:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:19:40:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:20:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:20:06:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:20:07:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:20:08:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:20:08:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:20:08:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:20:09:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:21:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_1_S139267.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_2_S139272.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_3_S139273.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_1_S139268.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_2_S139269.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_3_S139274.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_1_S139265.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_2_S139270.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:24:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_3_S139271.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_790_presort_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_790_high_1 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GCTCTAGTGAAAGCCAGTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GCTCTAGTGAAAGCCAGTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_790_high_3 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GCTCTAGTGAAAGCCAGTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_790_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GCTCTAGTGAAAGCCAGTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_790_presort_1 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GCTCTAGTGAAAGCCAGTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_790_high_2 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GCTCTAGTGAAAGCCAGTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_790_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GCTCTAGTGAAAGCCAGTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_790_presort_3 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:21:29:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GCTCTAGTGAAAGCCAGTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_790_low_2 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GCTCTAGTGAAAGCCAGTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t CRISPResso command failed (return value 2) on batch #0: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_1_S139267.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t CRISPResso command failed (return value 2) on batch #5: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_3_S139274.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t CRISPResso command failed (return value 2) on batch #8: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_3_S139271.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t CRISPResso command failed (return value 2) on batch #7: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_2_S139270.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t CRISPResso command failed (return value 2) on batch #6: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_pre_1_S139265.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t CRISPResso command failed (return value 2) on batch #4: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_2_S139269.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t CRISPResso command failed (return value 2) on batch #2: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_3_S139273.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t CRISPResso command failed (return value 2) on batch #1: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_2_S139272.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t CRISPResso command failed (return value 2) on batch #3: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_low_1_S139268.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "CRISPResso2 failed\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:21:30:\n",
      "\t \n",
      "\n",
      "ERROR: CRISPResso batch #0 failed. For more information, try running the command: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg790_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg790_unmerged_R2_middle --name sample_EvoCDA_790_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_790_high_1_S139267.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GCTCTAGTGAAAGCCAGTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "Processing EvoCDA_sg791\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Running CRISPResso with 20 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle --name sample_EvoCDA_791_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_1_S139224.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle --name sample_EvoCDA_791_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_2_S139225.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle --name sample_EvoCDA_791_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_3_S139230.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle --name sample_EvoCDA_791_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_1_S139302.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle --name sample_EvoCDA_791_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_2_S139226.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle --name sample_EvoCDA_791_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_3_S139227.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle --name sample_EvoCDA_791_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_1_S139303.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle --name sample_EvoCDA_791_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_2_S139223.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:33:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle --name sample_EvoCDA_791_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_3_S139228.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle/CRISPResso_on_sample_EvoCDA_791_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle/CRISPResso_on_sample_EvoCDA_791_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle/CRISPResso_on_sample_EvoCDA_791_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle/CRISPResso_on_sample_EvoCDA_791_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle/CRISPResso_on_sample_EvoCDA_791_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle/CRISPResso_on_sample_EvoCDA_791_presort_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle/CRISPResso_on_sample_EvoCDA_791_high_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle/CRISPResso_on_sample_EvoCDA_791_low_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/premerged/CRISPRessoBatch_on_EvoCDA_sg791_premerged_middle/CRISPResso_on_sample_EvoCDA_791_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 4 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:44:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:44:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:44:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:44:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:45:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:45:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:46:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:46:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:48:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:48:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:48:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:48:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:49:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:49:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:51:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:51:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:53:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5377 N_CACHED_ALN: 4135 N_COMPUTED_NOTALN: 377 N_CACHED_NOTALN: 111 \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:53:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:53:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:54:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5195 N_CACHED_ALN: 4291 N_COMPUTED_NOTALN: 354 N_CACHED_NOTALN: 160 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:55:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5384 N_CACHED_ALN: 4014 N_COMPUTED_NOTALN: 450 N_CACHED_NOTALN: 152 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:57:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5297 N_CACHED_ALN: 4625 N_COMPUTED_NOTALN: 61 N_CACHED_NOTALN: 17 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:57:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5757 N_CACHED_ALN: 3959 N_COMPUTED_NOTALN: 204 N_CACHED_NOTALN: 80 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:21:59:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5748 N_CACHED_ALN: 3887 N_COMPUTED_NOTALN: 269 N_CACHED_NOTALN: 96 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:00:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5563 N_CACHED_ALN: 3871 N_COMPUTED_NOTALN: 400 N_CACHED_NOTALN: 166 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:01:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5682 N_CACHED_ALN: 4187 N_COMPUTED_NOTALN: 95 N_CACHED_NOTALN: 36 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:02:\n",
      "\t Finished reads; N_TOT_READS: 18308 N_COMPUTED_ALN: 10104 N_CACHED_ALN: 7309 N_COMPUTED_NOTALN: 676 N_CACHED_NOTALN: 219 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:02:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:04:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5730 N_CACHED_ALN: 4132 N_COMPUTED_NOTALN: 110 N_CACHED_NOTALN: 28 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:04:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10695 N_CACHED_ALN: 8283 N_COMPUTED_NOTALN: 711 N_CACHED_NOTALN: 311 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Finished reads; N_TOT_READS: 21396 N_COMPUTED_ALN: 11544 N_CACHED_ALN: 8753 N_COMPUTED_NOTALN: 770 N_CACHED_NOTALN: 329 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Finished reads; N_TOT_READS: 19992 N_COMPUTED_ALN: 10633 N_CACHED_ALN: 8110 N_COMPUTED_NOTALN: 934 N_CACHED_NOTALN: 315 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:06:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:07:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10753 N_CACHED_ALN: 8705 N_COMPUTED_NOTALN: 359 N_CACHED_NOTALN: 183 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:08:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10587 N_CACHED_ALN: 9227 N_COMPUTED_NOTALN: 127 N_CACHED_NOTALN: 59 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10687 N_CACHED_ALN: 8574 N_COMPUTED_NOTALN: 501 N_CACHED_NOTALN: 238 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10221 N_CACHED_ALN: 8652 N_COMPUTED_NOTALN: 765 N_CACHED_NOTALN: 362 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10632 N_CACHED_ALN: 9088 N_COMPUTED_NOTALN: 191 N_CACHED_NOTALN: 89 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:10:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:11:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:11:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:12:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10761 N_CACHED_ALN: 9005 N_COMPUTED_NOTALN: 169 N_CACHED_NOTALN: 65 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:15:\n",
      "\t Finished reads; N_TOT_READS: 27500 N_COMPUTED_ALN: 14660 N_CACHED_ALN: 12074 N_COMPUTED_NOTALN: 510 N_CACHED_NOTALN: 256 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:15:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:18:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15422 N_CACHED_ALN: 14287 N_COMPUTED_NOTALN: 195 N_CACHED_NOTALN: 96 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:19:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15281 N_CACHED_ALN: 14309 N_COMPUTED_NOTALN: 280 N_CACHED_NOTALN: 130 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:20:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15272 N_CACHED_ALN: 13061 N_COMPUTED_NOTALN: 1116 N_CACHED_NOTALN: 551 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:20:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15882 N_CACHED_ALN: 12992 N_COMPUTED_NOTALN: 777 N_CACHED_NOTALN: 349 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:21:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15526 N_CACHED_ALN: 14108 N_COMPUTED_NOTALN: 255 N_CACHED_NOTALN: 111 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:21:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:21:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:22:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:25:\n",
      "\t Finished reads; N_TOT_READS: 35024 N_COMPUTED_ALN: 18585 N_CACHED_ALN: 16098 N_COMPUTED_NOTALN: 238 N_CACHED_NOTALN: 103 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:25:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:25:\n",
      "\t Finished reads; N_TOT_READS: 34333 N_COMPUTED_ALN: 18094 N_CACHED_ALN: 14909 N_COMPUTED_NOTALN: 909 N_CACHED_NOTALN: 421 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:25:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:30:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 20902 N_CACHED_ALN: 18539 N_COMPUTED_NOTALN: 393 N_CACHED_NOTALN: 166 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:30:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 20547 N_CACHED_ALN: 18970 N_COMPUTED_NOTALN: 339 N_CACHED_NOTALN: 144 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:31:\n",
      "\t Finished reads; N_TOT_READS: 37750 N_COMPUTED_ALN: 19136 N_CACHED_ALN: 16521 N_COMPUTED_NOTALN: 1373 N_CACHED_NOTALN: 720 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:31:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:32:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:32:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:32:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:32:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:33:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:33:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:35:\n",
      "\t Finished reads; N_TOT_READS: 44921 N_COMPUTED_ALN: 23633 N_CACHED_ALN: 20652 N_COMPUTED_NOTALN: 443 N_CACHED_NOTALN: 193 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:35:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:38:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:38:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:39:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:39:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 25288 N_CACHED_ALN: 24121 N_COMPUTED_NOTALN: 411 N_CACHED_NOTALN: 180 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:44:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:44:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:44:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:44:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:45:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:45:\n",
      "\t Finished reads; N_TOT_READS: 56516 N_COMPUTED_ALN: 28434 N_CACHED_ALN: 27405 N_COMPUTED_NOTALN: 461 N_CACHED_NOTALN: 216 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:45:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:55:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:55:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:22:56:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:16:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:17:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:18:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:22:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:22:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:23:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:31:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:31:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:33:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:52:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:52:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:52:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:24:54:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:00:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:01:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:02:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:04:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:04:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:04:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:04:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:06:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:06:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:06:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:06:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:08:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:13:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:13:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:15:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:15:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:15:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:25:16:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:27:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:30:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:30:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:31:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle --name sample_EvoCDA_791_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_1_S139224.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_1_S139224.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:31:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle --name sample_EvoCDA_791_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_3_S139230.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_3_S139230.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:31:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle --name sample_EvoCDA_791_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_2_S139225.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_2_S139225.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:31:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle --name sample_EvoCDA_791_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_1_S139302.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_1_S139302.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:31:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle --name sample_EvoCDA_791_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_2_S139226.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_2_S139226.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:31:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle --name sample_EvoCDA_791_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_3_S139227.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_3_S139227.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:31:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle --name sample_EvoCDA_791_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_1_S139303.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_1_S139303.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:31:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle --name sample_EvoCDA_791_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_2_S139223.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_2_S139223.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:31:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle --name sample_EvoCDA_791_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_3_S139228.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_3_S139228.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:36:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:36:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:36:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:36:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:37:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:38:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:38:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:38:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:47:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:47:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:47:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_1_S139303.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_1_S139303.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:48:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:49:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:49:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_3_S139228.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_3_S139228.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:50:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:50:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:50:\n",
      "\t Estimating average read length... \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:50:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:50:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:50:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_2_S139223.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_2_S139223.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_presort_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:50:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:50:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_3_S139227.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_3_S139227.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 13 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_2_S139225.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_2_S139225.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Done! \n",
      "\n",
      "Completed in 13 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Estimating average read length... \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_2_S139226.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_2_S139226.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:51:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_1_S139224.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_1_S139224.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:53:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:53:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:53:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:53:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Done! \n",
      "\n",
      "Completed in 16 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_3_S139230.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_3_S139230.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_high_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:54:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:57:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:26:57:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 21 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:00:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:00:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:00:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_1_S139302.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_1_S139302.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_791_low_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:01:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4076 N_CACHED_ALN: 4959 N_COMPUTED_NOTALN: 892 N_CACHED_NOTALN: 73 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:03:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3074 N_CACHED_ALN: 3834 N_COMPUTED_NOTALN: 2892 N_CACHED_NOTALN: 200 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:04:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:04:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:06:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3369 N_CACHED_ALN: 4274 N_COMPUTED_NOTALN: 2179 N_CACHED_NOTALN: 178 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:06:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3693 N_CACHED_ALN: 4150 N_COMPUTED_NOTALN: 1950 N_CACHED_NOTALN: 207 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:06:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3563 N_CACHED_ALN: 4101 N_COMPUTED_NOTALN: 2076 N_CACHED_NOTALN: 260 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:08:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7045 N_CACHED_ALN: 11215 N_COMPUTED_NOTALN: 1571 N_CACHED_NOTALN: 169 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:08:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4055 N_CACHED_ALN: 4667 N_COMPUTED_NOTALN: 1198 N_CACHED_NOTALN: 80 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:10:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2582 N_CACHED_ALN: 3063 N_COMPUTED_NOTALN: 4122 N_CACHED_NOTALN: 233 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:13:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2664 N_CACHED_ALN: 3051 N_COMPUTED_NOTALN: 4062 N_CACHED_NOTALN: 223 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:14:\n",
      "\t Finished reads; N_TOT_READS: 17429 N_COMPUTED_ALN: 5789 N_CACHED_ALN: 6603 N_COMPUTED_NOTALN: 4645 N_CACHED_NOTALN: 392 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:14:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:16:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3649 N_CACHED_ALN: 4730 N_COMPUTED_NOTALN: 1535 N_CACHED_NOTALN: 86 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:16:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:16:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6339 N_CACHED_ALN: 9468 N_COMPUTED_NOTALN: 3709 N_CACHED_NOTALN: 484 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:16:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:16:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6844 N_CACHED_ALN: 10539 N_COMPUTED_NOTALN: 2387 N_CACHED_NOTALN: 230 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:17:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:17:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6022 N_CACHED_ALN: 9337 N_COMPUTED_NOTALN: 3992 N_CACHED_NOTALN: 649 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:17:\n",
      "\t Finished reads; N_TOT_READS: 28215 N_COMPUTED_ALN: 11203 N_CACHED_ALN: 14740 N_COMPUTED_NOTALN: 2044 N_CACHED_NOTALN: 228 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:17:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:19:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6498 N_CACHED_ALN: 9013 N_COMPUTED_NOTALN: 4073 N_CACHED_NOTALN: 416 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:22:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:22:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:22:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:25:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 10211 N_CACHED_ALN: 15982 N_COMPUTED_NOTALN: 3436 N_CACHED_NOTALN: 371 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:26:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6064 N_CACHED_ALN: 10790 N_COMPUTED_NOTALN: 2936 N_CACHED_NOTALN: 210 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:26:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5253 N_CACHED_ALN: 6583 N_COMPUTED_NOTALN: 7642 N_CACHED_NOTALN: 522 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:27:\n",
      "\t Finished reads; N_TOT_READS: 25622 N_COMPUTED_ALN: 8894 N_CACHED_ALN: 11236 N_COMPUTED_NOTALN: 4977 N_CACHED_NOTALN: 515 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:27:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:29:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9767 N_CACHED_ALN: 13603 N_COMPUTED_NOTALN: 5671 N_CACHED_NOTALN: 959 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:29:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 10843 N_CACHED_ALN: 13404 N_COMPUTED_NOTALN: 5100 N_CACHED_NOTALN: 653 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:29:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5314 N_CACHED_ALN: 6528 N_COMPUTED_NOTALN: 7611 N_CACHED_NOTALN: 547 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:30:\n",
      "\t Finished reads; N_TOT_READS: 30777 N_COMPUTED_ALN: 11211 N_CACHED_ALN: 13706 N_COMPUTED_NOTALN: 5191 N_CACHED_NOTALN: 669 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:30:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:30:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:30:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:31:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:33:\n",
      "\t Finished reads; N_TOT_READS: 24016 N_COMPUTED_ALN: 6665 N_CACHED_ALN: 7942 N_COMPUTED_NOTALN: 8805 N_CACHED_NOTALN: 604 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:33:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:34:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 8568 N_CACHED_ALN: 16932 N_COMPUTED_NOTALN: 4165 N_CACHED_NOTALN: 335 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:34:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:34:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:34:\n",
      "\t Finished reads; N_TOT_READS: 34470 N_COMPUTED_ALN: 11737 N_CACHED_ALN: 15286 N_COMPUTED_NOTALN: 6352 N_CACHED_NOTALN: 1095 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:34:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:34:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:35:\n",
      "\t Finished reads; N_TOT_READS: 37595 N_COMPUTED_ALN: 14069 N_CACHED_ALN: 19005 N_COMPUTED_NOTALN: 4102 N_CACHED_NOTALN: 419 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:35:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:35:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:35:\n",
      "\t Finished reads; N_TOT_READS: 23394 N_COMPUTED_ALN: 6553 N_CACHED_ALN: 7552 N_COMPUTED_NOTALN: 8653 N_CACHED_NOTALN: 636 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:35:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:35:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:36:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:38:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:38:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:38:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:39:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:39:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:39:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:40:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:40:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:41:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:43:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 12083 N_CACHED_ALN: 22215 N_COMPUTED_NOTALN: 5274 N_CACHED_NOTALN: 428 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:48:\n",
      "\t Finished reads; N_TOT_READS: 44715 N_COMPUTED_ALN: 14145 N_CACHED_ALN: 24335 N_COMPUTED_NOTALN: 5776 N_CACHED_NOTALN: 459 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:48:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:53:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:53:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:27:54:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:29:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:29:48:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:29:49:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:29:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:29:50:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:29:50:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:29:51:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:29:52:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:04:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:05:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:07:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:08:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:08:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:10:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:12:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:12:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:12:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:12:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:13:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:13:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:14:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:15:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:20:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:20:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:20:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:21:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:21:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:23:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:30:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:30:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:32:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:32:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:32:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:30:33:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:47:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle --name sample_EvoCDA_791_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_1_S139224.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle --name sample_EvoCDA_791_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_2_S139225.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle --name sample_EvoCDA_791_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_3_S139230.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle --name sample_EvoCDA_791_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_1_S139302.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle --name sample_EvoCDA_791_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_2_S139226.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle --name sample_EvoCDA_791_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_3_S139227.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle --name sample_EvoCDA_791_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_1_S139303.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle --name sample_EvoCDA_791_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_2_S139223.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:51:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle --name sample_EvoCDA_791_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_3_S139228.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_791_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_791_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_791_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_791_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_791_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by t\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "he quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_791_high_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_791_presort_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_791_low_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:57:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:58:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg791/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_791_presort_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:58:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:31:58:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:03:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:03:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:05:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:05:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:05:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:05:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:06:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:06:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:06:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:06:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:08:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:08:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:11:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4062 N_CACHED_ALN: 4594 N_COMPUTED_NOTALN: 1180 N_CACHED_NOTALN: 164 \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:12:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2938 N_CACHED_ALN: 2784 N_COMPUTED_NOTALN: 3820 N_CACHED_NOTALN: 458 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:12:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:12:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:14:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3702 N_CACHED_ALN: 3473 N_COMPUTED_NOTALN: 2490 N_CACHED_NOTALN: 335 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:14:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3728 N_CACHED_ALN: 3265 N_COMPUTED_NOTALN: 2680 N_CACHED_NOTALN: 327 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:15:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3726 N_CACHED_ALN: 3539 N_COMPUTED_NOTALN: 2312 N_CACHED_NOTALN: 423 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:15:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4231 N_CACHED_ALN: 3907 N_COMPUTED_NOTALN: 1615 N_CACHED_NOTALN: 247 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:17:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7193 N_CACHED_ALN: 10296 N_COMPUTED_NOTALN: 2146 N_CACHED_NOTALN: 365 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:17:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2742 N_CACHED_ALN: 2294 N_COMPUTED_NOTALN: 4488 N_CACHED_NOTALN: 476 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:18:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2779 N_CACHED_ALN: 2137 N_COMPUTED_NOTALN: 4582 N_CACHED_NOTALN: 502 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:20:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5546 N_CACHED_ALN: 6341 N_COMPUTED_NOTALN: 7007 N_CACHED_NOTALN: 1106 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:20:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3619 N_CACHED_ALN: 3861 N_COMPUTED_NOTALN: 2268 N_CACHED_NOTALN: 252 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:21:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6576 N_CACHED_ALN: 7842 N_COMPUTED_NOTALN: 4761 N_CACHED_NOTALN: 821 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:22:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6582 N_CACHED_ALN: 7543 N_COMPUTED_NOTALN: 5064 N_CACHED_NOTALN: 811 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:22:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7451 N_CACHED_ALN: 8914 N_COMPUTED_NOTALN: 3046 N_CACHED_NOTALN: 589 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:22:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 10705 N_CACHED_ALN: 15825 N_COMPUTED_NOTALN: 2918 N_CACHED_NOTALN: 552 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:22:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6449 N_CACHED_ALN: 8149 N_COMPUTED_NOTALN: 4392 N_CACHED_NOTALN: 1010 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:26:\n",
      "\t Finished reads; N_TOT_READS: 36915 N_COMPUTED_ALN: 13168 N_CACHED_ALN: 19680 N_COMPUTED_NOTALN: 3406 N_CACHED_NOTALN: 661 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:26:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:27:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9493 N_CACHED_ALN: 12552 N_COMPUTED_NOTALN: 6628 N_CACHED_NOTALN: 1327 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:27:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4997 N_CACHED_ALN: 5109 N_COMPUTED_NOTALN: 8769 N_CACHED_NOTALN: 1125 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:28:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6431 N_CACHED_ALN: 8593 N_COMPUTED_NOTALN: 4413 N_CACHED_NOTALN: 563 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:28:\n",
      "\t Finished reads; N_TOT_READS: 29498 N_COMPUTED_ALN: 8276 N_CACHED_ALN: 9888 N_COMPUTED_NOTALN: 9636 N_CACHED_NOTALN: 1698 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:28:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:28:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 10456 N_CACHED_ALN: 14338 N_COMPUTED_NOTALN: 4274 N_CACHED_NOTALN: 932 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:28:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9694 N_CACHED_ALN: 11995 N_COMPUTED_NOTALN: 7089 N_CACHED_NOTALN: 1222 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:28:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9256 N_CACHED_ALN: 12976 N_COMPUTED_NOTALN: 6124 N_CACHED_NOTALN: 1644 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:28:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5139 N_CACHED_ALN: 4829 N_COMPUTED_NOTALN: 8865 N_CACHED_NOTALN: 1167 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:30:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:30:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:30:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:30:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:30:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:31:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:33:\n",
      "\t Finished reads; N_TOT_READS: 36053 N_COMPUTED_ALN: 11680 N_CACHED_ALN: 14669 N_COMPUTED_NOTALN: 8240 N_CACHED_NOTALN: 1464 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:33:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:33:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 13645 N_CACHED_ALN: 19732 N_COMPUTED_NOTALN: 5364 N_CACHED_NOTALN: 1259 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:33:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 12705 N_CACHED_ALN: 17100 N_COMPUTED_NOTALN: 8461 N_CACHED_NOTALN: 1734 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:34:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 8974 N_CACHED_ALN: 13682 N_COMPUTED_NOTALN: 6440 N_CACHED_NOTALN: 904 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:34:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 12356 N_CACHED_ALN: 17626 N_COMPUTED_NOTALN: 7785 N_CACHED_NOTALN: 2233 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:35:\n",
      "\t Finished reads; N_TOT_READS: 42359 N_COMPUTED_ALN: 13539 N_CACHED_ALN: 18140 N_COMPUTED_NOTALN: 8861 N_CACHED_NOTALN: 1819 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:35:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:36:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7320 N_CACHED_ALN: 8502 N_COMPUTED_NOTALN: 12389 N_CACHED_NOTALN: 1789 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:36:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:36:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:37:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:37:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7393 N_CACHED_ALN: 7988 N_COMPUTED_NOTALN: 12719 N_CACHED_NOTALN: 1900 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:38:\n",
      "\t Finished reads; N_TOT_READS: 45043 N_COMPUTED_ALN: 14157 N_CACHED_ALN: 19841 N_COMPUTED_NOTALN: 8568 N_CACHED_NOTALN: 2477 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:38:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:39:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:39:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:40:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:40:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 17295 N_CACHED_ALN: 24696 N_COMPUTED_NOTALN: 6496 N_CACHED_NOTALN: 1513 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:40:\n",
      "\t Finished reads; N_TOT_READS: 50512 N_COMPUTED_ALN: 17451 N_CACHED_ALN: 24988 N_COMPUTED_NOTALN: 6543 N_CACHED_NOTALN: 1530 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:40:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:41:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 11441 N_CACHED_ALN: 19067 N_COMPUTED_NOTALN: 8244 N_CACHED_NOTALN: 1248 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:42:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:42:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:42:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:42:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:43:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:44:\n",
      "\t Finished reads; N_TOT_READS: 37051 N_COMPUTED_ALN: 9170 N_CACHED_ALN: 10186 N_COMPUTED_NOTALN: 15357 N_CACHED_NOTALN: 2338 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:44:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:44:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:44:\n",
      "\t Finished reads; N_TOT_READS: 38812 N_COMPUTED_ALN: 9643 N_CACHED_ALN: 11385 N_COMPUTED_NOTALN: 15460 N_CACHED_NOTALN: 2324 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:44:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:44:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:45:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:46:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:46:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:46:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:46:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 13971 N_CACHED_ALN: 24591 N_COMPUTED_NOTALN: 9866 N_CACHED_NOTALN: 1572 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:47:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:47:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:47:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:48:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:48:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:48:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:53:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 17018 N_CACHED_ALN: 29608 N_COMPUTED_NOTALN: 11483 N_CACHED_NOTALN: 1891 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:32:59:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 20127 N_CACHED_ALN: 34705 N_COMPUTED_NOTALN: 12995 N_CACHED_NOTALN: 2173 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:33:01:\n",
      "\t Finished reads; N_TOT_READS: 72457 N_COMPUTED_ALN: 20778 N_CACHED_ALN: 36095 N_COMPUTED_NOTALN: 13343 N_CACHED_NOTALN: 2241 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:33:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:33:01:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:33:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:33:07:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:33:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:33:07:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:33:08:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:34:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:34:53:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:34:53:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:34:55:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:34:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:34:57:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:34:57:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:00:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:06:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:06:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:07:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:07:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:08:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:09:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:19:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:19:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:20:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:20:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:20:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:20:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:21:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:22:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:22:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:22:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:23:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:24:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:35:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:35:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:37:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:37:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:37:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:35:37:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:42:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_1_S139224.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_2_S139225.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_3_S139230.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_1_S139302.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_2_S139226.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_3_S139227.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_1_S139303.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_2_S139223.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:45:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_3_S139228.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_791_high_1 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GGCAGCTGGGTAGCTAATGA) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_791_low_3 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GGCAGCTGGGTAGCTAATGA) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_791_high_3 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GGCAGCTGGGTAGCTAATGA) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_791_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GGCAGCTGGGTAGCTAATGA) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_791_low_2 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GGCAGCTGGGTAGCTAATGA) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_791_presort_2 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GGCAGCTGGGTAGCTAATGA) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_791_high_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_791_low_1 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GGCAGCTGGGTAGCTAATGA) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GGCAGCTGGGTAGCTAATGA) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t CRISPResso command failed (return value 2) on batch #5: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_3_S139227.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t CRISPResso command failed (return value 2) on batch #0: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_1_S139224.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t CRISPResso command failed (return value 2) on batch #2: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_3_S139230.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t CRISPResso command failed (return value 2) on batch #8: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_3_S139228.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_791_presort_1 \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:36:50:\n",
      "\t CRISPResso command failed (return value 2) on batch #4: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_2_S139226.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:36:51:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (GGCAGCTGGGTAGCTAATGA) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:36:51:\n",
      "\t CRISPResso command failed (return value 2) on batch #3: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_low_1_S139302.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:36:51:\n",
      "\t CRISPResso command failed (return value 2) on batch #7: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_2_S139223.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:36:51:\n",
      "\t CRISPResso command failed (return value 2) on batch #1: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_2_S139225.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 16:36:51:\n",
      "\t CRISPResso command failed (return value 2) on batch #6: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_presort_1_S139303.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "CRISPResso2 failed\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:36:51:\n",
      "\t \n",
      "\n",
      "ERROR: CRISPResso batch #0 failed. For more information, try running the command: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg791_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg791_unmerged_R2_middle --name sample_EvoCDA_791_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_791_high_1_S139224.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq GGCAGCTGGGTAGCTAATGA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "Processing EvoCDA_sg805\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Running CRISPResso with 20 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle --name sample_EvoCDA_805_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_1_S139249.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle --name sample_EvoCDA_805_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_2_S139254.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle --name sample_EvoCDA_805_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_1_S139250.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle --name sample_EvoCDA_805_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_3_S139255.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle --name sample_EvoCDA_805_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_2_S139251.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle --name sample_EvoCDA_805_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_3_S139256.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle --name sample_EvoCDA_805_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_1_S139247.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle --name sample_EvoCDA_805_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_2_S139252.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:54:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle --name sample_EvoCDA_805_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_3_S139253.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle/CRISPResso_on_sample_EvoCDA_805_low_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle/CRISPResso_on_sample_EvoCDA_805_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle/CRISPResso_on_sample_EvoCDA_805_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle/CRISPResso_on_sample_EvoCDA_805_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle/CRISPResso_on_sample_EvoCDA_805_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle/CRISPResso_on_sample_EvoCDA_805_presort_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle/CRISPResso_on_sample_EvoCDA_805_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle/CRISPResso_on_sample_EvoCDA_805_presort_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:36:59:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:00:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:00:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:00:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:00:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:00:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:00:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/premerged/CRISPRessoBatch_on_EvoCDA_sg805_premerged_middle/CRISPResso_on_sample_EvoCDA_805_high_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:00:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:08:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:08:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:08:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:08:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:09:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:09:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:09:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:09:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:11:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:11:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:13:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4682 N_CACHED_ALN: 5169 N_COMPUTED_NOTALN: 117 N_CACHED_NOTALN: 32 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:14:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4163 N_CACHED_ALN: 5187 N_COMPUTED_NOTALN: 506 N_CACHED_NOTALN: 144 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:15:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4782 N_CACHED_ALN: 4337 N_COMPUTED_NOTALN: 658 N_CACHED_NOTALN: 223 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:15:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5014 N_CACHED_ALN: 4720 N_COMPUTED_NOTALN: 203 N_CACHED_NOTALN: 63 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:15:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4609 N_CACHED_ALN: 5078 N_COMPUTED_NOTALN: 225 N_CACHED_NOTALN: 88 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:15:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5202 N_CACHED_ALN: 4324 N_COMPUTED_NOTALN: 335 N_CACHED_NOTALN: 139 \n",
      "\n",
      "Completed in 16 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:16:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:16:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:17:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5860 N_CACHED_ALN: 4043 N_COMPUTED_NOTALN: 81 N_CACHED_NOTALN: 16 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:19:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5419 N_CACHED_ALN: 4241 N_COMPUTED_NOTALN: 282 N_CACHED_NOTALN: 58 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:19:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9458 N_CACHED_ALN: 10259 N_COMPUTED_NOTALN: 228 N_CACHED_NOTALN: 55 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:21:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8326 N_CACHED_ALN: 10418 N_COMPUTED_NOTALN: 990 N_CACHED_NOTALN: 266 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:21:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8546 N_CACHED_ALN: 9746 N_COMPUTED_NOTALN: 1202 N_CACHED_NOTALN: 506 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:21:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9465 N_CACHED_ALN: 10020 N_COMPUTED_NOTALN: 365 N_CACHED_NOTALN: 150 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:22:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9341 N_CACHED_ALN: 10070 N_COMPUTED_NOTALN: 407 N_CACHED_NOTALN: 182 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:22:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10127 N_CACHED_ALN: 8954 N_COMPUTED_NOTALN: 620 N_CACHED_NOTALN: 299 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:23:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4486 N_CACHED_ALN: 5179 N_COMPUTED_NOTALN: 228 N_CACHED_NOTALN: 107 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:23:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10938 N_CACHED_ALN: 8882 N_COMPUTED_NOTALN: 144 N_CACHED_NOTALN: 36 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:25:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9507 N_CACHED_ALN: 9770 N_COMPUTED_NOTALN: 540 N_CACHED_NOTALN: 183 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:26:\n",
      "\t Finished reads; N_TOT_READS: 25608 N_COMPUTED_ALN: 10952 N_CACHED_ALN: 12460 N_COMPUTED_NOTALN: 1547 N_CACHED_NOTALN: 649 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:26:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:27:\n",
      "\t Finished reads; N_TOT_READS: 28967 N_COMPUTED_ALN: 14323 N_CACHED_ALN: 14275 N_COMPUTED_NOTALN: 303 N_CACHED_NOTALN: 66 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:27:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:28:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7974 N_CACHED_ALN: 11383 N_COMPUTED_NOTALN: 404 N_CACHED_NOTALN: 239 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:28:\n",
      "\t Finished reads; N_TOT_READS: 28336 N_COMPUTED_ALN: 12056 N_CACHED_ALN: 14549 N_COMPUTED_NOTALN: 1379 N_CACHED_NOTALN: 352 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:28:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:28:\n",
      "\t Finished reads; N_TOT_READS: 29308 N_COMPUTED_ALN: 13546 N_CACHED_ALN: 15021 N_COMPUTED_NOTALN: 525 N_CACHED_NOTALN: 216 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:28:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:29:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 14530 N_CACHED_ALN: 14601 N_COMPUTED_NOTALN: 605 N_CACHED_NOTALN: 264 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:30:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:30:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:30:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15549 N_CACHED_ALN: 13110 N_COMPUTED_NOTALN: 906 N_CACHED_NOTALN: 435 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:30:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:30:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 16779 N_CACHED_ALN: 12928 N_COMPUTED_NOTALN: 223 N_CACHED_NOTALN: 70 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:31:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 13291 N_CACHED_ALN: 15578 N_COMPUTED_NOTALN: 770 N_CACHED_NOTALN: 361 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:32:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:32:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:32:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:32:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:32:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:33:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:33:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:33:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:33:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:33:\n",
      "\t Finished reads; N_TOT_READS: 32843 N_COMPUTED_ALN: 17257 N_CACHED_ALN: 14144 N_COMPUTED_NOTALN: 977 N_CACHED_NOTALN: 465 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:33:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:34:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 11943 N_CACHED_ALN: 17090 N_COMPUTED_NOTALN: 613 N_CACHED_NOTALN: 354 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:34:\n",
      "\t Finished reads; N_TOT_READS: 34256 N_COMPUTED_ALN: 16844 N_CACHED_ALN: 16429 N_COMPUTED_NOTALN: 696 N_CACHED_NOTALN: 287 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:34:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:37:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 17315 N_CACHED_ALN: 21195 N_COMPUTED_NOTALN: 983 N_CACHED_NOTALN: 507 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Finished reads; N_TOT_READS: 39930 N_COMPUTED_ALN: 22641 N_CACHED_ALN: 16901 N_COMPUTED_NOTALN: 297 N_CACHED_NOTALN: 91 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 16095 N_CACHED_ALN: 22665 N_COMPUTED_NOTALN: 794 N_CACHED_NOTALN: 446 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:40:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:41:\n",
      "\t Finished reads; N_TOT_READS: 44922 N_COMPUTED_ALN: 19769 N_CACHED_ALN: 23515 N_COMPUTED_NOTALN: 1082 N_CACHED_NOTALN: 556 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:41:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:46:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 20027 N_CACHED_ALN: 28447 N_COMPUTED_NOTALN: 975 N_CACHED_NOTALN: 551 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:48:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:48:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:48:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:48:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:48:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:49:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:53:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 24077 N_CACHED_ALN: 34082 N_COMPUTED_NOTALN: 1176 N_CACHED_NOTALN: 665 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:54:\n",
      "\t Finished reads; N_TOT_READS: 62264 N_COMPUTED_ALN: 25112 N_CACHED_ALN: 35253 N_COMPUTED_NOTALN: 1213 N_CACHED_NOTALN: 686 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:37:54:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:38:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:38:03:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:38:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:38:04:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:38:04:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:34:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:34:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:36:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:40:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:40:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:40:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:40:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:41:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:42:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:43:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:43:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:44:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:46:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:49:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:49:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:51:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:54:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:54:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:55:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:58:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:58:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:58:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:58:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:39:59:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:40:00:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:40:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:40:10:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:40:10:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:40:11:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:40:11:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:40:12:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:40:12:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:28:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle --name sample_EvoCDA_805_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_1_S139249.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_1_S139249.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle --name sample_EvoCDA_805_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_2_S139254.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_2_S139254.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle --name sample_EvoCDA_805_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_3_S139255.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_3_S139255.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle --name sample_EvoCDA_805_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_1_S139250.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_1_S139250.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle --name sample_EvoCDA_805_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_3_S139256.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_3_S139256.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle --name sample_EvoCDA_805_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_2_S139251.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_2_S139251.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle --name sample_EvoCDA_805_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_1_S139247.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_1_S139247.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle --name sample_EvoCDA_805_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_2_S139252.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_2_S139252.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:32:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle --name sample_EvoCDA_805_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_3_S139253.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_3_S139253.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:38:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:40:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:40:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:49:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:49:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:49:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_1_S139249.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_1_S139249.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:51:\n",
      "\t Done! \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:52:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:52:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:52:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:52:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:52:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_3_S139255.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_3_S139255.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:53:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:53:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:53:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_1_S139250.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_1_S139250.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:54:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:54:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:55:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:55:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:55:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_2_S139251.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_2_S139251.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 15 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:55:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:56:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:56:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_2_S139252.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_2_S139252.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:56:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:56:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:57:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:57:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:58:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:58:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 19 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:59:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:59:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:59:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_3_S139256.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_3_S139256.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_low_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:41:59:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2951 N_CACHED_ALN: 5694 N_COMPUTED_NOTALN: 1301 N_CACHED_NOTALN: 54 \n",
      "\n",
      "Completed in 20 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:00:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:00:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:00:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_1_S139247.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_1_S139247.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 22 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:02:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:02:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:02:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_2_S139254.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_2_S139254.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_high_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:03:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:03:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:05:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2915 N_CACHED_ALN: 5135 N_COMPUTED_NOTALN: 1830 N_CACHED_NOTALN: 120 \n",
      "\n",
      "Completed in 25 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:05:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:05:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:05:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_3_S139253.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_3_S139253.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg805/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_805_presort_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:06:\n",
      "\t Finished reads; N_TOT_READS: 19764 N_COMPUTED_ALN: 5902 N_CACHED_ALN: 11499 N_COMPUTED_NOTALN: 2252 N_CACHED_NOTALN: 111 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:06:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:06:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:06:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:08:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3495 N_CACHED_ALN: 4424 N_COMPUTED_NOTALN: 2007 N_CACHED_NOTALN: 74 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:09:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:09:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:09:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:09:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:09:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:10:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2820 N_CACHED_ALN: 3691 N_COMPUTED_NOTALN: 3292 N_CACHED_NOTALN: 197 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:10:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2574 N_CACHED_ALN: 4508 N_COMPUTED_NOTALN: 2789 N_CACHED_NOTALN: 129 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:14:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5387 N_CACHED_ALN: 10849 N_COMPUTED_NOTALN: 3507 N_CACHED_NOTALN: 257 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:16:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2897 N_CACHED_ALN: 5078 N_COMPUTED_NOTALN: 1885 N_CACHED_NOTALN: 140 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:17:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6095 N_CACHED_ALN: 9962 N_COMPUTED_NOTALN: 3795 N_CACHED_NOTALN: 148 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:19:\n",
      "\t Finished reads; N_TOT_READS: 24771 N_COMPUTED_ALN: 7204 N_CACHED_ALN: 13140 N_COMPUTED_NOTALN: 4125 N_CACHED_NOTALN: 302 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:19:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:20:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1789 N_CACHED_ALN: 3286 N_COMPUTED_NOTALN: 4774 N_CACHED_NOTALN: 151 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:20:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4479 N_CACHED_ALN: 9787 N_COMPUTED_NOTALN: 5424 N_CACHED_NOTALN: 310 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:21:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1435 N_CACHED_ALN: 3103 N_COMPUTED_NOTALN: 5180 N_CACHED_NOTALN: 282 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:22:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4879 N_CACHED_ALN: 8418 N_COMPUTED_NOTALN: 6176 N_CACHED_NOTALN: 527 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:22:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:22:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:22:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:23:\n",
      "\t Finished reads; N_TOT_READS: 26160 N_COMPUTED_ALN: 8676 N_CACHED_ALN: 12692 N_COMPUTED_NOTALN: 4612 N_CACHED_NOTALN: 180 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:23:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:23:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:24:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2051 N_CACHED_ALN: 3687 N_COMPUTED_NOTALN: 4118 N_CACHED_NOTALN: 144 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:25:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4696 N_CACHED_ALN: 11206 N_COMPUTED_NOTALN: 3732 N_CACHED_NOTALN: 366 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:26:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:26:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:27:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:27:\n",
      "\t Finished reads; N_TOT_READS: 26580 N_COMPUTED_ALN: 6538 N_CACHED_ALN: 12938 N_COMPUTED_NOTALN: 6690 N_CACHED_NOTALN: 414 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:27:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:29:\n",
      "\t Finished reads; N_TOT_READS: 26063 N_COMPUTED_ALN: 6983 N_CACHED_ALN: 10895 N_COMPUTED_NOTALN: 7533 N_CACHED_NOTALN: 652 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:29:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:29:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:29:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:29:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:31:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:31:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:32:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:32:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 6868 N_CACHED_ALN: 17430 N_COMPUTED_NOTALN: 5141 N_CACHED_NOTALN: 561 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:35:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3243 N_CACHED_ALN: 7299 N_COMPUTED_NOTALN: 9056 N_CACHED_NOTALN: 402 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:38:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4022 N_CACHED_ALN: 8057 N_COMPUTED_NOTALN: 7572 N_CACHED_NOTALN: 349 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:38:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2511 N_CACHED_ALN: 6465 N_COMPUTED_NOTALN: 10301 N_CACHED_NOTALN: 723 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:39:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 8591 N_CACHED_ALN: 23950 N_COMPUTED_NOTALN: 6682 N_CACHED_NOTALN: 777 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:47:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 11533 N_CACHED_ALN: 29536 N_COMPUTED_NOTALN: 7982 N_CACHED_NOTALN: 949 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:49:\n",
      "\t Finished reads; N_TOT_READS: 51398 N_COMPUTED_ALN: 11961 N_CACHED_ALN: 30313 N_COMPUTED_NOTALN: 8159 N_CACHED_NOTALN: 965 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:49:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:49:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5384 N_CACHED_ALN: 11239 N_COMPUTED_NOTALN: 12767 N_CACHED_NOTALN: 610 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:51:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5525 N_CACHED_ALN: 12722 N_COMPUTED_NOTALN: 11126 N_CACHED_NOTALN: 627 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:53:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:53:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:53:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:54:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 4127 N_CACHED_ALN: 9826 N_COMPUTED_NOTALN: 14979 N_CACHED_NOTALN: 1068 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:55:\n",
      "\t Finished reads; N_TOT_READS: 33906 N_COMPUTED_ALN: 6275 N_CACHED_ALN: 12961 N_COMPUTED_NOTALN: 13983 N_CACHED_NOTALN: 687 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:55:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:57:\n",
      "\t Finished reads; N_TOT_READS: 31531 N_COMPUTED_ALN: 4430 N_CACHED_ALN: 10413 N_COMPUTED_NOTALN: 15572 N_CACHED_NOTALN: 1116 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:57:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:57:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:57:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:58:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:58:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:58:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:42:59:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:43:04:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 7244 N_CACHED_ALN: 17277 N_COMPUTED_NOTALN: 14601 N_CACHED_NOTALN: 878 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:43:11:\n",
      "\t Finished reads; N_TOT_READS: 45889 N_COMPUTED_ALN: 8959 N_CACHED_ALN: 19917 N_COMPUTED_NOTALN: 16050 N_CACHED_NOTALN: 963 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:43:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:43:11:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:43:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:43:14:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:43:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:43:14:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:43:14:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:16:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:16:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:16:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:41:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:41:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:43:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:43:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:43:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:43:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:43:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:43:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:43:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:44:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:44:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:48:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:48:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:44:49:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:06:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:07:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:09:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:09:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:09:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:10:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:10:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:10:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:11:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:18:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:18:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:20:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:20:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:20:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:45:21:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:36:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 16:46:39:\n",
      "\t \n",
      "\n",
      "ERROR: Offset around cut would be greater than reference sequence length. Please decrease plot_window_size parameter. Cut point: 245 window: 20 reference: 250 \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle --name sample_EvoCDA_805_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_1_S139249.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle --name sample_EvoCDA_805_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_2_S139254.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle --name sample_EvoCDA_805_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_high_3_S139255.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle --name sample_EvoCDA_805_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_1_S139250.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle --name sample_EvoCDA_805_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_2_S139251.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle --name sample_EvoCDA_805_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_low_3_S139256.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle --name sample_EvoCDA_805_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_1_S139247.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle --name sample_EvoCDA_805_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_2_S139252.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:42:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle --name sample_EvoCDA_805_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_805_presort_3_S139253.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CAGCCTGGTGAGATGGGCCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:46:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_805_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:46:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_805_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:46:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_805_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_805_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_805_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_805_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_805_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_805_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg805_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg805_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_805_high_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:48:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:48:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:48:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 4 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:51:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:51:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:53:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:53:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:53:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:53:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:54:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:54:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:54:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:54:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:57:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:57:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:57:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:57:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:58:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:58:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:58:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5070 N_CACHED_ALN: 3525 N_COMPUTED_NOTALN: 1364 N_CACHED_NOTALN: 41 \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:59:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:46:59:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:01:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4283 N_CACHED_ALN: 3543 N_COMPUTED_NOTALN: 2066 N_CACHED_NOTALN: 108 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:03:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5953 N_CACHED_ALN: 1990 N_COMPUTED_NOTALN: 2003 N_CACHED_NOTALN: 54 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:04:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4595 N_CACHED_ALN: 1790 N_COMPUTED_NOTALN: 3435 N_CACHED_NOTALN: 180 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:04:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4434 N_CACHED_ALN: 2435 N_COMPUTED_NOTALN: 3016 N_CACHED_NOTALN: 115 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:05:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4428 N_CACHED_ALN: 3421 N_COMPUTED_NOTALN: 2043 N_CACHED_NOTALN: 108 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:08:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 12045 N_CACHED_ALN: 5521 N_COMPUTED_NOTALN: 2355 N_CACHED_NOTALN: 79 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:08:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2560 N_CACHED_ALN: 2386 N_COMPUTED_NOTALN: 4848 N_CACHED_NOTALN: 206 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:08:\n",
      "\t Finished reads; N_TOT_READS: 20080 N_COMPUTED_ALN: 12101 N_CACHED_ALN: 5540 N_COMPUTED_NOTALN: 2360 N_CACHED_NOTALN: 79 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:08:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:09:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2262 N_CACHED_ALN: 1930 N_COMPUTED_NOTALN: 5515 N_CACHED_NOTALN: 293 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:09:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9866 N_CACHED_ALN: 6282 N_COMPUTED_NOTALN: 3639 N_CACHED_NOTALN: 213 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:10:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3196 N_CACHED_ALN: 2479 N_COMPUTED_NOTALN: 4207 N_CACHED_NOTALN: 118 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:11:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:11:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:12:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 11928 N_CACHED_ALN: 4111 N_COMPUTED_NOTALN: 3844 N_CACHED_NOTALN: 117 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:12:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:12:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8131 N_CACHED_ALN: 7397 N_COMPUTED_NOTALN: 4153 N_CACHED_NOTALN: 319 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:13:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9229 N_CACHED_ALN: 4834 N_COMPUTED_NOTALN: 5678 N_CACHED_NOTALN: 259 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:14:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9444 N_CACHED_ALN: 3675 N_COMPUTED_NOTALN: 6458 N_CACHED_NOTALN: 423 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:14:\n",
      "\t Finished reads; N_TOT_READS: 25415 N_COMPUTED_ALN: 13118 N_CACHED_ALN: 7633 N_COMPUTED_NOTALN: 4429 N_CACHED_NOTALN: 235 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:14:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:18:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4712 N_CACHED_ALN: 5293 N_COMPUTED_NOTALN: 9497 N_CACHED_NOTALN: 498 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:18:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:18:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:19:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:19:\n",
      "\t Finished reads; N_TOT_READS: 26517 N_COMPUTED_ALN: 16973 N_CACHED_ALN: 4702 N_COMPUTED_NOTALN: 4696 N_CACHED_NOTALN: 146 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:19:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:19:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 12166 N_CACHED_ALN: 11368 N_COMPUTED_NOTALN: 5933 N_CACHED_NOTALN: 533 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:19:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5979 N_CACHED_ALN: 5335 N_COMPUTED_NOTALN: 8351 N_CACHED_NOTALN: 335 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:20:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4156 N_CACHED_ALN: 4346 N_COMPUTED_NOTALN: 10728 N_CACHED_NOTALN: 770 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:22:\n",
      "\t Finished reads; N_TOT_READS: 26653 N_COMPUTED_ALN: 13797 N_CACHED_ALN: 4272 N_COMPUTED_NOTALN: 8095 N_CACHED_NOTALN: 489 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:22:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:22:\n",
      "\t Finished reads; N_TOT_READS: 27246 N_COMPUTED_ALN: 14024 N_CACHED_ALN: 5754 N_COMPUTED_NOTALN: 7162 N_CACHED_NOTALN: 306 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:22:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:24:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:24:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:24:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:25:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:26:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:26:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:26:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:26:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:26:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:29:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9105 N_CACHED_ALN: 8253 N_COMPUTED_NOTALN: 12106 N_CACHED_NOTALN: 536 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:29:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 18574 N_CACHED_ALN: 13433 N_COMPUTED_NOTALN: 7382 N_CACHED_NOTALN: 611 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:30:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9565 N_CACHED_ALN: 6848 N_COMPUTED_NOTALN: 12971 N_CACHED_NOTALN: 616 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:32:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 8210 N_CACHED_ALN: 5444 N_COMPUTED_NOTALN: 15374 N_CACHED_NOTALN: 972 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:36:\n",
      "\t Finished reads; N_TOT_READS: 35215 N_COMPUTED_ALN: 11993 N_CACHED_ALN: 7845 N_COMPUTED_NOTALN: 14677 N_CACHED_NOTALN: 700 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:36:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:36:\n",
      "\t Finished reads; N_TOT_READS: 32949 N_COMPUTED_ALN: 9446 N_CACHED_ALN: 5835 N_COMPUTED_NOTALN: 16613 N_CACHED_NOTALN: 1055 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:36:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:38:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 24524 N_CACHED_ALN: 15999 N_COMPUTED_NOTALN: 8760 N_CACHED_NOTALN: 717 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:39:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:39:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:39:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:39:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:39:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:40:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:40:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 14466 N_CACHED_ALN: 9829 N_COMPUTED_NOTALN: 15078 N_CACHED_NOTALN: 627 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:40:\n",
      "\t Finished reads; N_TOT_READS: 52669 N_COMPUTED_ALN: 25962 N_CACHED_ALN: 16812 N_COMPUTED_NOTALN: 9154 N_CACHED_NOTALN: 741 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:40:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:48:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:48:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:48:\n",
      "\t Finished reads; N_TOT_READS: 47158 N_COMPUTED_ALN: 18239 N_CACHED_ALN: 11242 N_COMPUTED_NOTALN: 17000 N_CACHED_NOTALN: 677 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:48:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:48:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:53:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:53:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:47:54:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:16:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:16:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:17:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:37:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:37:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:37:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:38:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:39:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:40:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:41:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:55:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:55:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:56:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:58:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:58:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:58:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:58:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:49:58:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:00:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:02:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:02:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:03:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:11:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:11:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:12:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:36:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:36:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:39:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:39:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:39:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:50:40:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:42:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "Processing EvoCDA_sg806\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:47:\n",
      "\t Running CRISPResso with 20 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:47:\n",
      "\t Running CRISPResso on batch #0/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle --name sample_EvoCDA_806_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_1_S139260.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:47:\n",
      "\t Running CRISPResso on batch #1/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle --name sample_EvoCDA_806_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_2_S139261.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:47:\n",
      "\t Running CRISPResso on batch #2/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle --name sample_EvoCDA_806_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_1_S139257.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:47:\n",
      "\t Running CRISPResso on batch #3/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle --name sample_EvoCDA_806_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_2_S139262.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:47:\n",
      "\t Running CRISPResso on batch #4/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle --name sample_EvoCDA_806_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_3_S139263.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:47:\n",
      "\t Running CRISPResso on batch #5/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle --name sample_EvoCDA_806_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_1_S139258.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:47:\n",
      "\t Running CRISPResso on batch #6/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle --name sample_EvoCDA_806_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_2_S139259.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:47:\n",
      "\t Running CRISPResso on batch #7/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle --name sample_EvoCDA_806_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_3_S139264.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:51:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle/CRISPResso_on_sample_EvoCDA_806_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle/CRISPResso_on_sample_EvoCDA_806_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle/CRISPResso_on_sample_EvoCDA_806_high_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle/CRISPResso_on_sample_EvoCDA_806_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle/CRISPResso_on_sample_EvoCDA_806_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle/CRISPResso_on_sample_EvoCDA_806_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle/CRISPResso_on_sample_EvoCDA_806_presort_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/premerged/CRISPRessoBatch_on_EvoCDA_sg806_premerged_middle/CRISPResso_on_sample_EvoCDA_806_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:52:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:53:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:53:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:58:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:58:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:58:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:51:58:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:04:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:04:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:05:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4100 N_CACHED_ALN: 5207 N_COMPUTED_NOTALN: 546 N_CACHED_NOTALN: 147 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:07:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5982 N_CACHED_ALN: 3826 N_COMPUTED_NOTALN: 165 N_CACHED_NOTALN: 27 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:10:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5534 N_CACHED_ALN: 4221 N_COMPUTED_NOTALN: 197 N_CACHED_NOTALN: 48 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:10:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6039 N_CACHED_ALN: 3705 N_COMPUTED_NOTALN: 187 N_CACHED_NOTALN: 69 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:10:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6642 N_CACHED_ALN: 3266 N_COMPUTED_NOTALN: 77 N_CACHED_NOTALN: 15 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:11:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6302 N_CACHED_ALN: 3338 N_COMPUTED_NOTALN: 283 N_CACHED_NOTALN: 77 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:11:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6307 N_CACHED_ALN: 3334 N_COMPUTED_NOTALN: 269 N_CACHED_NOTALN: 90 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:12:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5213 N_CACHED_ALN: 4400 N_COMPUTED_NOTALN: 258 N_CACHED_NOTALN: 129 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:12:\n",
      "\t Finished reads; N_TOT_READS: 18952 N_COMPUTED_ALN: 7862 N_CACHED_ALN: 9787 N_COMPUTED_NOTALN: 1007 N_CACHED_NOTALN: 296 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:12:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:12:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:15:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:15:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:15:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 12145 N_CACHED_ALN: 7520 N_COMPUTED_NOTALN: 281 N_CACHED_NOTALN: 54 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:16:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:17:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10748 N_CACHED_ALN: 8744 N_COMPUTED_NOTALN: 372 N_CACHED_NOTALN: 136 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:17:\n",
      "\t Finished reads; N_TOT_READS: 21574 N_COMPUTED_ALN: 13121 N_CACHED_ALN: 8100 N_COMPUTED_NOTALN: 299 N_CACHED_NOTALN: 54 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:17:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:18:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 11325 N_CACHED_ALN: 8175 N_COMPUTED_NOTALN: 345 N_CACHED_NOTALN: 155 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:18:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 12236 N_CACHED_ALN: 7555 N_COMPUTED_NOTALN: 161 N_CACHED_NOTALN: 48 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:18:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 11711 N_CACHED_ALN: 7570 N_COMPUTED_NOTALN: 519 N_CACHED_NOTALN: 200 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:19:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 11907 N_CACHED_ALN: 7427 N_COMPUTED_NOTALN: 469 N_CACHED_NOTALN: 197 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:20:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10149 N_CACHED_ALN: 9047 N_COMPUTED_NOTALN: 500 N_CACHED_NOTALN: 304 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:22:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:22:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:23:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:25:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 16406 N_CACHED_ALN: 12868 N_COMPUTED_NOTALN: 547 N_CACHED_NOTALN: 179 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:26:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 17065 N_CACHED_ALN: 12214 N_COMPUTED_NOTALN: 507 N_CACHED_NOTALN: 214 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:26:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 17629 N_CACHED_ALN: 12042 N_COMPUTED_NOTALN: 246 N_CACHED_NOTALN: 83 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:26:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 17144 N_CACHED_ALN: 11785 N_COMPUTED_NOTALN: 761 N_CACHED_NOTALN: 310 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:27:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15193 N_CACHED_ALN: 13645 N_COMPUTED_NOTALN: 723 N_CACHED_NOTALN: 439 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:28:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 17609 N_CACHED_ALN: 11373 N_COMPUTED_NOTALN: 727 N_CACHED_NOTALN: 291 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:31:\n",
      "\t Finished reads; N_TOT_READS: 35411 N_COMPUTED_ALN: 20690 N_CACHED_ALN: 14338 N_COMPUTED_NOTALN: 284 N_CACHED_NOTALN: 99 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:31:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:32:\n",
      "\t Finished reads; N_TOT_READS: 37747 N_COMPUTED_ALN: 20564 N_CACHED_ALN: 16273 N_COMPUTED_NOTALN: 690 N_CACHED_NOTALN: 220 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:32:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:33:\n",
      "\t Finished reads; N_TOT_READS: 36496 N_COMPUTED_ALN: 21033 N_CACHED_ALN: 14239 N_COMPUTED_NOTALN: 858 N_CACHED_NOTALN: 366 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:33:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:33:\n",
      "\t Finished reads; N_TOT_READS: 37078 N_COMPUTED_ALN: 21584 N_CACHED_ALN: 14600 N_COMPUTED_NOTALN: 644 N_CACHED_NOTALN: 250 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:33:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:34:\n",
      "\t Finished reads; N_TOT_READS: 38153 N_COMPUTED_ALN: 21445 N_CACHED_ALN: 15364 N_COMPUTED_NOTALN: 941 N_CACHED_NOTALN: 403 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:34:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:37:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 20360 N_CACHED_ALN: 18097 N_COMPUTED_NOTALN: 976 N_CACHED_NOTALN: 567 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:38:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:39:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:39:\n",
      "\t Finished reads; N_TOT_READS: 42319 N_COMPUTED_ALN: 21693 N_CACHED_ALN: 18992 N_COMPUTED_NOTALN: 1038 N_CACHED_NOTALN: 596 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:39:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:39:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:40:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:40:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:40:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:41:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:42:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:42:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:47:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:47:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:52:47:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:21:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:21:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:22:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:28:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:28:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:29:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:49:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:49:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:49:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:49:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:50:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:51:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:51:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:51:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:51:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:51:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:51:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:51:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:51:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:51:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:52:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:52:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:53:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:58:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:54:58:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:55:00:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:55:00:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:55:00:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:55:01:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:12:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:16:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:16:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:16:\n",
      "\t Running CRISPResso on batch #0/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle --name sample_EvoCDA_806_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_1_S139260.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_1_S139260.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:16:\n",
      "\t Running CRISPResso on batch #1/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle --name sample_EvoCDA_806_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_2_S139261.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_2_S139261.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:16:\n",
      "\t Running CRISPResso on batch #2/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle --name sample_EvoCDA_806_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_1_S139257.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_1_S139257.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:16:\n",
      "\t Running CRISPResso on batch #3/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle --name sample_EvoCDA_806_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_2_S139262.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_2_S139262.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:16:\n",
      "\t Running CRISPResso on batch #4/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle --name sample_EvoCDA_806_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_3_S139263.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_3_S139263.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:16:\r\n",
      "\t Running CRISPResso on batch #5/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle --name sample_EvoCDA_806_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_1_S139258.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_1_S139258.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:16:\r\n",
      "\t Running CRISPResso on batch #6/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle --name sample_EvoCDA_806_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_2_S139259.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_2_S139259.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:16:\r\n",
      "\t Running CRISPResso on batch #7/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle --name sample_EvoCDA_806_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_3_S139264.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_3_S139264.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:20:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:21:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:21:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:21:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:21:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:21:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:21:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:21:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:21:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:21:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:22:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:22:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:22:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:22:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:22:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:22:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:34:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:34:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:34:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_3_S139264.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_3_S139264.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:35:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:35:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:36:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:36:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:37:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:37:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_3_S139263.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_3_S139263.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:37:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:37:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_2_S139262.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_2_S139262.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 16 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:37:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:37:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:37:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_1_S139260.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_1_S139260.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_high_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_high_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 15 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:37:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:38:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:38:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_1_S139258.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_1_S139258.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:39:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:39:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:39:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:39:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:40:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:40:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:40:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:40:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 18 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:41:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:41:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:41:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_2_S139259.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_2_S139259.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_presort_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 19 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:42:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:42:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:42:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_2_S139261.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_2_S139261.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_high_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_high_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:43:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:44:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:44:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:45:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:45:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:45:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 23 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:45:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:45:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:45:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_1_S139257.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_1_S139257.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_806_low_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:47:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3860 N_CACHED_ALN: 3864 N_COMPUTED_NOTALN: 2222 N_CACHED_NOTALN: 54 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:49:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4176 N_CACHED_ALN: 4187 N_COMPUTED_NOTALN: 1582 N_CACHED_NOTALN: 55 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:49:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:49:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:50:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4036 N_CACHED_ALN: 3829 N_COMPUTED_NOTALN: 1999 N_CACHED_NOTALN: 136 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:53:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4035 N_CACHED_ALN: 3412 N_COMPUTED_NOTALN: 2472 N_CACHED_NOTALN: 81 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:57:\n",
      "\t Finished reads; N_TOT_READS: 18302 N_COMPUTED_ALN: 7611 N_CACHED_ALN: 6949 N_COMPUTED_NOTALN: 3639 N_CACHED_NOTALN: 103 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:57:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:57:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7272 N_CACHED_ALN: 9676 N_COMPUTED_NOTALN: 2908 N_CACHED_NOTALN: 144 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:57:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3627 N_CACHED_ALN: 3596 N_COMPUTED_NOTALN: 2685 N_CACHED_NOTALN: 92 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:59:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3509 N_CACHED_ALN: 3401 N_COMPUTED_NOTALN: 2932 N_CACHED_NOTALN: 158 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:56:59:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1486 N_CACHED_ALN: 3307 N_COMPUTED_NOTALN: 5021 N_CACHED_NOTALN: 186 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:00:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6688 N_CACHED_ALN: 9087 N_COMPUTED_NOTALN: 3879 N_CACHED_NOTALN: 346 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:00:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:00:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:00:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:03:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6678 N_CACHED_ALN: 8389 N_COMPUTED_NOTALN: 4693 N_CACHED_NOTALN: 240 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:04:\n",
      "\t Finished reads; N_TOT_READS: 26372 N_COMPUTED_ALN: 10806 N_CACHED_ALN: 11807 N_COMPUTED_NOTALN: 3578 N_CACHED_NOTALN: 181 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:04:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:05:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2974 N_CACHED_ALN: 3144 N_COMPUTED_NOTALN: 3686 N_CACHED_NOTALN: 196 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:08:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:08:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:08:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:09:\n",
      "\t Finished reads; N_TOT_READS: 27667 N_COMPUTED_ALN: 10581 N_CACHED_ALN: 11597 N_COMPUTED_NOTALN: 5032 N_CACHED_NOTALN: 457 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:09:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:09:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5898 N_CACHED_ALN: 8581 N_COMPUTED_NOTALN: 5262 N_CACHED_NOTALN: 259 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:12:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5967 N_CACHED_ALN: 7659 N_COMPUTED_NOTALN: 5957 N_CACHED_NOTALN: 417 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:13:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:13:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:13:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:14:\n",
      "\t Finished reads; N_TOT_READS: 29754 N_COMPUTED_ALN: 11244 N_CACHED_ALN: 11955 N_COMPUTED_NOTALN: 6231 N_CACHED_NOTALN: 324 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:14:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:17:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2969 N_CACHED_ALN: 6703 N_COMPUTED_NOTALN: 9898 N_CACHED_NOTALN: 430 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:18:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5097 N_CACHED_ALN: 7401 N_COMPUTED_NOTALN: 7021 N_CACHED_NOTALN: 481 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:18:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:18:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:19:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:21:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9626 N_CACHED_ALN: 12703 N_COMPUTED_NOTALN: 7273 N_CACHED_NOTALN: 398 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:25:\n",
      "\t Finished reads; N_TOT_READS: 32800 N_COMPUTED_ALN: 10884 N_CACHED_ALN: 13717 N_COMPUTED_NOTALN: 7765 N_CACHED_NOTALN: 434 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:25:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:25:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9367 N_CACHED_ALN: 11669 N_COMPUTED_NOTALN: 8334 N_CACHED_NOTALN: 630 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:25:\n",
      "\t Finished reads; N_TOT_READS: 25215 N_COMPUTED_ALN: 4026 N_CACHED_ALN: 8765 N_COMPUTED_NOTALN: 11892 N_CACHED_NOTALN: 532 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:25:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:27:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:27:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:27:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:29:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:29:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:29:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:30:\n",
      "\t Finished reads; N_TOT_READS: 33169 N_COMPUTED_ALN: 10766 N_CACHED_ALN: 12771 N_COMPUTED_NOTALN: 8951 N_CACHED_NOTALN: 681 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:30:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:30:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7170 N_CACHED_ALN: 11792 N_COMPUTED_NOTALN: 10264 N_CACHED_NOTALN: 774 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:34:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:34:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:35:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:44:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 10481 N_CACHED_ALN: 15320 N_COMPUTED_NOTALN: 13200 N_CACHED_NOTALN: 999 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:48:\n",
      "\t Finished reads; N_TOT_READS: 42700 N_COMPUTED_ALN: 11471 N_CACHED_ALN: 16291 N_COMPUTED_NOTALN: 13870 N_CACHED_NOTALN: 1068 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:48:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:52:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:52:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:52:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:52:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:57:52:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:00:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:01:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:01:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:11:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:12:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:13:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:19:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:19:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:20:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:22:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:22:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:24:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:24:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:24:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:25:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:33:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:34:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:36:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:46:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:46:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:46:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:47:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:54:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:54:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:56:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:56:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:56:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 16:59:57:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:02:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:05:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:05:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:05:\n",
      "\t Running CRISPResso on batch #0/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_1_S139260.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:05:\n",
      "\t Running CRISPResso on batch #1/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_2_S139261.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:05:\n",
      "\t Running CRISPResso on batch #2/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_1_S139257.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:05:\n",
      "\t Running CRISPResso on batch #3/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_2_S139262.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:05:\r\n",
      "\t Running CRISPResso on batch #4/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_3_S139263.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:05:\r\n",
      "\t Running CRISPResso on batch #5/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_1_S139258.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:05:\r\n",
      "\t Running CRISPResso on batch #6/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_2_S139259.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:05:\r\n",
      "\t Running CRISPResso on batch #7/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_3_S139264.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:09:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_806_presort_3 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:01:09:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (CACTGAGAAAGTGGGTCTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_806_low_3 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (CACTGAGAAAGTGGGTCTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_806_high_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_806_low_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_806_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_806_high_2 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (CACTGAGAAAGTGGGTCTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (CACTGAGAAAGTGGGTCTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_806_presort_2 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (CACTGAGAAAGTGGGTCTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (CACTGAGAAAGTGGGTCTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (CACTGAGAAAGTGGGTCTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t CRISPResso command failed (return value 2) on batch #7: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_3_S139264.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_806_presort_1 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (CACTGAGAAAGTGGGTCTCT) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t CRISPResso command failed (return value 2) on batch #2: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_1_S139257.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t CRISPResso command failed (return value 2) on batch #1: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_2_S139261.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t CRISPResso command failed (return value 2) on batch #4: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_3_S139263.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t CRISPResso command failed (return value 2) on batch #0: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_1_S139260.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t CRISPResso command failed (return value 2) on batch #6: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_2_S139259.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t CRISPResso command failed (return value 2) on batch #3: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_2_S139262.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t CRISPResso command failed (return value 2) on batch #5: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_1_S139258.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "CRISPResso2 failed\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:01:10:\n",
      "\t \n",
      "\n",
      "ERROR: CRISPResso batch #0 failed. For more information, try running the command: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg806/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R1_middle --name sample_EvoCDA_806_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_1_S139260.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:13:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:14:\n",
      "\t Running CRISPResso on batch #0/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle --name sample_EvoCDA_806_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_1_S139260.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:14:\n",
      "\t Running CRISPResso on batch #1/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle --name sample_EvoCDA_806_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_high_2_S139261.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:14:\n",
      "\t Running CRISPResso on batch #2/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle --name sample_EvoCDA_806_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_1_S139257.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:14:\n",
      "\t Running CRISPResso on batch #3/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle --name sample_EvoCDA_806_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_2_S139262.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:14:\n",
      "\t Running CRISPResso on batch #4/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle --name sample_EvoCDA_806_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_low_3_S139263.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:14:\n",
      "\t Running CRISPResso on batch #5/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle --name sample_EvoCDA_806_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_1_S139258.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:14:\n",
      "\t Running CRISPResso on batch #6/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle --name sample_EvoCDA_806_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_2_S139259.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:14:\n",
      "\t Running CRISPResso on batch #7/8: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle --name sample_EvoCDA_806_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_806_presort_3_S139264.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq CACTGAGAAAGTGGGTCTCT --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_806_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_806_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_806_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_806_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_806_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_806_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_806_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:18:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg806_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg806_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_806_presort_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:19:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:25:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:25:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:25:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:25:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:26:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:26:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:26:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:26:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:26:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:26:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:27:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:27:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:28:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:28:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:29:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:29:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:34:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5385 N_CACHED_ALN: 2053 N_COMPUTED_NOTALN: 2501 N_CACHED_NOTALN: 61 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:34:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6358 N_CACHED_ALN: 1835 N_COMPUTED_NOTALN: 1741 N_CACHED_NOTALN: 66 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:35:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6123 N_CACHED_ALN: 1427 N_COMPUTED_NOTALN: 2312 N_CACHED_NOTALN: 138 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:36:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5566 N_CACHED_ALN: 1650 N_COMPUTED_NOTALN: 2710 N_CACHED_NOTALN: 74 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:37:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5049 N_CACHED_ALN: 1922 N_COMPUTED_NOTALN: 2907 N_CACHED_NOTALN: 122 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:38:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2048 N_CACHED_ALN: 2389 N_COMPUTED_NOTALN: 5334 N_CACHED_NOTALN: 229 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:38:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4784 N_CACHED_ALN: 1818 N_COMPUTED_NOTALN: 3255 N_CACHED_NOTALN: 143 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:40:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4309 N_CACHED_ALN: 1641 N_COMPUTED_NOTALN: 3845 N_CACHED_NOTALN: 205 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:43:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 12793 N_CACHED_ALN: 3760 N_COMPUTED_NOTALN: 3300 N_CACHED_NOTALN: 147 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:43:\n",
      "\t Finished reads; N_TOT_READS: 18891 N_COMPUTED_ALN: 11612 N_CACHED_ALN: 3161 N_COMPUTED_NOTALN: 4015 N_CACHED_NOTALN: 103 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:43:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:43:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:44:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 12287 N_CACHED_ALN: 3149 N_COMPUTED_NOTALN: 4271 N_CACHED_NOTALN: 293 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:45:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 11075 N_CACHED_ALN: 3703 N_COMPUTED_NOTALN: 4996 N_CACHED_NOTALN: 226 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:46:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9741 N_CACHED_ALN: 4382 N_COMPUTED_NOTALN: 5595 N_CACHED_NOTALN: 282 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:47:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:47:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:47:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 9098 N_CACHED_ALN: 4189 N_COMPUTED_NOTALN: 6367 N_CACHED_NOTALN: 346 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:47:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:49:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5034 N_CACHED_ALN: 4489 N_COMPUTED_NOTALN: 10010 N_CACHED_NOTALN: 467 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:50:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7967 N_CACHED_ALN: 3926 N_COMPUTED_NOTALN: 7567 N_CACHED_NOTALN: 540 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:50:\n",
      "\t Finished reads; N_TOT_READS: 26906 N_COMPUTED_ALN: 18171 N_CACHED_ALN: 4464 N_COMPUTED_NOTALN: 4100 N_CACHED_NOTALN: 171 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:50:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:53:\n",
      "\t Finished reads; N_TOT_READS: 28232 N_COMPUTED_ALN: 18393 N_CACHED_ALN: 3821 N_COMPUTED_NOTALN: 5659 N_CACHED_NOTALN: 359 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:53:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:56:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:56:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:56:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 18260 N_CACHED_ALN: 4737 N_COMPUTED_NOTALN: 6730 N_CACHED_NOTALN: 273 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:56:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:56:\n",
      "\t Finished reads; N_TOT_READS: 30402 N_COMPUTED_ALN: 18535 N_CACHED_ALN: 4799 N_COMPUTED_NOTALN: 6793 N_CACHED_NOTALN: 275 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:56:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:57:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 16403 N_CACHED_ALN: 5577 N_COMPUTED_NOTALN: 7637 N_CACHED_NOTALN: 383 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:57:\n",
      "\t Finished reads; N_TOT_READS: 26369 N_COMPUTED_ALN: 7731 N_CACHED_ALN: 5487 N_COMPUTED_NOTALN: 12568 N_CACHED_NOTALN: 583 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:57:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:58:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15436 N_CACHED_ALN: 5307 N_COMPUTED_NOTALN: 8811 N_CACHED_NOTALN: 446 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:59:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:59:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:59:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:59:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:59:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:01:59:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:00:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 12462 N_CACHED_ALN: 5779 N_COMPUTED_NOTALN: 10968 N_CACHED_NOTALN: 791 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:00:\n",
      "\t Finished reads; N_TOT_READS: 33587 N_COMPUTED_ALN: 18732 N_CACHED_ALN: 6102 N_COMPUTED_NOTALN: 8340 N_CACHED_NOTALN: 413 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:00:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:02:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:02:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:02:\n",
      "\t Finished reads; N_TOT_READS: 33972 N_COMPUTED_ALN: 17893 N_CACHED_ALN: 5893 N_COMPUTED_NOTALN: 9691 N_CACHED_NOTALN: 495 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:02:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:02:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:06:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:06:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:07:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:07:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:07:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:08:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:11:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 18252 N_CACHED_ALN: 6890 N_COMPUTED_NOTALN: 13948 N_CACHED_NOTALN: 910 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:16:\n",
      "\t Finished reads; N_TOT_READS: 44117 N_COMPUTED_ALN: 20659 N_CACHED_ALN: 7404 N_COMPUTED_NOTALN: 15073 N_CACHED_NOTALN: 981 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:16:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:22:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:22:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:02:23:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:02:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:02:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:03:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:03:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:03:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:05:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:11:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:11:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:12:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:12:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:12:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:13:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:14:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:17:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:17:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:18:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:20:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:20:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:21:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:21:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:22:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:23:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:33:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:33:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:34:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:34:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:34:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:04:35:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:31:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "Processing EvoCDA_sg801\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Running CRISPResso with 20 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle --name sample_EvoCDA_801_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_1_S139231.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle --name sample_EvoCDA_801_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_2_S139236.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle --name sample_EvoCDA_801_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_3_S139237.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle --name sample_EvoCDA_801_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_1_S139232.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle --name sample_EvoCDA_801_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_2_S139233.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle --name sample_EvoCDA_801_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_3_S139238.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle --name sample_EvoCDA_801_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_1_S139229.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle --name sample_EvoCDA_801_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_2_S139234.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:34:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle --name sample_EvoCDA_801_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_3_S139235.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle/CRISPResso_on_sample_EvoCDA_801_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle/CRISPResso_on_sample_EvoCDA_801_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle/CRISPResso_on_sample_EvoCDA_801_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle/CRISPResso_on_sample_EvoCDA_801_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle/CRISPResso_on_sample_EvoCDA_801_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle/CRISPResso_on_sample_EvoCDA_801_low_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle/CRISPResso_on_sample_EvoCDA_801_presort_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle/CRISPResso_on_sample_EvoCDA_801_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/premerged/CRISPRessoBatch_on_EvoCDA_sg801_premerged_middle/CRISPResso_on_sample_EvoCDA_801_low_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:39:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:46:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:46:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:46:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:46:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:46:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:46:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:48:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:48:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:49:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:49:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:49:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:49:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:50:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:50:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:51:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:51:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:51:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3341 N_CACHED_ALN: 6491 N_COMPUTED_NOTALN: 123 N_CACHED_NOTALN: 45 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:51:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3629 N_CACHED_ALN: 6205 N_COMPUTED_NOTALN: 130 N_CACHED_NOTALN: 36 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:52:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3522 N_CACHED_ALN: 5797 N_COMPUTED_NOTALN: 469 N_CACHED_NOTALN: 212 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:54:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3821 N_CACHED_ALN: 5843 N_COMPUTED_NOTALN: 253 N_CACHED_NOTALN: 83 \n",
      "\n",
      "Completed in 15 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:55:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3757 N_CACHED_ALN: 5993 N_COMPUTED_NOTALN: 183 N_CACHED_NOTALN: 67 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:55:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:55:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:55:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3866 N_CACHED_ALN: 5988 N_COMPUTED_NOTALN: 117 N_CACHED_NOTALN: 29 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:55:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3849 N_CACHED_ALN: 5851 N_COMPUTED_NOTALN: 248 N_CACHED_NOTALN: 52 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:56:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3409 N_CACHED_ALN: 6427 N_COMPUTED_NOTALN: 136 N_CACHED_NOTALN: 28 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:57:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7278 N_CACHED_ALN: 12401 N_COMPUTED_NOTALN: 235 N_CACHED_NOTALN: 86 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:57:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7257 N_CACHED_ALN: 12394 N_COMPUTED_NOTALN: 277 N_CACHED_NOTALN: 72 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:59:\n",
      "\t Finished reads; N_TOT_READS: 23151 N_COMPUTED_ALN: 8672 N_CACHED_ALN: 14109 N_COMPUTED_NOTALN: 276 N_CACHED_NOTALN: 94 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:59:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:05:59:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7840 N_CACHED_ALN: 10891 N_COMPUTED_NOTALN: 888 N_CACHED_NOTALN: 381 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:00:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7582 N_CACHED_ALN: 11758 N_COMPUTED_NOTALN: 471 N_CACHED_NOTALN: 189 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:00:\n",
      "\t Finished reads; N_TOT_READS: 23974 N_COMPUTED_ALN: 8918 N_CACHED_ALN: 14636 N_COMPUTED_NOTALN: 332 N_CACHED_NOTALN: 88 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:00:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:00:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3254 N_CACHED_ALN: 6538 N_COMPUTED_NOTALN: 144 N_CACHED_NOTALN: 64 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:00:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7083 N_CACHED_ALN: 12604 N_COMPUTED_NOTALN: 225 N_CACHED_NOTALN: 88 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:01:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8051 N_CACHED_ALN: 11487 N_COMPUTED_NOTALN: 326 N_CACHED_NOTALN: 136 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:01:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 7472 N_CACHED_ALN: 11900 N_COMPUTED_NOTALN: 500 N_CACHED_NOTALN: 128 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:01:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6881 N_CACHED_ALN: 12791 N_COMPUTED_NOTALN: 252 N_CACHED_NOTALN: 76 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:02:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:02:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:03:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:03:\n",
      "\t Finished reads; N_TOT_READS: 23588 N_COMPUTED_ALN: 9450 N_CACHED_ALN: 12683 N_COMPUTED_NOTALN: 1042 N_CACHED_NOTALN: 413 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:03:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:03:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:03:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:03:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:05:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 6214 N_CACHED_ALN: 13402 N_COMPUTED_NOTALN: 256 N_CACHED_NOTALN: 128 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:06:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 11171 N_CACHED_ALN: 18371 N_COMPUTED_NOTALN: 324 N_CACHED_NOTALN: 134 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:06:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:06:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:06:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 10319 N_CACHED_ALN: 19236 N_COMPUTED_NOTALN: 345 N_CACHED_NOTALN: 100 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:06:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 11890 N_CACHED_ALN: 17455 N_COMPUTED_NOTALN: 460 N_CACHED_NOTALN: 195 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:06:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 11590 N_CACHED_ALN: 17461 N_COMPUTED_NOTALN: 675 N_CACHED_NOTALN: 274 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:07:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:08:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 11578 N_CACHED_ALN: 17462 N_COMPUTED_NOTALN: 782 N_CACHED_NOTALN: 178 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:09:\n",
      "\t Finished reads; N_TOT_READS: 34011 N_COMPUTED_ALN: 13302 N_CACHED_ALN: 19634 N_COMPUTED_NOTALN: 765 N_CACHED_NOTALN: 310 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:09:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:10:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 9816 N_CACHED_ALN: 19616 N_COMPUTED_NOTALN: 376 N_CACHED_NOTALN: 192 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:12:\n",
      "\t Finished reads; N_TOT_READS: 38799 N_COMPUTED_ALN: 14705 N_CACHED_ALN: 23504 N_COMPUTED_NOTALN: 425 N_CACHED_NOTALN: 165 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:12:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:12:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:13:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 14296 N_CACHED_ALN: 25106 N_COMPUTED_NOTALN: 455 N_CACHED_NOTALN: 143 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Finished reads; N_TOT_READS: 38342 N_COMPUTED_ALN: 14889 N_CACHED_ALN: 22225 N_COMPUTED_NOTALN: 1001 N_CACHED_NOTALN: 227 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 16503 N_CACHED_ALN: 22647 N_COMPUTED_NOTALN: 615 N_CACHED_NOTALN: 235 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Finished reads; N_TOT_READS: 40842 N_COMPUTED_ALN: 16790 N_CACHED_ALN: 23188 N_COMPUTED_NOTALN: 623 N_CACHED_NOTALN: 241 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:14:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:15:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:15:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 13264 N_CACHED_ALN: 26026 N_COMPUTED_NOTALN: 461 N_CACHED_NOTALN: 249 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:15:\n",
      "\t Finished reads; N_TOT_READS: 42332 N_COMPUTED_ALN: 15281 N_CACHED_ALN: 26433 N_COMPUTED_NOTALN: 471 N_CACHED_NOTALN: 147 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:15:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:17:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:17:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:18:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:19:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:19:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:20:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:20:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:20:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:20:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 17071 N_CACHED_ALN: 32080 N_COMPUTED_NOTALN: 550 N_CACHED_NOTALN: 299 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:21:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:21:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:21:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:21:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:27:\n",
      "\t Finished reads; N_TOT_READS: 59653 N_COMPUTED_ALN: 20883 N_CACHED_ALN: 37761 N_COMPUTED_NOTALN: 658 N_CACHED_NOTALN: 351 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:27:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:35:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:35:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:06:35:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:28:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:28:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:29:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:29:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:30:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:30:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:33:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:33:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:34:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:43:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:43:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:43:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:45:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:48:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:48:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:48:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:49:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:49:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:50:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:50:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:51:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:52:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:52:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:52:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:53:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:54:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:58:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:08:58:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:09:00:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:09:00:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:09:00:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:09:01:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:04:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle --name sample_EvoCDA_801_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_1_S139231.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_1_S139231.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle --name sample_EvoCDA_801_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_2_S139236.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_2_S139236.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle --name sample_EvoCDA_801_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_3_S139237.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_3_S139237.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle --name sample_EvoCDA_801_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_1_S139232.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_1_S139232.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle --name sample_EvoCDA_801_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_2_S139233.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_2_S139233.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle --name sample_EvoCDA_801_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_3_S139238.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_3_S139238.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle --name sample_EvoCDA_801_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_1_S139229.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_1_S139229.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle --name sample_EvoCDA_801_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_2_S139234.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_2_S139234.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:07:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle --name sample_EvoCDA_801_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_3_S139235.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_3_S139235.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:12:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:13:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:14:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:14:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:14:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:22:\n",
      "\t Estimating average read length... \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:22:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:22:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:22:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_1_S139232.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_1_S139232.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:22:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:22:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_1_S139229.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_1_S139229.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:24:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:24:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:24:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:24:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:26:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:26:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:26:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_3_S139235.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_3_S139235.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:27:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:27:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:27:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_1_S139231.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_1_S139231.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 14 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:28:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:28:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:28:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_2_S139233.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_2_S139233.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 15 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:28:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:29:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:29:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:29:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:29:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_3_S139238.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_3_S139238.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_low_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 15 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:29:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:29:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:29:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_2_S139234.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_2_S139234.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_presort_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:30:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:30:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:30:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2048 N_CACHED_ALN: 6715 N_COMPUTED_NOTALN: 1155 N_CACHED_NOTALN: 82 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:30:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2231 N_CACHED_ALN: 6596 N_COMPUTED_NOTALN: 1106 N_CACHED_NOTALN: 67 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:31:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:31:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:31:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:31:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 18 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:32:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:32:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:32:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_2_S139236.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_2_S139236.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 19 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:33:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:33:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:33:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:33:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:33:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_3_S139237.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_3_S139237.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_801_high_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:35:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:35:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:35:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2123 N_CACHED_ALN: 6738 N_COMPUTED_NOTALN: 1084 N_CACHED_NOTALN: 55 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:36:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2339 N_CACHED_ALN: 6810 N_COMPUTED_NOTALN: 794 N_CACHED_NOTALN: 57 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:36:\n",
      "\t Finished reads; N_TOT_READS: 19979 N_COMPUTED_ALN: 4432 N_CACHED_ALN: 13379 N_COMPUTED_NOTALN: 2005 N_CACHED_NOTALN: 163 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:36:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:36:\n",
      "\t Finished reads; N_TOT_READS: 19754 N_COMPUTED_ALN: 4625 N_CACHED_ALN: 13092 N_COMPUTED_NOTALN: 1902 N_CACHED_NOTALN: 135 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:36:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:36:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:36:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:38:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:38:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:38:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:38:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:38:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:38:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:39:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1923 N_CACHED_ALN: 6002 N_COMPUTED_NOTALN: 1959 N_CACHED_NOTALN: 116 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:40:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3616 N_CACHED_ALN: 14114 N_COMPUTED_NOTALN: 2113 N_CACHED_NOTALN: 157 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:41:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4218 N_CACHED_ALN: 14169 N_COMPUTED_NOTALN: 1467 N_CACHED_NOTALN: 146 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:42:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2108 N_CACHED_ALN: 6936 N_COMPUTED_NOTALN: 891 N_CACHED_NOTALN: 65 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:45:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1796 N_CACHED_ALN: 4874 N_COMPUTED_NOTALN: 3166 N_CACHED_NOTALN: 164 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:45:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2006 N_CACHED_ALN: 5325 N_COMPUTED_NOTALN: 2560 N_CACHED_NOTALN: 109 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:46:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 6392 N_CACHED_ALN: 21415 N_COMPUTED_NOTALN: 1960 N_CACHED_NOTALN: 233 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:46:\n",
      "\t Finished reads; N_TOT_READS: 29771 N_COMPUTED_ALN: 6159 N_CACHED_ALN: 20537 N_COMPUTED_NOTALN: 2844 N_CACHED_NOTALN: 231 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:46:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:46:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:47:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3720 N_CACHED_ALN: 14326 N_COMPUTED_NOTALN: 1763 N_CACHED_NOTALN: 191 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:47:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3493 N_CACHED_ALN: 12435 N_COMPUTED_NOTALN: 3799 N_CACHED_NOTALN: 273 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:48:\n",
      "\t Finished reads; N_TOT_READS: 33956 N_COMPUTED_ALN: 7524 N_CACHED_ALN: 24043 N_COMPUTED_NOTALN: 2137 N_CACHED_NOTALN: 252 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:48:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:48:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1484 N_CACHED_ALN: 3766 N_COMPUTED_NOTALN: 4527 N_CACHED_NOTALN: 223 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:49:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:49:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:49:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:50:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:50:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:51:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:52:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5401 N_CACHED_ALN: 21686 N_COMPUTED_NOTALN: 2604 N_CACHED_NOTALN: 309 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:54:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5538 N_CACHED_ALN: 18742 N_COMPUTED_NOTALN: 5308 N_CACHED_NOTALN: 412 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:55:\n",
      "\t Finished reads; N_TOT_READS: 30260 N_COMPUTED_ALN: 5601 N_CACHED_ALN: 18911 N_COMPUTED_NOTALN: 5334 N_CACHED_NOTALN: 414 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:55:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:55:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3481 N_CACHED_ALN: 11247 N_COMPUTED_NOTALN: 5018 N_CACHED_NOTALN: 254 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:56:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3456 N_CACHED_ALN: 10356 N_COMPUTED_NOTALN: 5811 N_CACHED_NOTALN: 377 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:57:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:57:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:57:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 7189 N_CACHED_ALN: 28996 N_COMPUTED_NOTALN: 3391 N_CACHED_NOTALN: 424 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:10:57:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:00:\n",
      "\t Finished reads; N_TOT_READS: 46292 N_COMPUTED_ALN: 8656 N_CACHED_ALN: 33351 N_COMPUTED_NOTALN: 3793 N_CACHED_NOTALN: 492 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:00:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:03:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:03:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:04:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5423 N_CACHED_ALN: 17128 N_COMPUTED_NOTALN: 7015 N_CACHED_NOTALN: 434 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:04:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2813 N_CACHED_ALN: 7911 N_COMPUTED_NOTALN: 8763 N_CACHED_NOTALN: 513 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:04:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:06:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5245 N_CACHED_ALN: 15935 N_COMPUTED_NOTALN: 8259 N_CACHED_NOTALN: 561 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:07:\n",
      "\t Finished reads; N_TOT_READS: 33986 N_COMPUTED_ALN: 6361 N_CACHED_ALN: 19452 N_COMPUTED_NOTALN: 7699 N_CACHED_NOTALN: 474 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:07:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:08:\n",
      "\t Finished reads; N_TOT_READS: 31978 N_COMPUTED_ALN: 5709 N_CACHED_ALN: 16999 N_COMPUTED_NOTALN: 8668 N_CACHED_NOTALN: 602 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:08:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:09:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:09:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:09:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:11:10:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:12:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:12:54:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:12:54:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:12:55:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:12:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:12:56:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:12:57:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:12:58:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:17:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:17:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:19:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:19:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:20:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:21:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:22:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:22:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:23:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:27:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:27:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:29:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:29:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:29:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:30:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:30:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:30:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:31:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:31:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:32:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:33:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:33:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:33:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:13:34:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:32:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle --name sample_EvoCDA_801_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_1_S139231.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle --name sample_EvoCDA_801_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_2_S139236.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle --name sample_EvoCDA_801_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_high_3_S139237.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle --name sample_EvoCDA_801_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_1_S139232.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle --name sample_EvoCDA_801_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_2_S139233.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle --name sample_EvoCDA_801_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_low_3_S139238.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle --name sample_EvoCDA_801_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_1_S139229.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle --name sample_EvoCDA_801_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_2_S139234.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:35:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle --name sample_EvoCDA_801_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_801_presort_3_S139235.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq AAGAATGAGGCATGAATGAA --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:40:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_801_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:40:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_801_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:40:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_801_high_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:40:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_801_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:40:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_801_presort_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:40:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_801_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:40:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_801_high_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_801_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg801/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg801_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_801_presort_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:41:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:46:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:46:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:47:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:47:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:50:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:50:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:50:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:50:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:51:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2511 N_CACHED_ALN: 5682 N_COMPUTED_NOTALN: 1561 N_CACHED_NOTALN: 246 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:52:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2578 N_CACHED_ALN: 5580 N_COMPUTED_NOTALN: 1660 N_CACHED_NOTALN: 182 \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:52:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:52:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:52:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:52:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:53:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:53:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:54:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:54:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:54:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2559 N_CACHED_ALN: 6182 N_COMPUTED_NOTALN: 1090 N_CACHED_NOTALN: 169 \n",
      "\n",
      "Completed in 13 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:54:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:54:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:55:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2276 N_CACHED_ALN: 5760 N_COMPUTED_NOTALN: 1774 N_CACHED_NOTALN: 190 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:55:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4715 N_CACHED_ALN: 12083 N_COMPUTED_NOTALN: 2701 N_CACHED_NOTALN: 501 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:56:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4851 N_CACHED_ALN: 11829 N_COMPUTED_NOTALN: 2971 N_CACHED_NOTALN: 349 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:57:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4536 N_CACHED_ALN: 12953 N_COMPUTED_NOTALN: 2131 N_CACHED_NOTALN: 380 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:58:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2379 N_CACHED_ALN: 6174 N_COMPUTED_NOTALN: 1240 N_CACHED_NOTALN: 207 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:59:\n",
      "\t Finished reads; N_TOT_READS: 28226 N_COMPUTED_ALN: 6757 N_CACHED_ALN: 17226 N_COMPUTED_NOTALN: 3563 N_CACHED_NOTALN: 680 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:59:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:59:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2218 N_CACHED_ALN: 4873 N_COMPUTED_NOTALN: 2634 N_CACHED_NOTALN: 275 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:59:\n",
      "\t Finished reads; N_TOT_READS: 27376 N_COMPUTED_ALN: 6731 N_CACHED_ALN: 16396 N_COMPUTED_NOTALN: 3760 N_CACHED_NOTALN: 489 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:14:59:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:00:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4110 N_CACHED_ALN: 12015 N_COMPUTED_NOTALN: 3417 N_CACHED_NOTALN: 458 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:00:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2200 N_CACHED_ALN: 4013 N_COMPUTED_NOTALN: 3393 N_CACHED_NOTALN: 394 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:01:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 6764 N_CACHED_ALN: 19737 N_COMPUTED_NOTALN: 2895 N_CACHED_NOTALN: 604 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:01:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:01:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:01:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:01:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:01:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:02:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:02:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4345 N_CACHED_ALN: 12813 N_COMPUTED_NOTALN: 2343 N_CACHED_NOTALN: 499 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:02:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 2458 N_CACHED_ALN: 3888 N_COMPUTED_NOTALN: 3401 N_CACHED_NOTALN: 253 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:03:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1754 N_CACHED_ALN: 2661 N_COMPUTED_NOTALN: 5069 N_CACHED_NOTALN: 516 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:04:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 9109 N_CACHED_ALN: 26505 N_COMPUTED_NOTALN: 3593 N_CACHED_NOTALN: 793 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:04:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 6009 N_CACHED_ALN: 18507 N_COMPUTED_NOTALN: 4748 N_CACHED_NOTALN: 736 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:05:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4030 N_CACHED_ALN: 10127 N_COMPUTED_NOTALN: 5177 N_CACHED_NOTALN: 666 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:06:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 6147 N_CACHED_ALN: 19708 N_COMPUTED_NOTALN: 3392 N_CACHED_NOTALN: 753 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:06:\n",
      "\t Finished reads; N_TOT_READS: 46561 N_COMPUTED_ALN: 10724 N_CACHED_ALN: 30941 N_COMPUTED_NOTALN: 4014 N_CACHED_NOTALN: 882 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:06:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:06:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:07:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 8193 N_CACHED_ALN: 24928 N_COMPUTED_NOTALN: 5904 N_CACHED_NOTALN: 975 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:08:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4122 N_CACHED_ALN: 8435 N_COMPUTED_NOTALN: 6509 N_CACHED_NOTALN: 934 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:09:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 8087 N_CACHED_ALN: 26689 N_COMPUTED_NOTALN: 4219 N_CACHED_NOTALN: 1005 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:09:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:10:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:10:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:10:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:10:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 4652 N_CACHED_ALN: 7929 N_COMPUTED_NOTALN: 6775 N_CACHED_NOTALN: 644 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:11:\n",
      "\t Finished reads; N_TOT_READS: 47533 N_COMPUTED_ALN: 9838 N_CACHED_ALN: 29815 N_COMPUTED_NOTALN: 6733 N_CACHED_NOTALN: 1147 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:11:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:11:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5707 N_CACHED_ALN: 15790 N_COMPUTED_NOTALN: 7407 N_CACHED_NOTALN: 1096 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:12:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 10020 N_CACHED_ALN: 33715 N_COMPUTED_NOTALN: 4994 N_CACHED_NOTALN: 1271 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:12:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 3463 N_CACHED_ALN: 5565 N_COMPUTED_NOTALN: 9795 N_CACHED_NOTALN: 1177 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:13:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:14:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:14:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 5861 N_CACHED_ALN: 13286 N_COMPUTED_NOTALN: 9352 N_CACHED_NOTALN: 1501 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:14:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:15:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 12224 N_CACHED_ALN: 40482 N_COMPUTED_NOTALN: 5764 N_CACHED_NOTALN: 1530 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:16:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 7505 N_CACHED_ALN: 21750 N_COMPUTED_NOTALN: 9285 N_CACHED_NOTALN: 1460 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:18:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 6607 N_CACHED_ALN: 12335 N_COMPUTED_NOTALN: 9983 N_CACHED_NOTALN: 1075 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:18:\n",
      "\t Finished reads; N_TOT_READS: 69096 N_COMPUTED_ALN: 14222 N_CACHED_ALN: 46733 N_COMPUTED_NOTALN: 6427 N_CACHED_NOTALN: 1714 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:18:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:20:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 7551 N_CACHED_ALN: 18466 N_COMPUTED_NOTALN: 11932 N_CACHED_NOTALN: 2051 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:20:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 9501 N_CACHED_ALN: 27604 N_COMPUTED_NOTALN: 11084 N_CACHED_NOTALN: 1811 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:21:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 4885 N_CACHED_ALN: 8841 N_COMPUTED_NOTALN: 14290 N_CACHED_NOTALN: 1984 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:22:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:23:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:23:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:23:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:23:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 8307 N_CACHED_ALN: 17421 N_COMPUTED_NOTALN: 12740 N_CACHED_NOTALN: 1532 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:25:\n",
      "\t Finished reads; N_TOT_READS: 58640 N_COMPUTED_ALN: 11275 N_CACHED_ALN: 32765 N_COMPUTED_NOTALN: 12532 N_CACHED_NOTALN: 2068 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:25:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:26:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 9521 N_CACHED_ALN: 23682 N_COMPUTED_NOTALN: 14261 N_CACHED_NOTALN: 2536 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:28:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:28:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:28:\n",
      "\t Finished reads; N_TOT_READS: 53516 N_COMPUTED_ALN: 10275 N_CACHED_ALN: 25470 N_COMPUTED_NOTALN: 15075 N_CACHED_NOTALN: 2696 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:28:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:28:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:29:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 6321 N_CACHED_ALN: 12552 N_COMPUTED_NOTALN: 18402 N_CACHED_NOTALN: 2725 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:29:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 10204 N_CACHED_ALN: 22497 N_COMPUTED_NOTALN: 15353 N_CACHED_NOTALN: 1946 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:31:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:31:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:32:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:35:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 12085 N_CACHED_ALN: 27654 N_COMPUTED_NOTALN: 17891 N_CACHED_NOTALN: 2370 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:36:\n",
      "\t Finished reads; N_TOT_READS: 61820 N_COMPUTED_ALN: 12405 N_CACHED_ALN: 28630 N_COMPUTED_NOTALN: 18331 N_CACHED_NOTALN: 2454 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:36:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:38:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 7948 N_CACHED_ALN: 16162 N_COMPUTED_NOTALN: 22506 N_CACHED_NOTALN: 3384 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:39:\n",
      "\t Finished reads; N_TOT_READS: 51344 N_COMPUTED_ALN: 8127 N_CACHED_ALN: 16692 N_COMPUTED_NOTALN: 23041 N_CACHED_NOTALN: 3484 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:39:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:39:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:40:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:40:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:40:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:41:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:41:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:15:41:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:13:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:14:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:15:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:15:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:16:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:17:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:29:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:29:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:30:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:36:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:36:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:37:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:46:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:46:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:46:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:47:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:51:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:51:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:51:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:52:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:56:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:56:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:56:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:57:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:57:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:58:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:58:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:17:59:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:00:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:00:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:00:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:01:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:50:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:18:53:\n",
      "\t \n",
      "\n",
      "ERROR: Offset around cut would extend to the left of the amplicon. Please decrease plot_window_size parameter. Cut point: 18 window: 20 reference: 250 \n",
      "\n",
      "Processing EvoCDA_sg792\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Running CRISPResso with 20 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle --name sample_EvoCDA_792_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_1_S139278.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle --name sample_EvoCDA_792_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_2_S139279.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle --name sample_EvoCDA_792_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_3_S139283.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle --name sample_EvoCDA_792_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_1_S139275.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle --name sample_EvoCDA_792_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_2_S139280.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle --name sample_EvoCDA_792_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_3_S139281.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle --name sample_EvoCDA_792_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_1_S139276.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle --name sample_EvoCDA_792_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_2_S139277.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:18:56:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle --name sample_EvoCDA_792_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/FLASH/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_3_S139282.extendedFrags.fastq\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle/CRISPResso_on_sample_EvoCDA_792_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle/CRISPResso_on_sample_EvoCDA_792_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle/CRISPResso_on_sample_EvoCDA_792_high_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle/CRISPResso_on_sample_EvoCDA_792_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle/CRISPResso_on_sample_EvoCDA_792_low_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle/CRISPResso_on_sample_EvoCDA_792_presort_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle/CRISPResso_on_sample_EvoCDA_792_presort_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle/CRISPResso_on_sample_EvoCDA_792_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:01:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/premerged/CRISPRessoBatch_on_EvoCDA_sg792_premerged_middle/CRISPResso_on_sample_EvoCDA_792_low_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:02:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:02:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:02:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:02:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:02:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 2 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:04:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:04:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 4 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "Completed in 8 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:10:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:10:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:10:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:10:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:13:\n",
      "\t Finished reads; N_TOT_READS: 8788 N_COMPUTED_ALN: 6111 N_CACHED_ALN: 2281 N_COMPUTED_NOTALN: 348 N_CACHED_NOTALN: 48 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:13:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:13:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:13:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:15:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:15:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:15:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4965 N_CACHED_ALN: 4041 N_COMPUTED_NOTALN: 761 N_CACHED_NOTALN: 233 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:15:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:16:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6630 N_CACHED_ALN: 3071 N_COMPUTED_NOTALN: 229 N_CACHED_NOTALN: 70 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:16:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6450 N_CACHED_ALN: 3213 N_COMPUTED_NOTALN: 264 N_CACHED_NOTALN: 73 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:17:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6857 N_CACHED_ALN: 2752 N_COMPUTED_NOTALN: 293 N_CACHED_NOTALN: 98 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:17:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 7226 N_CACHED_ALN: 2402 N_COMPUTED_NOTALN: 316 N_CACHED_NOTALN: 56 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:19:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6793 N_CACHED_ALN: 3011 N_COMPUTED_NOTALN: 155 N_CACHED_NOTALN: 41 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:19:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6663 N_CACHED_ALN: 3125 N_COMPUTED_NOTALN: 159 N_CACHED_NOTALN: 53 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:22:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6578 N_CACHED_ALN: 3144 N_COMPUTED_NOTALN: 199 N_CACHED_NOTALN: 79 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:24:\n",
      "\t Finished reads; N_TOT_READS: 19711 N_COMPUTED_ALN: 9831 N_CACHED_ALN: 7945 N_COMPUTED_NOTALN: 1467 N_CACHED_NOTALN: 468 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:24:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:25:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 13612 N_CACHED_ALN: 5846 N_COMPUTED_NOTALN: 406 N_CACHED_NOTALN: 136 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:25:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 13240 N_CACHED_ALN: 6128 N_COMPUTED_NOTALN: 487 N_CACHED_NOTALN: 145 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:26:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 13478 N_CACHED_ALN: 5749 N_COMPUTED_NOTALN: 599 N_CACHED_NOTALN: 174 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:26:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 12921 N_CACHED_ALN: 6336 N_COMPUTED_NOTALN: 544 N_CACHED_NOTALN: 199 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:26:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 12864 N_CACHED_ALN: 6731 N_COMPUTED_NOTALN: 290 N_CACHED_NOTALN: 115 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:27:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 13039 N_CACHED_ALN: 6573 N_COMPUTED_NOTALN: 269 N_CACHED_NOTALN: 119 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:28:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:28:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:28:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:28:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 11461 N_CACHED_ALN: 7976 N_COMPUTED_NOTALN: 378 N_CACHED_NOTALN: 185 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:29:\n",
      "\t Finished reads; N_TOT_READS: 22735 N_COMPUTED_ALN: 15710 N_CACHED_ALN: 6438 N_COMPUTED_NOTALN: 444 N_CACHED_NOTALN: 143 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:29:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:30:\n",
      "\t Finished reads; N_TOT_READS: 23892 N_COMPUTED_ALN: 15414 N_CACHED_ALN: 7590 N_COMPUTED_NOTALN: 659 N_CACHED_NOTALN: 229 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:30:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:31:\n",
      "\t Finished reads; N_TOT_READS: 24328 N_COMPUTED_ALN: 16193 N_CACHED_ALN: 7377 N_COMPUTED_NOTALN: 588 N_CACHED_NOTALN: 170 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:31:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:31:\n",
      "\t Finished reads; N_TOT_READS: 24495 N_COMPUTED_ALN: 16509 N_CACHED_ALN: 7065 N_COMPUTED_NOTALN: 713 N_CACHED_NOTALN: 208 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:31:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:34:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:34:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:35:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 19347 N_CACHED_ALN: 10043 N_COMPUTED_NOTALN: 435 N_CACHED_NOTALN: 175 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:35:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:36:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:36:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:36:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 16965 N_CACHED_ALN: 12168 N_COMPUTED_NOTALN: 580 N_CACHED_NOTALN: 287 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:36:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:36:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 19523 N_CACHED_ALN: 9878 N_COMPUTED_NOTALN: 427 N_CACHED_NOTALN: 172 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:37:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:37:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:37:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:37:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:37:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:37:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:37:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:37:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:37:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:38:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:41:\n",
      "\t Finished reads; N_TOT_READS: 35722 N_COMPUTED_ALN: 23344 N_CACHED_ALN: 11650 N_COMPUTED_NOTALN: 529 N_CACHED_NOTALN: 199 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:41:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:41:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:44:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 22292 N_CACHED_ALN: 16542 N_COMPUTED_NOTALN: 751 N_CACHED_NOTALN: 415 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:44:\n",
      "\t Finished reads; N_TOT_READS: 36713 N_COMPUTED_ALN: 23988 N_CACHED_ALN: 11995 N_COMPUTED_NOTALN: 535 N_CACHED_NOTALN: 195 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:44:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:44:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:49:\n",
      "\t Finished reads; N_TOT_READS: 45697 N_COMPUTED_ALN: 25610 N_CACHED_ALN: 18785 N_COMPUTED_NOTALN: 849 N_CACHED_NOTALN: 453 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:49:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:49:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:50:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:50:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:50:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:51:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:53:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:53:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:54:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:58:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:58:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:59:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:19:59:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:21:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:21:29:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:21:29:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:21:31:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:21:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:21:48:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:21:48:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:21:50:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:01:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:01:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:03:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:04:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:05:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:05:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:05:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:06:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:07:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:08:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:08:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:10:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:13:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:14:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:15:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:22:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:22:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:22:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:22:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:23:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:24:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:24:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:24:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:22:25:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:38:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle --name sample_EvoCDA_792_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_1_S139278.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_1_S139278.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle --name sample_EvoCDA_792_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_2_S139279.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_2_S139279.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle --name sample_EvoCDA_792_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_3_S139283.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_3_S139283.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle --name sample_EvoCDA_792_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_1_S139275.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_1_S139275.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle --name sample_EvoCDA_792_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_2_S139280.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_2_S139280.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle --name sample_EvoCDA_792_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_3_S139281.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_3_S139281.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle --name sample_EvoCDA_792_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_1_S139276.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_1_S139276.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle --name sample_EvoCDA_792_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_2_S139277.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_2_S139277.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:41:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle --name sample_EvoCDA_792_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_3_S139282.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --fastq_r2 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_3_S139282.R2.fastq.gz\" --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:47:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:48:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:48:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:48:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:48:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 7 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:55:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:55:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:55:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_2_S139280.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_2_S139280.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:57:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:57:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:57:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:58:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:58:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_2_S139279.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_2_S139279.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 10 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:58:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:58:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:58:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_2_S139277.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_2/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_2_S139277.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_2 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_2/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:59:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:59:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 11 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:59:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:59:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:59:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:59:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:23:59:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_3_S139283.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_3_S139283.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:00:\n",
      "\t Estimating average read length... \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:00:\n",
      "\t Estimating average read length... \n",
      "\n",
      "Completed in 12 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:00:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:00:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:00:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_1_S139276.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_1_S139276.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:00:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:00:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_1_S139275.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_1_S139275.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:01:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:01:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_1_S139278.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_1_S139278.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_1 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_high_1/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:01:\n",
      "\t Done! \n",
      "\n",
      "Completed in 13 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:01:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:01:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:01:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:01:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:01:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_3_S139282.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_3_S139282.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_presort_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:02:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:02:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:03:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:03:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:09:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4340 N_CACHED_ALN: 4120 N_COMPUTED_NOTALN: 1419 N_CACHED_NOTALN: 121 \n",
      "\n",
      "Completed in 21 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:09:\n",
      "\t Estimating average read length... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:10:\n",
      "\t Merging paired sequences with Flash... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:10:\n",
      "\t Running FLASH command: flash \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_3_S139281.R1_filtered.fastq.gz\" \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_3/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_3_S139281.R2_filtered.fastq.gz\" --min-overlap 10 --max-overlap 100 --allow-outies -z -d /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_3 -o out >>/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_PE/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_PE_middle/CRISPResso_on_sample_EvoCDA_792_low_3/CRISPResso_RUNNING_LOG.txt 2>&1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:13:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:13:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:13:\n",
      "\t Finished reads; N_TOT_READS: 8441 N_COMPUTED_ALN: 2554 N_CACHED_ALN: 1668 N_COMPUTED_NOTALN: 4131 N_CACHED_NOTALN: 88 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:13:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:13:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4182 N_CACHED_ALN: 3623 N_COMPUTED_NOTALN: 2074 N_CACHED_NOTALN: 121 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:14:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:14:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:14:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:14:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3832 N_CACHED_ALN: 3076 N_COMPUTED_NOTALN: 2987 N_CACHED_NOTALN: 105 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:14:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:16:\n",
      "\t Finished reads; N_TOT_READS: 17543 N_COMPUTED_ALN: 7502 N_CACHED_ALN: 7469 N_COMPUTED_NOTALN: 2372 N_CACHED_NOTALN: 200 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:16:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:17:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:18:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3427 N_CACHED_ALN: 2894 N_COMPUTED_NOTALN: 3520 N_CACHED_NOTALN: 159 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:19:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3113 N_CACHED_ALN: 2693 N_COMPUTED_NOTALN: 4062 N_CACHED_NOTALN: 132 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:19:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:19:\n",
      "\t Finished reads; N_TOT_READS: 13924 N_COMPUTED_ALN: 5662 N_CACHED_ALN: 4223 N_COMPUTED_NOTALN: 3883 N_CACHED_NOTALN: 156 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:19:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:19:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3484 N_CACHED_ALN: 2731 N_COMPUTED_NOTALN: 3640 N_CACHED_NOTALN: 145 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:19:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:19:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1932 N_CACHED_ALN: 3259 N_COMPUTED_NOTALN: 4507 N_CACHED_NOTALN: 302 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:20:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:21:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:21:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:22:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:24:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 8095 N_CACHED_ALN: 7711 N_COMPUTED_NOTALN: 3906 N_CACHED_NOTALN: 288 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:24:\n",
      "\t Finished reads; N_TOT_READS: 20574 N_COMPUTED_ALN: 8421 N_CACHED_ALN: 7877 N_COMPUTED_NOTALN: 3984 N_CACHED_NOTALN: 292 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:24:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:27:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3286 N_CACHED_ALN: 3201 N_COMPUTED_NOTALN: 3380 N_CACHED_NOTALN: 133 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:27:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:28:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:28:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:29:\n",
      "\t Finished reads; N_TOT_READS: 16419 N_COMPUTED_ALN: 5738 N_CACHED_ALN: 4675 N_COMPUTED_NOTALN: 5743 N_CACHED_NOTALN: 263 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:29:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:29:\n",
      "\t Finished reads; N_TOT_READS: 17666 N_COMPUTED_ALN: 6098 N_CACHED_ALN: 5284 N_COMPUTED_NOTALN: 5965 N_CACHED_NOTALN: 319 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:29:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:30:\n",
      "\t Finished reads; N_TOT_READS: 17310 N_COMPUTED_ALN: 3421 N_CACHED_ALN: 5894 N_COMPUTED_NOTALN: 7415 N_CACHED_NOTALN: 580 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:30:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Finished reads; N_TOT_READS: 17846 N_COMPUTED_ALN: 5802 N_CACHED_ALN: 4933 N_COMPUTED_NOTALN: 6858 N_CACHED_NOTALN: 253 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:31:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:32:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:32:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:32:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:33:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:33:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:33:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:41:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 5586 N_CACHED_ALN: 7403 N_COMPUTED_NOTALN: 6644 N_CACHED_NOTALN: 367 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:24:54:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 7905 N_CACHED_ALN: 11749 N_COMPUTED_NOTALN: 9771 N_CACHED_NOTALN: 575 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:25:03:\n",
      "\t Finished reads; N_TOT_READS: 37133 N_COMPUTED_ALN: 10713 N_CACHED_ALN: 13989 N_COMPUTED_NOTALN: 11738 N_CACHED_NOTALN: 693 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:25:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:25:03:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:25:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:25:07:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:25:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:25:07:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:25:07:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:28:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:28:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:29:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:47:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:47:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:49:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:49:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:52:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:52:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:52:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:53:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:54:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:54:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:55:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:59:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:59:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:26:59:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:00:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:00:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:01:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:01:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:01:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:02:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:02:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:02:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:04:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:22:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:22:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:22:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:24:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:24:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:24:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:27:25:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:46:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_1_S139278.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_2_S139279.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_3_S139283.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_1_S139275.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_2_S139280.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_3_S139281.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_1_S139276.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_2_S139277.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:49:\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_3_S139282.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_792_presort_3 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (ATTCCCAGGCCCATCTCACC) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_792_high_2 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (ATTCCCAGGCCCATCTCACC) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_792_high_1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_792_presort_1 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (ATTCCCAGGCCCATCTCACC) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (ATTCCCAGGCCCATCTCACC) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_792_high_3 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (ATTCCCAGGCCCATCTCACC) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_792_low_1 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (ATTCCCAGGCCCATCTCACC) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_792_presort_2 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (ATTCCCAGGCCCATCTCACC) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_792_low_2 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (ATTCCCAGGCCCATCTCACC) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle/CRISPResso_on_sample_EvoCDA_792_low_3 \n",
      "\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t sgRNA error, please check your input.\n",
      "\n",
      "ERROR: The guide sequence 0 (ATTCCCAGGCCCATCTCACC) provided is not present in the amplicon sequences!\n",
      "\n",
      "Please check your input! \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t CRISPResso command failed (return value 2) on batch #8: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_3_S139282.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t CRISPResso command failed (return value 2) on batch #0: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_1_S139278.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t CRISPResso command failed (return value 2) on batch #1: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_2_S139279.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t CRISPResso command failed (return value 2) on batch #2: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_3_S139283.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:28:54:\n",
      "\t CRISPResso command failed (return value 2) on batch #4: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_2_S139280.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:28:55:\n",
      "\t CRISPResso command failed (return value 2) on batch #3: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_1_S139275.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:28:55:\n",
      "\t CRISPResso command failed (return value 2) on batch #5: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_3_S139281.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:28:55:\n",
      "\t CRISPResso command failed (return value 2) on batch #7: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_2_S139277.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "WARNING @ Thu, 05 Oct 2023 17:28:55:\n",
      "\t CRISPResso command failed (return value 2) on batch #6: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_1_S139276.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "CRISPResso2 failed\n",
      "CRITICAL @ Thu, 05 Oct 2023 17:28:55:\n",
      "\t \n",
      "\n",
      "ERROR: CRISPResso batch #0 failed. For more information, try running the command: \"CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/20231005_v1_CRISPResso2Batch_Results_q30/EvoCDA_sg792/unmerged_R1/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R1_middle --name sample_EvoCDA_792_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_1_S139278.R1.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3\" \n",
      "\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                 _                                             _                \n",
      "                '  )                                          '  )              \n",
      "                .-'            _________________              .-'               \n",
      "               (____          | __    ___ __    |            (____              \n",
      "            C)|     \\         ||__) /\\ | /  |__||         C)|     \\             \n",
      "              \\     /         ||__)/--\\| \\__|  ||           \\     /             \n",
      "               \\___/          |_________________|            \\___/              \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\n",
      "\t Running CRISPResso on batch #0/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle --name sample_EvoCDA_792_high_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_1_S139278.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\n",
      "\t Running CRISPResso on batch #1/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle --name sample_EvoCDA_792_high_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_2_S139279.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\n",
      "\t Running CRISPResso on batch #2/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle --name sample_EvoCDA_792_high_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_high_3_S139283.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\n",
      "\t Running CRISPResso on batch #3/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle --name sample_EvoCDA_792_low_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_1_S139275.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\n",
      "\t Running CRISPResso on batch #4/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle --name sample_EvoCDA_792_low_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_2_S139280.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\r\n",
      "\t Running CRISPResso on batch #5/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle --name sample_EvoCDA_792_low_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_low_3_S139281.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\r\n",
      "\t Running CRISPResso on batch #6/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle --name sample_EvoCDA_792_presort_1 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_1_S139276.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\r\n",
      "\t Running CRISPResso on batch #7/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle --name sample_EvoCDA_792_presort_2 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_2_S139277.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "INFO  @ Thu, 05 Oct 2023 17:28:58:\r\n",
      "\t Running CRISPResso on batch #8/9: CRISPResso -o /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle --name sample_EvoCDA_792_presort_3 --needleman_wunsch_gap_extend -2 --aln_seed_count 5 --amplicon_name cd19 --amplicon_seq GCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG --max_rows_alleles_around_cut_to_plot 500 --prime_editing_pegRNA_extension_quantification_window_size 5 --fastq_r1 \"/data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Data/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous/ProcessedData/BSF_1488_000000000-KJJ4P_L1/BSF_1488_000000000-KJJ4P_1_EvoCDA_792_pre_3_S139282.R2.fastq.gz\" --quantification_window_size 0 --quantification_window_center \"-3\" --trimmomatic_command trimmomatic --conversion_nuc_from C --min_bp_quality_or_N 0 --base_editor_output --default_min_aln_score 60 --needleman_wunsch_gap_incentive 1 --min_paired_end_reads_overlap 10 --plot_window_size 20 --prime_editing_pegRNA_scaffold_min_match_length 1 --aln_seed_min 2 --aln_seed_len 10 --needleman_wunsch_gap_open -20 --max_paired_end_reads_overlap 100 --guide_seq ATTCCCAGGCCCATCTCACC --conversion_nuc_to T --flexiguide_homology 80 --flash_command flash --min_single_bp_quality 0 --exclude_bp_from_left 3 --no_rerun --needleman_wunsch_aln_matrix_loc EDNAFULL --min_average_read_quality 30 --min_frequency_alleles_around_cut_to_plot 0.001 --exclude_bp_from_right 3 \r\n",
      "\r\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:03:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_792_high_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:03:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_792_low_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:03:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_792_high_3 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:03:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_792_high_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:03:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_792_low_2 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:03:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_792_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:03:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_792_presort_2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:03:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_792_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.1.3]                           \n",
      "[Note that starting in version 2.1.0 insertion quantification has been changed\n",
      "to only include insertions completely contained by the quantification window.\n",
      "To use the legacy quantification method (i.e. include insertions directly adjacent\n",
      "to the quantification window) please use the parameter --use_legacy_insertion_quantification]\n",
      "                 [For support contact kclement@mgh.harvard.edu]                 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:03:\n",
      "\t Creating Folder /data/pinello/PROJECTS/2023_08_CD19_manuscript/Core_Processing_Analysis/Analysis/ABE8e_Pooled_Endogenous_Redo_AND_ABE8e_sg219_Endogenous_AND_EvoCDA_Hits_Endogenous_CRISPREsso2_Analysis/Davide_CD19_enhancer_BE_screen_EvoCDA_sg792_unmerged_R2.settings/CRISPRessoBatch_on_EvoCDA_sg792_unmerged_R2_middle/CRISPResso_on_sample_EvoCDA_792_presort_3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 3 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:07:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:07:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 4 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:08:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:08:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 4 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:08:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:08:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:09:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:09:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:09:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:09:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:10:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:10:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 5 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:10:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:10:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 6 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:11:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:11:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 9 seconds\n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:14:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:14:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:18:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 7463 N_CACHED_ALN: 891 N_COMPUTED_NOTALN: 1584 N_CACHED_NOTALN: 62 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:18:\n",
      "\t Finished reads; N_TOT_READS: 8584 N_COMPUTED_ALN: 3794 N_CACHED_ALN: 514 N_COMPUTED_NOTALN: 4208 N_CACHED_NOTALN: 68 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:18:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:19:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 6725 N_CACHED_ALN: 995 N_COMPUTED_NOTALN: 2199 N_CACHED_NOTALN: 81 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:19:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5972 N_CACHED_ALN: 909 N_COMPUTED_NOTALN: 3059 N_CACHED_NOTALN: 60 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:20:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:20:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:20:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:20:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:21:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5380 N_CACHED_ALN: 880 N_COMPUTED_NOTALN: 3619 N_CACHED_NOTALN: 121 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:21:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 3515 N_CACHED_ALN: 1565 N_COMPUTED_NOTALN: 4678 N_CACHED_NOTALN: 242 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:22:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 4997 N_CACHED_ALN: 812 N_COMPUTED_NOTALN: 4118 N_CACHED_NOTALN: 73 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:22:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5335 N_CACHED_ALN: 817 N_COMPUTED_NOTALN: 3768 N_CACHED_NOTALN: 80 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:25:\n",
      "\t Finished reads; N_TOT_READS: 14130 N_COMPUTED_ALN: 8990 N_CACHED_ALN: 1010 N_COMPUTED_NOTALN: 4057 N_CACHED_NOTALN: 73 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:25:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:25:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 5366 N_CACHED_ALN: 987 N_COMPUTED_NOTALN: 3533 N_CACHED_NOTALN: 114 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:26:\n",
      "\t Finished reads; N_TOT_READS: 17759 N_COMPUTED_ALN: 13550 N_CACHED_ALN: 1489 N_COMPUTED_NOTALN: 2618 N_CACHED_NOTALN: 102 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:26:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:26:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:27:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:27:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:27:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:28:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:29:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 13712 N_CACHED_ALN: 1917 N_COMPUTED_NOTALN: 4194 N_CACHED_NOTALN: 177 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Finished reads; N_TOT_READS: 16709 N_COMPUTED_ALN: 9244 N_CACHED_ALN: 1284 N_COMPUTED_NOTALN: 6042 N_CACHED_NOTALN: 139 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Finished reads; N_TOT_READS: 20880 N_COMPUTED_ALN: 14432 N_CACHED_ALN: 1943 N_COMPUTED_NOTALN: 4328 N_CACHED_NOTALN: 177 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:30:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:31:\n",
      "\t Finished reads; N_TOT_READS: 17949 N_COMPUTED_ALN: 10132 N_CACHED_ALN: 1347 N_COMPUTED_NOTALN: 6259 N_CACHED_NOTALN: 211 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:31:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:31:\n",
      "\t Finished reads; N_TOT_READS: 17958 N_COMPUTED_ALN: 7161 N_CACHED_ALN: 2314 N_COMPUTED_NOTALN: 8078 N_CACHED_NOTALN: 405 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:31:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:31:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:32:\n",
      "\t Finished reads; N_TOT_READS: 18166 N_COMPUTED_ALN: 9567 N_CACHED_ALN: 1335 N_COMPUTED_NOTALN: 7126 N_CACHED_NOTALN: 138 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:32:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:32:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:33:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:33:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:33:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:33:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:33:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:34:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:34:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:34:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:34:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:34:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 10151 N_CACHED_ALN: 2632 N_COMPUTED_NOTALN: 6911 N_CACHED_NOTALN: 306 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:35:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:47:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 15895 N_CACHED_ALN: 3594 N_COMPUTED_NOTALN: 10051 N_CACHED_NOTALN: 460 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:56:\n",
      "\t Finished reads; N_TOT_READS: 37831 N_COMPUTED_ALN: 21058 N_CACHED_ALN: 3998 N_COMPUTED_NOTALN: 12255 N_CACHED_NOTALN: 520 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:29:56:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:30:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:30:02:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:30:02:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:30:02:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:30:03:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:33:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:33:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:33:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:34:\n",
      "\t Finished CRISPResso batch #4 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:55:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:56:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:56:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:56:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:56:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:57:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:57:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:57:\n",
      "\t Finished CRISPResso batch #6 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:57:\n",
      "\t Finished CRISPResso batch #7 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:57:\n",
      "\t Finished CRISPResso batch #1 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:59:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:59:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:59:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:59:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:31:59:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:00:\n",
      "\t Finished CRISPResso batch #2 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:00:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:01:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:01:\n",
      "\t Finished CRISPResso batch #8 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:01:\n",
      "\t Finished CRISPResso batch #0 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:03:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:03:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:05:\n",
      "\t Finished CRISPResso batch #3 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:21:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:21:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:22:\n",
      "\t Finished CRISPResso batch #5 \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:22:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:22:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 05 Oct 2023 17:32:23:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "/PHShome/bfb12/.conda/envs/crispresso2_patch/lib/python2.7/site-packages/matplotlib/font_manager.py:1331: UserWarning: findfont: Font family [u'sans-serif'] not found. Falling back to DejaVu Sans\n",
      "  (prop.get_family(), self.defaultFamily[fontext]))\n",
      "INFO  @ Thu, 05 Oct 2023 17:33:30:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                             \n",
      "                                       _                                     \n",
      "                                      '  )                                   \n",
      "                                      .-'                                    \n",
      "                                     (____                                   \n",
      "                                  C)|     \\                                  \n",
      "                                    \\     /                                  \n",
      "                                     \\___/                                   \n",
      "\n",
      "CPU times: user 2min 7s, sys: 1min 57s, total: 4min 4s\n",
      "Wall time: 1h 59min 17s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "for _, validation_guide_row in CD19_guide_validation_sequences_df.iterrows():\n",
    "    validation_name = validation_guide_row[\"editor\"] + \"_sg\" + validation_guide_row[\"sgID\"]\n",
    "    crispresso_name = validation_guide_row[\"crispresso_name\"]\n",
    "    \n",
    "    validation_names_enumeration = [(index, name) for index, name in enumerate(names) if crispresso_name in name]\n",
    "    if len(validation_names_enumeration) > 0:\n",
    "        print(\"Processing \" + validation_name)\n",
    "        validate_results_output_dir = \"./20231005_v1_CRISPResso2Batch_Results_q30/\" + validation_name\n",
    "\n",
    "        premerged_results_dir = validate_results_output_dir + \"/premerged\"\n",
    "        unmerged_PE_results_dir = validate_results_output_dir + \"/unmerged_PE\"\n",
    "        unmerged_R1_results_dir = validate_results_output_dir + \"/unmerged_R1\"\n",
    "        unmerged_R2_results_dir = validate_results_output_dir + \"/unmerged_R2\"\n",
    "\n",
    "\n",
    "        os.makedirs(premerged_results_dir)\n",
    "        os.makedirs(unmerged_PE_results_dir)\n",
    "\n",
    "        os.makedirs(unmerged_R1_results_dir)\n",
    "        os.makedirs(unmerged_R2_results_dir)\n",
    "\n",
    "\n",
    "        premerged_settings_fn = \"Davide_CD19_enhancer_BE_screen_{}_premerged.settings\".format(validation_name)\n",
    "        unmerged_PE_settings_fn = 'Davide_CD19_enhancer_BE_screen_{}_unmerged_PE.settings'.format(validation_name)\n",
    "        unmerged_R1_settings_fn = 'Davide_CD19_enhancer_BE_screen_{}_unmerged_R1.settings'.format(validation_name)\n",
    "        unmerged_R2_settings_fn = 'Davide_CD19_enhancer_BE_screen_{}_unmerged_R2.settings'.format(validation_name)\n",
    "\n",
    "\n",
    "        #premerged\n",
    "        #ABE8e_pooled_premerged_middle\n",
    "        !CRISPRessoBatch -bs {settings_output_dir + \"/\" + premerged_settings_fn} -a {pooled_amplicon_sequence} \\\n",
    "        -an cd19 -q 30 \\\n",
    "        --exclude_bp_from_left 3 \\\n",
    "        --exclude_bp_from_right 3 --no_rerun -n {validation_name + \"_premerged\"} \\\n",
    "        --min_frequency_alleles_around_cut_to_plot 0.001 --max_rows_alleles_around_cut_to_plot 500 -p 20  \\\n",
    "        --plot_window_size 20 --base_editor_output -w 0 -bo {premerged_results_dir}\n",
    "\n",
    "\n",
    "        #ABE8e_pooled_unmerged_PE_middle\n",
    "        !CRISPRessoBatch -bs {settings_output_dir + \"/\" + unmerged_PE_settings_fn} -a {pooled_amplicon_sequence} \\\n",
    "        -an cd19 -q 30 \\\n",
    "        --exclude_bp_from_left 3 \\\n",
    "        --exclude_bp_from_right 3 --no_rerun -n {validation_name + \"_unmerged_PE\"} \\\n",
    "        --min_frequency_alleles_around_cut_to_plot 0.001 --max_rows_alleles_around_cut_to_plot 500 -p 64  \\\n",
    "        --plot_window_size 20 --base_editor_output -w 0 -bo {unmerged_PE_results_dir}\n",
    "\n",
    "\n",
    "        #ABE8e_pooled_unmerged_R1_middle\n",
    "        !CRISPRessoBatch -bs {settings_output_dir + \"/\" + unmerged_R1_settings_fn} -a {pooled_amplicon_sequence[:250]} \\\n",
    "        -an cd19 -q 30 \\\n",
    "        --exclude_bp_from_left 3 \\\n",
    "        --exclude_bp_from_right 3 --no_rerun -n {validation_name + \"_unmerged_R1\"} \\\n",
    "        --min_frequency_alleles_around_cut_to_plot 0.001 --max_rows_alleles_around_cut_to_plot 500 -p 64  \\\n",
    "        --plot_window_size 20 --base_editor_output -w 0 -bo {unmerged_R1_results_dir}\n",
    "\n",
    "        #ABE8e_pooled_unmerged_R2_middle\n",
    "        !CRISPRessoBatch -bs {settings_output_dir + \"/\" + unmerged_R2_settings_fn} -a {pooled_amplicon_sequence[-250:]} \\\n",
    "        -an cd19 -q 30 \\\n",
    "        --exclude_bp_from_left 3 \\\n",
    "        --exclude_bp_from_right 3 --no_rerun -n {validation_name + \"_unmerged_R2\"} \\\n",
    "        --min_frequency_alleles_around_cut_to_plot 0.001 --max_rows_alleles_around_cut_to_plot 500 -p 64  \\\n",
    "        --plot_window_size 20 --base_editor_output -w 0 -bo {unmerged_R2_results_dir}"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python [conda env:.conda-crispresso2_patch]",
   "language": "python",
   "name": "conda-env-.conda-crispresso2_patch-py"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.18"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
