{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "08cd525e",
   "metadata": {},
   "source": [
    "# Run CRISPResso2 on ABE8e sg218 Data"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "14763fbd",
   "metadata": {},
   "source": [
    "### Import Packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "e1e13dcd",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import os\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "46327532",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Install BioPython \"conda install conda-forge::biopython\" \n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "bdfb73ab",
   "metadata": {},
   "source": [
    "### Download Data\n",
    "\n",
    "As an example, download the sg218 endogenous data from GEO that was used in the CRISPR-CLEAR manuscript: [GSE278581](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE278581)\n",
    "\n",
    "\n",
    "\n",
    "- GSM8549747\tABE8e, unedited, pooled, endogenous\n",
    "- GSM8549748\tABE8e, replicate 1, presort, sg218\n",
    "- GSM8549749\tABE8e, replicate 1, CD19 positive, sg218\n",
    "- GSM8549750\tABE8e, replicate 1, CD19 negative, sg218\n",
    "- GSM8549751\tABE8e, replicate 2, presort, sg218\n",
    "- GSM8549752\tABE8e, replicate 2, CD19 positive, sg218\n",
    "- GSM8549753\tABE8e, replicate 2, CD19 negative, sg218\n",
    "- GSM8549754\tABE8e, replicate 3, presort, sg218\n",
    "- GSM8549755\tABE8e, replicate 3, CD19 positive, sg218\n",
    "- GSM8549756\tABE8e, replicate 3, CD19 negative, sg218\n",
    "\n",
    "You can alternatively download the data from [Zenodo](https://doi.org/10.5281/zenodo.13737880) which may be easier: Unzip the file CRISPR-CLEAR-data/data/raw_FASTQs/endogenous_sequencing/sg218_fastqs.zip\n",
    "\n"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "ef5fee43",
   "metadata": {},
   "source": [
    "### Prepare sample sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "1497f13a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Set the directory of the downloaded data, replace with your directory\n",
    "unmerged_all_dir = \"./sg218_fastqs/\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "26835e9c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "./sg219_fastqs/\n"
     ]
    }
   ],
   "source": [
    "# OPTIONAL: In all filenames, need to replace '#' character with '_' to allow correct parsing of filename in CRISPResso2\n",
    "def replicate_hashtag(directory_path):\n",
    "    print(directory_path)\n",
    "    file_list = os.listdir(directory_path)\n",
    "    for filename in file_list:\n",
    "        # Check if the '#' character is present in the filename\n",
    "        if '#' in filename:\n",
    "            # Replace '#' with '_'\n",
    "            new_filename = filename.replace('#', '_')\n",
    "\n",
    "            # Create the full old and new file paths\n",
    "            old_file_path = os.path.join(directory_path, filename)\n",
    "            new_file_path = os.path.join(directory_path, new_filename)\n",
    "\n",
    "            # Rename the file\n",
    "            os.rename(old_file_path, new_file_path)\n",
    "            print('Renamed: {} to {}'.format(filename, new_filename))\n",
    "            \n",
    "replicate_hashtag(unmerged_all_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "472dfcbc",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get filenames from directory\n",
    "unmerged_all_fns = os.listdir(unmerged_all_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "e509ecb4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get the R1 and R2 files\n",
    "r1_unmerged_fastq = [fn for fn in unmerged_all_fns if (\"R1.fastq\" in fn) and (\"PAX5sg\" in fn)]\n",
    "r2_unmerged_fastq = [fn for fn in unmerged_all_fns if (\"R2.fastq\" in fn) and (\"PAX5sg\" in fn)]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "979fa936",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Sort the R1 and R2 files so that corresponding files are in same list index\n",
    "r1_unmerged_fastq = sorted(r1_unmerged_fastq)\n",
    "r2_unmerged_fastq = sorted(r2_unmerged_fastq)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "5ccf44eb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_S139294.R1.fastq.gz',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_S139299.R1.fastq.gz',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_S139300.R1.fastq.gz',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S139295.R1.fastq.gz',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S139296.R1.fastq.gz',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S139301.R1.fastq.gz',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_1_S139292.R1.fastq.gz',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_2_S139297.R1.fastq.gz',\n",
       " 'BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg_3_S139298.R1.fastq.gz']"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "r1_unmerged_fastq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "7abc61aa",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Prepare the sample metadata values\n",
    "editors_list = [\"ABE8e\"] * 9\n",
    "population_list = [\"high\"] * 3 + [\"low\"] * 3 + [\"presort\"] * 3\n",
    "replicate_list = [1,2,3] * 3\n",
    "experiment_list = [\"ABE8e_PAX5sg\"] * 9"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "88e673be",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Create dataframe from metadata files and save sample sheet\n",
    "sample_sheet_df = pd.DataFrame({\n",
    "    \"editors\": editors_list,\n",
    "    \"population\": population_list,\n",
    "    \"replicate\": replicate_list,\n",
    "    \"experiment\": experiment_list,\n",
    "    \"R1_FASTQ\": r1_unmerged_fastq,\n",
    "    \"R2_FASTQ\": r2_unmerged_fastq\n",
    "})\n",
    "\n",
    "sample_sheet_df = sample_sheet_df.loc[:, [\"editors\", \"population\", \"replicate\", \"experiment\", \"R1_FASTQ\", \"R2_FASTQ\"]]\n",
    "sample_sheet_df.to_csv(\"20230922_sample_sheet_2.tsv\", sep=\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "516cacd5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>editors</th>\n",
       "      <th>population</th>\n",
       "      <th>replicate</th>\n",
       "      <th>experiment</th>\n",
       "      <th>R1_FASTQ</th>\n",
       "      <th>R2_FASTQ</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>high</td>\n",
       "      <td>1</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>high</td>\n",
       "      <td>2</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>high</td>\n",
       "      <td>3</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>low</td>\n",
       "      <td>1</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>low</td>\n",
       "      <td>2</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>low</td>\n",
       "      <td>3</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>presort</td>\n",
       "      <td>1</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>presort</td>\n",
       "      <td>2</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>presort</td>\n",
       "      <td>3</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "      <td>BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  editors population  replicate    experiment  \\\n",
       "0   ABE8e       high          1  ABE8e_PAX5sg   \n",
       "1   ABE8e       high          2  ABE8e_PAX5sg   \n",
       "2   ABE8e       high          3  ABE8e_PAX5sg   \n",
       "3   ABE8e        low          1  ABE8e_PAX5sg   \n",
       "4   ABE8e        low          2  ABE8e_PAX5sg   \n",
       "5   ABE8e        low          3  ABE8e_PAX5sg   \n",
       "6   ABE8e    presort          1  ABE8e_PAX5sg   \n",
       "7   ABE8e    presort          2  ABE8e_PAX5sg   \n",
       "8   ABE8e    presort          3  ABE8e_PAX5sg   \n",
       "\n",
       "                                            R1_FASTQ  \\\n",
       "0  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_...   \n",
       "1  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_...   \n",
       "2  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_...   \n",
       "3  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S...   \n",
       "4  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S...   \n",
       "5  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S...   \n",
       "6  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...   \n",
       "7  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...   \n",
       "8  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...   \n",
       "\n",
       "                                            R2_FASTQ  \n",
       "0  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_1_...  \n",
       "1  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_2_...  \n",
       "2  BSF_1488_000000000-KJJ4P_1_CD19_high_PAX5sg_3_...  \n",
       "3  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_1_S...  \n",
       "4  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_2_S...  \n",
       "5  BSF_1488_000000000-KJJ4P_1_CD19_low_PAX5sg_3_S...  \n",
       "6  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...  \n",
       "7  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...  \n",
       "8  BSF_1488_000000000-KJJ4P_1_CD19_presort_PAX5sg...  "
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sample_sheet_df"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "672b0ddb",
   "metadata": {},
   "source": [
    "### Read in guide sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "5c19094b",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Read in the guide library file (contain in the repository), replace with filepath:\n",
    "guide_library_fn = \"CD19_sgRNA_count_libraries_DS.txt\"\n",
    "pooled_guide_sequence_df = pd.read_table(guide_library_fn)\n",
    "pooled_guide_sequence_df.columns = [col.strip() for col in pooled_guide_sequence_df.columns]\n",
    "pooled_guide_sequences = [str(val) for val in pooled_guide_sequence_df[\"sgRNA\"].values.tolist()]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "5380bcf4",
   "metadata": {},
   "outputs": [],
   "source": [
    "editor = [\"ABE8e\"]\n",
    "sgID = [\"218\"]\n",
    "guide_sequence = [\"ACAGGAAGTCACAGCCTGGT\"]\n",
    "\n",
    "validation_origin = [\"ABE8e_v1_endogenous\"]\n",
    "\n",
    "\n",
    "CD19_guide_validation_sequences_df = pd.DataFrame({\n",
    "    \"editor\": editor,\n",
    "    \"sgID\": sgID,\n",
    "    \"screen_origin\": validation_origin,\n",
    "    \"guide_sequence\": guide_sequence\n",
    "})\n",
    "\n",
    "CD19_guide_validation_sequences_df = CD19_guide_validation_sequences_df.loc[:, [\"editor\", \"sgID\", \"screen_origin\", \"guide_sequence\"]]\n",
    "\n",
    "# Add the field corresponding to the CRISPRessoBatch sample name:\n",
    "CD19_guide_validation_sequences_df[\"crispresso_name\"] = CD19_guide_validation_sequences_df[\"editor\"] + \"_\" + CD19_guide_validation_sequences_df[\"sgID\"]\n",
    "CD19_guide_validation_sequences_df[\"crispresso_name\"][0] = \"ABE8e_PAX5sg\"\n",
    "CD19_guide_validation_sequences_df.to_csv(\"20230922_CD19_guide_validation_sequences_2.tsv\", sep=\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "b74620a8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>editor</th>\n",
       "      <th>sgID</th>\n",
       "      <th>screen_origin</th>\n",
       "      <th>guide_sequence</th>\n",
       "      <th>crispresso_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ABE8e</td>\n",
       "      <td>218</td>\n",
       "      <td>ABE8e_v1_endogenous</td>\n",
       "      <td>ACAGGAAGTCACAGCCTGGT</td>\n",
       "      <td>ABE8e_PAX5sg</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  editor sgID        screen_origin        guide_sequence crispresso_name\n",
       "0  ABE8e  218  ABE8e_v1_endogenous  ACAGGAAGTCACAGCCTGGT    ABE8e_PAX5sg"
      ]
     },
     "execution_count": 44,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "CD19_guide_validation_sequences_df"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "d8013102",
   "metadata": {},
   "source": [
    "## Run CRISPRessoBatch"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "0f32d99b",
   "metadata": {},
   "source": [
    "Prepare amplicon sequence and guide list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "a5a82406",
   "metadata": {},
   "outputs": [],
   "source": [
    "pooled_amplicon_sequence = Seq(\"ACATGCTCTAGTGAAAGCCAGTCTGGGCAGCTGGGTAGCTAATGAGGGGATTAGAGAGATTTTGTTGAATGAAAGGCAGATTGAGTCCTGCTACTCGCCCCCTTCATTCCCCTTCATTCATGCCTCATTCTTCCGCCTCCCAGCCGCCTCAACTGGCCAAAGGGAAGTGGAGGCCCTGCCACCTGTAGGGAGGGTCCCCTGGGGCTTGCCCACAGCAAACAGGAAGTCACAGCCTGGTGAGATGGGCCTGGGAATCAGCCACTGAGAAAGTGGGTCTCTTGGGTCCCTGAATTCTTTTTCTGAGTCCCTGCAGCAGTGAAAAAGACACAGAGGCACATAGAGAGTG\")\n",
    "\n",
    "pooled_amplicon_sequence_revcomp = pooled_amplicon_sequence.reverse_complement() \n",
    "pooled_amplicon_sequence = str(pooled_amplicon_sequence)\n",
    "pooled_amplicon_sequence_revcomp = str(pooled_amplicon_sequence_revcomp)\n",
    "\n",
    "guide_sequences_present = [seq for seq in pooled_guide_sequences if ((seq in pooled_amplicon_sequence) or (seq in pooled_amplicon_sequence_revcomp))]\n",
    "\n",
    "pooled_middle_guide = \"GAGGCCCTGCCACCTGTAGG\" # Set guide in middle of amplicon for visualization\n",
    "r1_middle_guide = \"ATTCATGCCTCATTCTTCCG\"\n",
    "r2_middle_guide = \"ACAGCAAACAGGAAGTCACA\"\n",
    "# TODO (20230925): Might need another middle guide for R1 and R2 individually"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "dfc802d8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['CATGCTCTAGTGAAAGCCAG',\n",
       " 'GCTCTAGTGAAAGCCAGTCT',\n",
       " 'CTAGTGAAAGCCAGTCTGGG',\n",
       " 'GTGAAAGCCAGTCTGGGCAG',\n",
       " 'AAAGCCAGTCTGGGCAGCTG',\n",
       " 'GCCAGTCTGGGCAGCTGGGT',\n",
       " 'AGTCTGGGCAGCTGGGTAGC',\n",
       " 'CTGGGCAGCTGGGTAGCTAA',\n",
       " 'GGCAGCTGGGTAGCTAATGA',\n",
       " 'AGCTGGGTAGCTAATGAGGG',\n",
       " 'TGGGTAGCTAATGAGGGGAT',\n",
       " 'GCTAATGAGGGGATTAGAGA',\n",
       " 'AATGAGGGGATTAGAGAGAT',\n",
       " 'GAGGGGATTAGAGAGATTTT',\n",
       " 'GGGATTAGAGAGATTTTGTT',\n",
       " 'ATTAGAGAGATTTTGTTGAA',\n",
       " 'GAGATTTTGTTGAATGAAAG',\n",
       " 'ATTTTGTTGAATGAAAGGCA',\n",
       " 'TTGTTGAATGAAAGGCAGAT',\n",
       " 'TTGAATGAAAGGCAGATTGA',\n",
       " 'AATGAAAGGCAGATTGAGTC',\n",
       " 'GAAAGGCAGATTGAGTCCTG',\n",
       " 'AGGCAGATTGAGTCCTGCTA',\n",
       " 'CAGATTGAGTCCTGCTACTC',\n",
       " 'ATTGAGTCCTGCTACTCGCC',\n",
       " 'GAGTCCTGCTACTCGCCCCC',\n",
       " 'TCCTGCTACTCGCCCCCTTC',\n",
       " 'TGCTACTCGCCCCCTTCATT',\n",
       " 'TACTCGCCCCCTTCATTCCC',\n",
       " 'TCGCCCCCTTCATTCCCCTT',\n",
       " 'CCCCCTTCATTCCCCTTCAT',\n",
       " 'CCTTCATTCCCCTTCATTCA',\n",
       " 'TCATTCCCCTTCATTCATGC',\n",
       " 'TTCCCCTTCATTCATGCCTC',\n",
       " 'CCCTTCATTCATGCCTCATT',\n",
       " 'TTCATTCATGCCTCATTCTT',\n",
       " 'ATTCATGCCTCATTCTTCCG',\n",
       " 'CATGCCTCATTCTTCCGCCT',\n",
       " 'GCCTCATTCTTCCGCCTCCC',\n",
       " 'TCATTCTTCCGCCTCCCAGC',\n",
       " 'TTCTTCCGCCTCCCAGCCGC',\n",
       " 'TTCCGCCTCCCAGCCGCCTC',\n",
       " 'CGCCTCCCAGCCGCCTCAAC',\n",
       " 'CTCCCAGCCGCCTCAACTGG',\n",
       " 'CCAGCCGCCTCAACTGGCCA',\n",
       " 'GCCGCCTCAACTGGCCAAAG',\n",
       " 'TCAACTGGCCAAAGGGAAGT',\n",
       " 'ACTGGCCAAAGGGAAGTGGA',\n",
       " 'GGCCAAAGGGAAGTGGAGGC',\n",
       " 'CAAAGGGAAGTGGAGGCCCT',\n",
       " 'AGGGAAGTGGAGGCCCTGCC',\n",
       " 'GAAGTGGAGGCCCTGCCACC',\n",
       " 'GTGGAGGCCCTGCCACCTGT',\n",
       " 'GAGGCCCTGCCACCTGTAGG',\n",
       " 'CTGCCACCTGTAGGGAGGGT',\n",
       " 'CCACCTGTAGGGAGGGTCCC',\n",
       " 'CCTGTAGGGAGGGTCCCCTG',\n",
       " 'GTAGGGAGGGTCCCCTGGGG',\n",
       " 'GGGAGGGTCCCCTGGGGCTT',\n",
       " 'AGGGTCCCCTGGGGCTTGCC',\n",
       " 'GTCCCCTGGGGCTTGCCCAC',\n",
       " 'CCCTGGGGCTTGCCCACAGC',\n",
       " 'TGGGGCTTGCCCACAGCAAA',\n",
       " 'GGCTTGCCCACAGCAAACAG',\n",
       " 'CCCACAGCAAACAGGAAGTC',\n",
       " 'ACAGCAAACAGGAAGTCACA',\n",
       " 'AACAGGAAGTCACAGCCTGG',\n",
       " 'AGGAAGTCACAGCCTGGTGA',\n",
       " 'AAGTCACAGCCTGGTGAGAT',\n",
       " 'TCACAGCCTGGTGAGATGGG',\n",
       " 'CAGCCTGGTGAGATGGGCCT',\n",
       " 'CCTGGTGAGATGGGCCTGGG',\n",
       " 'GGTGAGATGGGCCTGGGAAT',\n",
       " 'ATGGGCCTGGGAATCAGCCA',\n",
       " 'GGCCTGGGAATCAGCCACTG',\n",
       " 'CTGGGAATCAGCCACTGAGA',\n",
       " 'ATCAGCCACTGAGAAAGTGG',\n",
       " 'AGCCACTGAGAAAGTGGGTC',\n",
       " 'CACTGAGAAAGTGGGTCTCT',\n",
       " 'TGAGAAAGTGGGTCTCTTGG',\n",
       " 'GAAAGTGGGTCTCTTGGGTC',\n",
       " 'GGGTCTCTTGGGTCCCTGAA',\n",
       " 'TCTCTTGGGTCCCTGAATTC',\n",
       " 'CTTGGGTCCCTGAATTCTTT',\n",
       " 'GGGTCCCTGAATTCTTTTTC',\n",
       " 'TCCCTGAATTCTTTTTCTGA',\n",
       " 'CTGAATTCTTTTTCTGAGTC',\n",
       " 'AATTCTTTTTCTGAGTCCCT',\n",
       " 'TCTTTTTCTGAGTCCCTGCA',\n",
       " 'GAGTCCCTGCAGCAGTGAAA',\n",
       " 'TCCCTGCAGCAGTGAAAAAG',\n",
       " 'CTGCAGCAGTGAAAAAGACA',\n",
       " 'CAGTGAAAAAGACACAGAGG',\n",
       " 'TGAAAAAGACACAGAGGCAC',\n",
       " 'AAAAGACACAGAGGCACATA',\n",
       " 'AGACACAGAGGCACATAGAG',\n",
       " 'CACAGAGGCACATAGAGAGT',\n",
       " 'CTGGCTTTCACTAGAGCATG',\n",
       " 'AGACTGGCTTTCACTAGAGC',\n",
       " 'CCCAGACTGGCTTTCACTAG',\n",
       " 'CTGCCCAGACTGGCTTTCAC',\n",
       " 'ACCCAGCTGCCCAGACTGGC',\n",
       " 'TCATTAGCTACCCAGCTGCC',\n",
       " 'CCCTCATTAGCTACCCAGCT',\n",
       " 'ATCCCCTCATTAGCTACCCA',\n",
       " 'CTAATCCCCTCATTAGCTAC',\n",
       " 'TCTCTAATCCCCTCATTAGC',\n",
       " 'ATCTCTCTAATCCCCTCATT',\n",
       " 'AAAATCTCTCTAATCCCCTC',\n",
       " 'AACAAAATCTCTCTAATCCC',\n",
       " 'TCATTCAACAAAATCTCTCT',\n",
       " 'CTTTCATTCAACAAAATCTC',\n",
       " 'TGCCTTTCATTCAACAAAAT',\n",
       " 'ATCTGCCTTTCATTCAACAA',\n",
       " 'TCAATCTGCCTTTCATTCAA',\n",
       " 'GACTCAATCTGCCTTTCATT',\n",
       " 'CAGGACTCAATCTGCCTTTC',\n",
       " 'GAGTAGCAGGACTCAATCTG',\n",
       " 'GGCGAGTAGCAGGACTCAAT',\n",
       " 'GGGGGCGAGTAGCAGGACTC',\n",
       " 'AATGAAGGGGGCGAGTAGCA',\n",
       " 'AAGGGGAATGAAGGGGGCGA',\n",
       " 'ATGAAGGGGAATGAAGGGGG',\n",
       " 'TGAATGAAGGGGAATGAAGG',\n",
       " 'GCATGAATGAAGGGGAATGA',\n",
       " 'GAGGCATGAATGAAGGGGAA',\n",
       " 'AATGAGGCATGAATGAAGGG',\n",
       " 'AAGAATGAGGCATGAATGAA',\n",
       " 'CGGAAGAATGAGGCATGAAT',\n",
       " 'AGGCGGAAGAATGAGGCATG',\n",
       " 'GGGAGGCGGAAGAATGAGGC',\n",
       " 'GCTGGGAGGCGGAAGAATGA',\n",
       " 'GCGGCTGGGAGGCGGAAGAA',\n",
       " 'GAGGCGGCTGGGAGGCGGAA',\n",
       " 'GTTGAGGCGGCTGGGAGGCG',\n",
       " 'CCAGTTGAGGCGGCTGGGAG',\n",
       " 'TGGCCAGTTGAGGCGGCTGG',\n",
       " 'CTTTGGCCAGTTGAGGCGGC',\n",
       " 'ACTTCCCTTTGGCCAGTTGA',\n",
       " 'TCCACTTCCCTTTGGCCAGT',\n",
       " 'GCCTCCACTTCCCTTTGGCC',\n",
       " 'AGGGCCTCCACTTCCCTTTG',\n",
       " 'GGCAGGGCCTCCACTTCCCT',\n",
       " 'GGTGGCAGGGCCTCCACTTC',\n",
       " 'ACAGGTGGCAGGGCCTCCAC',\n",
       " 'CCTACAGGTGGCAGGGCCTC',\n",
       " 'CTCCCTACAGGTGGCAGGGC',\n",
       " 'ACCCTCCCTACAGGTGGCAG',\n",
       " 'GGGACCCTCCCTACAGGTGG',\n",
       " 'CAGGGGACCCTCCCTACAGG',\n",
       " 'CCCCAGGGGACCCTCCCTAC',\n",
       " 'AAGCCCCAGGGGACCCTCCC',\n",
       " 'GGCAAGCCCCAGGGGACCCT',\n",
       " 'GCTGTGGGCAAGCCCCAGGG',\n",
       " 'TTTGCTGTGGGCAAGCCCCA',\n",
       " 'CTGTTTGCTGTGGGCAAGCC',\n",
       " 'TTCCTGTTTGCTGTGGGCAA',\n",
       " 'GACTTCCTGTTTGCTGTGGG',\n",
       " 'TGTGACTTCCTGTTTGCTGT',\n",
       " 'GGCTGTGACTTCCTGTTTGC',\n",
       " 'CCAGGCTGTGACTTCCTGTT',\n",
       " 'TCACCAGGCTGTGACTTCCT',\n",
       " 'ATCTCACCAGGCTGTGACTT',\n",
       " 'CCCATCTCACCAGGCTGTGA',\n",
       " 'AGGCCCATCTCACCAGGCTG',\n",
       " 'CCCAGGCCCATCTCACCAGG',\n",
       " 'ATTCCCAGGCCCATCTCACC',\n",
       " 'CTGATTCCCAGGCCCATCTC',\n",
       " 'TGGCTGATTCCCAGGCCCAT',\n",
       " 'TCTCAGTGGCTGATTCCCAG',\n",
       " 'CCACTTTCTCAGTGGCTGAT',\n",
       " 'GACCCACTTTCTCAGTGGCT',\n",
       " 'AGAGACCCACTTTCTCAGTG',\n",
       " 'CCAAGAGACCCACTTTCTCA',\n",
       " 'GACCCAAGAGACCCACTTTC',\n",
       " 'AGGGACCCAAGAGACCCACT',\n",
       " 'TTCAGGGACCCAAGAGACCC',\n",
       " 'GAATTCAGGGACCCAAGAGA',\n",
       " 'AAAGAATTCAGGGACCCAAG',\n",
       " 'GAAAAAGAATTCAGGGACCC',\n",
       " 'TCAGAAAAAGAATTCAGGGA',\n",
       " 'GACTCAGAAAAAGAATTCAG',\n",
       " 'AGGGACTCAGAAAAAGAATT',\n",
       " 'TGCTGCAGGGACTCAGAAAA',\n",
       " 'CACTGCTGCAGGGACTCAGA',\n",
       " 'TTTCACTGCTGCAGGGACTC',\n",
       " 'CTTTTTCACTGCTGCAGGGA',\n",
       " 'TGTCTTTTTCACTGCTGCAG',\n",
       " 'CTGTGTCTTTTTCACTGCTG',\n",
       " 'CCTCTGTGTCTTTTTCACTG',\n",
       " 'GTGCCTCTGTGTCTTTTTCA',\n",
       " 'TATGTGCCTCTGTGTCTTTT']"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "guide_sequences_present"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "97e6a8c7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 192 guides present in amplicon sequence out of 206\n"
     ]
    }
   ],
   "source": [
    "print(\"There are {} guides present in amplicon sequence out of {}\".format(len(guide_sequences_present), len(pooled_guide_sequences)))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "6cf55400",
   "metadata": {},
   "source": [
    "Prepare filenames"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "f5e427c8",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Prepare R1 and R2 FASTQ files for CRISPResso2Batch settings file\n",
    "filenames_single_end_dir = \"./sg218_fastqs/\"\n",
    "\n",
    "names = sample_sheet_df[\"experiment\"].str.cat(sample_sheet_df[[\"population\", \"replicate\"]].astype(str), sep=\"_\")\n",
    "sample_sheet_df[\"names\"] = names\n",
    "filenames_R1 = filenames_single_end_dir + sample_sheet_df[\"R1_FASTQ\"] # TODO, prefix with directory\n",
    "filenames_R2 = filenames_single_end_dir + sample_sheet_df[\"R2_FASTQ\"] # TODO, prefix with directory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "373c980f",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Create directory for all the different settings file\n",
    "settings_output_dir = \"./20241004_CRISPResso2Batch_Settings\"\n",
    "os.makedirs(settings_output_dir)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "1469cac4",
   "metadata": {},
   "source": [
    "Create settings files for pooled screen"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "aa5bf78e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Prepare sample names that will be batched\n",
    "pooled_names_enumeration = [(index, name) for index, name in enumerate(names) if (\"pooled\" in name) or (\"mock_presort_1\" in name)]\n",
    "display(pooled_names_enumeration)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "e76bb8a6",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0       ABE8e_PAX5sg_high_1\n",
       "1       ABE8e_PAX5sg_high_2\n",
       "2       ABE8e_PAX5sg_high_3\n",
       "3        ABE8e_PAX5sg_low_1\n",
       "4        ABE8e_PAX5sg_low_2\n",
       "5        ABE8e_PAX5sg_low_3\n",
       "6    ABE8e_PAX5sg_presort_1\n",
       "7    ABE8e_PAX5sg_presort_2\n",
       "8    ABE8e_PAX5sg_presort_3\n",
       "Name: experiment, dtype: object"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "names"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "d5a945ff",
   "metadata": {},
   "source": [
    "Create settings files for validation guides"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "50eba465",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ABE8e_sg218\n",
      "[(0, 'ABE8e_PAX5sg_high_1'), (1, 'ABE8e_PAX5sg_high_2'), (2, 'ABE8e_PAX5sg_high_3'), (3, 'ABE8e_PAX5sg_low_1'), (4, 'ABE8e_PAX5sg_low_2'), (5, 'ABE8e_PAX5sg_low_3'), (6, 'ABE8e_PAX5sg_presort_1'), (7, 'ABE8e_PAX5sg_presort_2'), (8, 'ABE8e_PAX5sg_presort_3')]\n"
     ]
    }
   ],
   "source": [
    "global skipped_ids, available_ids\n",
    "skipped_ids = []\n",
    "available_ids = []\n",
    "def generate_validation_guide_crispresso_settings(validation_guide_row):\n",
    "    validation_name = validation_guide_row[\"editor\"] + \"_sg\" + validation_guide_row[\"sgID\"]\n",
    "    crispresso_name = validation_guide_row[\"crispresso_name\"]\n",
    "    print(validation_name)\n",
    "    \n",
    "    unmerged_PE_settings_fn = 'Davide_CD19_enhancer_BE_screen_{}_unmerged_PE.settings'.format(validation_name)\n",
    "    unmerged_R1_settings_fn = 'Davide_CD19_enhancer_BE_screen_{}_unmerged_R1.settings'.format(validation_name)\n",
    "    unmerged_R2_settings_fn = 'Davide_CD19_enhancer_BE_screen_{}_unmerged_R2.settings'.format(validation_name)\n",
    "    \n",
    "    validation_names_enumeration = [(index, name) for index, name in enumerate(names) if crispresso_name in name]\n",
    "    print(validation_names_enumeration)\n",
    "    \n",
    "    if len(validation_names_enumeration) == 0:\n",
    "        skipped_ids.append(validation_name)\n",
    "    else:\n",
    "        available_ids.append(validation_name)\n",
    "\n",
    "        # Un-merged settings\n",
    "        with open(settings_output_dir + \"/\" + unmerged_PE_settings_fn, 'w') as out_handle:\n",
    "            out_handle.write('name\\tfastq_r1\\tfastq_r2\\tguide_seq\\n')\n",
    "            for sample_index, _ in validation_names_enumeration:\n",
    "                out_handle.write('sample_%s\\t%s\\t%s\\t%s\\n' % (names[sample_index], filenames_R1[sample_index], filenames_R2[sample_index], validation_guide_row[\"guide_sequence\"]))\n",
    "\n",
    "        # R1 settings\n",
    "        with open(settings_output_dir + \"/\" + unmerged_R1_settings_fn, 'w') as out_handle:\n",
    "            out_handle.write('name\\tfastq_r1\\tguide_seq\\n')\n",
    "            for sample_index, _ in validation_names_enumeration:\n",
    "                out_handle.write('sample_%s\\t%s\\t%s\\n' % (names[sample_index], filenames_R1[sample_index], validation_guide_row[\"guide_sequence\"]))\n",
    "\n",
    "        # R2 settings\n",
    "        with open(settings_output_dir + \"/\" + unmerged_R2_settings_fn, 'w') as out_handle:\n",
    "            out_handle.write('name\\tfastq_r1\\tguide_seq\\n')\n",
    "            for sample_index, _ in validation_names_enumeration:\n",
    "                out_handle.write('sample_%s\\t%s\\t%s\\n' % (names[sample_index], filenames_R2[sample_index], validation_guide_row[\"guide_sequence\"]))\n",
    "\n",
    "for _, row in CD19_guide_validation_sequences_df.iterrows():\n",
    "    generate_validation_guide_crispresso_settings(row)            "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "26d126c0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "skipped_ids"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "dddfd762",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['ABE8e_sg218']"
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "available_ids"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "48cffa12",
   "metadata": {},
   "source": [
    "Run CRISPResso2Batch"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "a88fbe54",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing ABE8e_sg218\n",
      "\n",
      "                             ~~~CRISPRessoBatch~~~                              \n",
      "       -Analysis of CRISPR/Cas9 outcomes from batch deep sequencing data-       \n",
      "                                                                                \n",
      "                _                                               _               \n",
      "               '  )                                            '  )             \n",
      "               .-'             _________________               .-'              \n",
      "              (____           | __    ___ __    |             (____             \n",
      "           C)|     \\          ||__) /\\ | /  |__||          C)|     \\            \n",
      "             \\     /          ||__)/--\\| \\__|  ||            \\     /            \n",
      "              \\___/           |_________________|             \\___/             \n",
      "\n",
      "                           [CRISPResso version 2.3.1]                           \n",
      "[Note that as of version 2.3.0 FLASh and Trimmomatic have been replaced by fastp for read merging and trimming. Accordingly, the --flash_command and --trimmomatic_command parameters have been replaced with --fastp_command. Also, --trimmomatic_options_string has been replaced with --fastp_options_string.\n",
      "\n",
      "Also in version 2.3.2, when running CRISPRessoPooled in mixed-mode (amplicon file and genome are provided) the default behavior will be as if the --demultiplex_only_at_amplicons parameter is provided. This change means that reads and amplicons do not need to align to the exact locations.]\n",
      "       [For support contact k.clement@utah.edu or support@edilytics.com]        \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:46:\n",
      "\t Creating Folder /Users/zp083/Desktop/CRISPRMillipedeTest/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg218/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg218_unmerged_PE \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:46:\n",
      "\t Running CRISPResso with 64 processes \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.3.1]                           \n",
      "[Note that as of version 2.3.0 FLASh and Trimmomatic have been replaced by fastp for read merging and trimming. Accordingly, the --flash_command and --trimmomatic_command parameters have been replaced with --fastp_command. Also, --trimmomatic_options_string has been replaced with --fastp_options_string.\n",
      "\n",
      "Also in version 2.3.2, when running CRISPRessoPooled in mixed-mode (amplicon file and genome are provided) the default behavior will be as if the --demultiplex_only_at_amplicons parameter is provided. This change means that reads and amplicons do not need to align to the exact locations.]\n",
      "       [For support contact k.clement@utah.edu or support@edilytics.com]        \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Creating Folder /Users/zp083/Desktop/CRISPRMillipedeTest/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg218/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg218_unmerged_PE/CRISPResso_on_sample_ABE8e_PAX5sg_presort_3 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Computing quantification windows \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t CRISPRessoPro not installed \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Added 0 guides with flexible matching\n",
      "\tOriginal flexiguides: ['None']\n",
      "\tFound guides: []\n",
      "\tMismatch locations: [] \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.3.1]                           \n",
      "[Note that as of version 2.3.0 FLASh and Trimmomatic have been replaced by fastp for read merging and trimming. Accordingly, the --flash_command and --trimmomatic_command parameters have been replaced with --fastp_command. Also, --trimmomatic_options_string has been replaced with --fastp_options_string.\n",
      "\n",
      "Also in version 2.3.2, when running CRISPRessoPooled in mixed-mode (amplicon file and genome are provided) the default behavior will be as if the --demultiplex_only_at_amplicons parameter is provided. This change means that reads and amplicons do not need to align to the exact locations.]\n",
      "       [For support contact k.clement@utah.edu or support@edilytics.com]        \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Creating Folder /Users/zp083/Desktop/CRISPRMillipedeTest/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg218/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg218_unmerged_PE/CRISPResso_on_sample_ABE8e_PAX5sg_low_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.3.1]                           \n",
      "[Note that as of version 2.3.0 FLASh and Trimmomatic have been replaced by fastp for read merging and trimming. Accordingly, the --flash_command and --trimmomatic_command parameters have been replaced with --fastp_command. Also, --trimmomatic_options_string has been replaced with --fastp_options_string.\n",
      "\n",
      "Also in version 2.3.2, when running CRISPRessoPooled in mixed-mode (amplicon file and genome are provided) the default behavior will be as if the --demultiplex_only_at_amplicons parameter is provided. This change means that reads and amplicons do not need to align to the exact locations.]\n",
      "       [For support contact k.clement@utah.edu or support@edilytics.com]        \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Computing quantification windows \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t CRISPRessoPro not installed \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Added 0 guides with flexible matching\n",
      "\tOriginal flexiguides: ['None']\n",
      "\tFound guides: []\n",
      "\tMismatch locations: [] \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Creating Folder /Users/zp083/Desktop/CRISPRMillipedeTest/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg218/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg218_unmerged_PE/CRISPResso_on_sample_ABE8e_PAX5sg_high_1 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Computing quantification windows \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t CRISPRessoPro not installed \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Added 0 guides with flexible matching\n",
      "\tOriginal flexiguides: ['None']\n",
      "\tFound guides: []\n",
      "\tMismatch locations: [] \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.3.1]                           \n",
      "[Note that as of version 2.3.0 FLASh and Trimmomatic have been replaced by fastp for read merging and trimming. Accordingly, the --flash_command and --trimmomatic_command parameters have been replaced with --fastp_command. Also, --trimmomatic_options_string has been replaced with --fastp_options_string.\n",
      "\n",
      "Also in version 2.3.2, when running CRISPRessoPooled in mixed-mode (amplicon file and genome are provided) the default behavior will be as if the --demultiplex_only_at_amplicons parameter is provided. This change means that reads and amplicons do not need to align to the exact locations.]\n",
      "       [For support contact k.clement@utah.edu or support@edilytics.com]        \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Creating Folder /Users/zp083/Desktop/CRISPRMillipedeTest/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg218/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg218_unmerged_PE/CRISPResso_on_sample_ABE8e_PAX5sg_presort_1 \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.3.1]                           \n",
      "[Note that as of version 2.3.0 FLASh and Trimmomatic have been replaced by fastp for read merging and trimming. Accordingly, the --flash_command and --trimmomatic_command parameters have been replaced with --fastp_command. Also, --trimmomatic_options_string has been replaced with --fastp_options_string.\n",
      "\n",
      "Also in version 2.3.2, when running CRISPRessoPooled in mixed-mode (amplicon file and genome are provided) the default behavior will be as if the --demultiplex_only_at_amplicons parameter is provided. This change means that reads and amplicons do not need to align to the exact locations.]\n",
      "       [For support contact k.clement@utah.edu or support@edilytics.com]        \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Computing quantification windows \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t CRISPRessoPro not installed \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Added 0 guides with flexible matching\n",
      "\tOriginal flexiguides: ['None']\n",
      "\tFound guides: []\n",
      "\tMismatch locations: [] \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Creating Folder /Users/zp083/Desktop/CRISPRMillipedeTest/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg218/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg218_unmerged_PE/CRISPResso_on_sample_ABE8e_PAX5sg_low_2 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Computing quantification windows \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t CRISPRessoPro not installed \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Added 0 guides with flexible matching\n",
      "\tOriginal flexiguides: ['None']\n",
      "\tFound guides: []\n",
      "\tMismatch locations: [] \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.3.1]                           \n",
      "[Note that as of version 2.3.0 FLASh and Trimmomatic have been replaced by fastp for read merging and trimming. Accordingly, the --flash_command and --trimmomatic_command parameters have been replaced with --fastp_command. Also, --trimmomatic_options_string has been replaced with --fastp_options_string.\n",
      "\n",
      "Also in version 2.3.2, when running CRISPRessoPooled in mixed-mode (amplicon file and genome are provided) the default behavior will be as if the --demultiplex_only_at_amplicons parameter is provided. This change means that reads and amplicons do not need to align to the exact locations.]\n",
      "       [For support contact k.clement@utah.edu or support@edilytics.com]        \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:49:\n",
      "\t Creating Folder /Users/zp083/Desktop/CRISPRMillipedeTest/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg218/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg218_unmerged_PE/CRISPResso_on_sample_ABE8e_PAX5sg_presort_2 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Computing quantification windows \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t CRISPRessoPro not installed \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Added 0 guides with flexible matching\n",
      "\tOriginal flexiguides: ['None']\n",
      "\tFound guides: []\n",
      "\tMismatch locations: [] \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.3.1]                           \n",
      "[Note that as of version 2.3.0 FLASh and Trimmomatic have been replaced by fastp for read merging and trimming. Accordingly, the --flash_command and --trimmomatic_command parameters have been replaced with --fastp_command. Also, --trimmomatic_options_string has been replaced with --fastp_options_string.\n",
      "\n",
      "Also in version 2.3.2, when running CRISPRessoPooled in mixed-mode (amplicon file and genome are provided) the default behavior will be as if the --demultiplex_only_at_amplicons parameter is provided. This change means that reads and amplicons do not need to align to the exact locations.]\n",
      "       [For support contact k.clement@utah.edu or support@edilytics.com]        \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Creating Folder /Users/zp083/Desktop/CRISPRMillipedeTest/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg218/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg218_unmerged_PE/CRISPResso_on_sample_ABE8e_PAX5sg_high_2 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Computing quantification windows \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t CRISPRessoPro not installed \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Added 0 guides with flexible matching\n",
      "\tOriginal flexiguides: ['None']\n",
      "\tFound guides: []\n",
      "\tMismatch locations: [] \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.3.1]                           \n",
      "[Note that as of version 2.3.0 FLASh and Trimmomatic have been replaced by fastp for read merging and trimming. Accordingly, the --flash_command and --trimmomatic_command parameters have been replaced with --fastp_command. Also, --trimmomatic_options_string has been replaced with --fastp_options_string.\n",
      "\n",
      "Also in version 2.3.2, when running CRISPRessoPooled in mixed-mode (amplicon file and genome are provided) the default behavior will be as if the --demultiplex_only_at_amplicons parameter is provided. This change means that reads and amplicons do not need to align to the exact locations.]\n",
      "       [For support contact k.clement@utah.edu or support@edilytics.com]        \n",
      "\n",
      "\n",
      "                               ~~~CRISPResso 2~~~                               \n",
      "        -Analysis of genome editing outcomes from deep sequencing data-         \n",
      "                                                                                \n",
      "                                        _                                       \n",
      "                                       '  )                                     \n",
      "                                       .-'                                      \n",
      "                                      (____                                     \n",
      "                                   C)|     \\                                    \n",
      "                                     \\     /                                    \n",
      "                                      \\___/                                     \n",
      "\n",
      "                           [CRISPResso version 2.3.1]                           \n",
      "[Note that as of version 2.3.0 FLASh and Trimmomatic have been replaced by fastp for read merging and trimming. Accordingly, the --flash_command and --trimmomatic_command parameters have been replaced with --fastp_command. Also, --trimmomatic_options_string has been replaced with --fastp_options_string.\n",
      "\n",
      "Also in version 2.3.2, when running CRISPRessoPooled in mixed-mode (amplicon file and genome are provided) the default behavior will be as if the --demultiplex_only_at_amplicons parameter is provided. This change means that reads and amplicons do not need to align to the exact locations.]\n",
      "       [For support contact k.clement@utah.edu or support@edilytics.com]        \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Creating Folder /Users/zp083/Desktop/CRISPRMillipedeTest/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg218/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg218_unmerged_PE/CRISPResso_on_sample_ABE8e_PAX5sg_high_3 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Creating Folder /Users/zp083/Desktop/CRISPRMillipedeTest/20231005_v1_CRISPResso2Batch_Results_q30/ABE8e_sg218/unmerged_PE/CRISPRessoBatch_on_ABE8e_sg218_unmerged_PE/CRISPResso_on_sample_ABE8e_PAX5sg_low_3 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Computing quantification windows \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t CRISPRessoPro not installed \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Added 0 guides with flexible matching\n",
      "\tOriginal flexiguides: ['None']\n",
      "\tFound guides: []\n",
      "\tMismatch locations: [] \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Computing quantification windows \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t CRISPRessoPro not installed \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Added 0 guides with flexible matching\n",
      "\tOriginal flexiguides: ['None']\n",
      "\tFound guides: []\n",
      "\tMismatch locations: [] \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Processing sequences with fastp... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Processing sequences with fastp... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Processing sequences with fastp... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Processing sequences with fastp... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Processing sequences with fastp... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Processing sequences with fastp... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:50:\n",
      "\t Processing sequences with fastp... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:51:\n",
      "\t Processing sequences with fastp... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:15:51:\n",
      "\t Processing sequences with fastp... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:01:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:04:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:05:\n",
      "\t Filtering reads with average bp quality < 30 and single bp quality < 0 and replacing bases with quality < 0 with N ... \n",
      "\n",
      "Completed in 22 seconds\n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:24:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:24:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 20 seconds\n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:26:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:26:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:26:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1267 N_CACHED_ALN: 3394 N_COMPUTED_NOTALN: 1656 N_CACHED_NOTALN: 3683 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:28:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2327 N_CACHED_ALN: 7059 N_COMPUTED_NOTALN: 2739 N_CACHED_NOTALN: 7875 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:28:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1097 N_CACHED_ALN: 2930 N_COMPUTED_NOTALN: 1779 N_CACHED_NOTALN: 4194 \n",
      "\n",
      "Completed in 25 seconds\n",
      "\n",
      "Completed in 25 seconds\n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:30:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3268 N_CACHED_ALN: 10885 N_COMPUTED_NOTALN: 3689 N_CACHED_NOTALN: 12158 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:30:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:30:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:30:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:30:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:30:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2100 N_CACHED_ALN: 6130 N_COMPUTED_NOTALN: 2965 N_CACHED_NOTALN: 8805 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:31:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4223 N_CACHED_ALN: 14827 N_COMPUTED_NOTALN: 4583 N_CACHED_NOTALN: 16367 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:32:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3021 N_CACHED_ALN: 9495 N_COMPUTED_NOTALN: 4003 N_CACHED_NOTALN: 13481 \n",
      "\n",
      "Completed in 27 seconds\n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:32:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:32:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:33:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1204 N_CACHED_ALN: 3097 N_COMPUTED_NOTALN: 1810 N_CACHED_NOTALN: 3889 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:33:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1303 N_CACHED_ALN: 3781 N_COMPUTED_NOTALN: 1698 N_CACHED_NOTALN: 3218 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:33:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5079 N_CACHED_ALN: 18807 N_COMPUTED_NOTALN: 5397 N_CACHED_NOTALN: 20717 \n",
      "\n",
      "Completed in 29 seconds\n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:34:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 3967 N_CACHED_ALN: 12938 N_COMPUTED_NOTALN: 4938 N_CACHED_NOTALN: 18157 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:34:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:34:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 28 seconds\n",
      "\n",
      "Completed in 29 seconds\n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:34:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:34:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:34:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:34:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "Completed in 29 seconds\n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:35:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1145 N_CACHED_ALN: 3206 N_COMPUTED_NOTALN: 1493 N_CACHED_NOTALN: 4156 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:35:\n",
      "\t Aligning sequences... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:35:\n",
      "\t Processing reads; N_TOT_READS: 0 N_COMPUTED_ALN: 0 N_CACHED_ALN: 0 N_COMPUTED_NOTALN: 0 N_CACHED_NOTALN: 0 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:35:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 6052 N_CACHED_ALN: 22906 N_COMPUTED_NOTALN: 6278 N_CACHED_NOTALN: 24764 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:35:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2246 N_CACHED_ALN: 6374 N_COMPUTED_NOTALN: 3092 N_CACHED_NOTALN: 8288 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:35:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2436 N_CACHED_ALN: 7899 N_COMPUTED_NOTALN: 2870 N_CACHED_NOTALN: 6795 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:35:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 4810 N_CACHED_ALN: 16396 N_COMPUTED_NOTALN: 5833 N_CACHED_NOTALN: 22961 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:37:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 6910 N_CACHED_ALN: 27064 N_COMPUTED_NOTALN: 6996 N_CACHED_NOTALN: 29030 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:37:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1102 N_CACHED_ALN: 3166 N_COMPUTED_NOTALN: 1887 N_CACHED_NOTALN: 3845 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:37:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2241 N_CACHED_ALN: 6716 N_COMPUTED_NOTALN: 2466 N_CACHED_NOTALN: 8577 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:37:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1084 N_CACHED_ALN: 2839 N_COMPUTED_NOTALN: 1847 N_CACHED_NOTALN: 4230 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:37:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3267 N_CACHED_ALN: 9876 N_COMPUTED_NOTALN: 4147 N_CACHED_NOTALN: 12710 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:37:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 5674 N_CACHED_ALN: 19994 N_COMPUTED_NOTALN: 6645 N_CACHED_NOTALN: 27687 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:37:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3544 N_CACHED_ALN: 12064 N_COMPUTED_NOTALN: 3914 N_CACHED_NOTALN: 10478 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:37:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1201 N_CACHED_ALN: 3391 N_COMPUTED_NOTALN: 1839 N_CACHED_NOTALN: 3569 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:38:\n",
      "\t Processing reads; N_TOT_READS: 10000 N_COMPUTED_ALN: 1224 N_CACHED_ALN: 3305 N_COMPUTED_NOTALN: 1933 N_CACHED_NOTALN: 3538 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:38:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 7859 N_CACHED_ALN: 31171 N_COMPUTED_NOTALN: 7798 N_CACHED_NOTALN: 33172 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:38:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3257 N_CACHED_ALN: 10237 N_COMPUTED_NOTALN: 3312 N_CACHED_NOTALN: 13194 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:39:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4180 N_CACHED_ALN: 13336 N_COMPUTED_NOTALN: 5106 N_CACHED_NOTALN: 17378 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:39:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2125 N_CACHED_ALN: 5825 N_COMPUTED_NOTALN: 3052 N_CACHED_NOTALN: 8998 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:39:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 6469 N_CACHED_ALN: 23608 N_COMPUTED_NOTALN: 7637 N_CACHED_NOTALN: 32286 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:39:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2179 N_CACHED_ALN: 6591 N_COMPUTED_NOTALN: 3161 N_CACHED_NOTALN: 8069 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:39:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4500 N_CACHED_ALN: 16411 N_COMPUTED_NOTALN: 4845 N_CACHED_NOTALN: 14244 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:40:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2340 N_CACHED_ALN: 6950 N_COMPUTED_NOTALN: 3079 N_CACHED_NOTALN: 7631 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:40:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 8900 N_CACHED_ALN: 35076 N_COMPUTED_NOTALN: 8729 N_CACHED_NOTALN: 37295 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:40:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4205 N_CACHED_ALN: 13966 N_COMPUTED_NOTALN: 4086 N_CACHED_NOTALN: 17743 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:40:\n",
      "\t Processing reads; N_TOT_READS: 20000 N_COMPUTED_ALN: 2370 N_CACHED_ALN: 6777 N_COMPUTED_NOTALN: 3283 N_CACHED_NOTALN: 7570 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:41:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 7323 N_CACHED_ALN: 27379 N_COMPUTED_NOTALN: 8398 N_CACHED_NOTALN: 36900 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:41:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5054 N_CACHED_ALN: 16871 N_COMPUTED_NOTALN: 5992 N_CACHED_NOTALN: 22083 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:41:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 2968 N_CACHED_ALN: 9017 N_COMPUTED_NOTALN: 4110 N_CACHED_NOTALN: 13905 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:41:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5527 N_CACHED_ALN: 20780 N_COMPUTED_NOTALN: 5659 N_CACHED_NOTALN: 18034 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:41:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3138 N_CACHED_ALN: 10095 N_COMPUTED_NOTALN: 4317 N_CACHED_NOTALN: 12450 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:42:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3318 N_CACHED_ALN: 10740 N_COMPUTED_NOTALN: 4214 N_CACHED_NOTALN: 11728 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:42:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5106 N_CACHED_ALN: 17558 N_COMPUTED_NOTALN: 4830 N_CACHED_NOTALN: 22506 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:42:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 8111 N_CACHED_ALN: 31222 N_COMPUTED_NOTALN: 9146 N_CACHED_NOTALN: 41521 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:42:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 10161 N_CACHED_ALN: 38998 N_COMPUTED_NOTALN: 9602 N_CACHED_NOTALN: 41239 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:43:\n",
      "\t Processing reads; N_TOT_READS: 30000 N_COMPUTED_ALN: 3357 N_CACHED_ALN: 10401 N_COMPUTED_NOTALN: 4443 N_CACHED_NOTALN: 11799 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:43:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 5994 N_CACHED_ALN: 20535 N_COMPUTED_NOTALN: 6897 N_CACHED_NOTALN: 26574 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:43:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 3727 N_CACHED_ALN: 12265 N_COMPUTED_NOTALN: 5090 N_CACHED_NOTALN: 18918 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:43:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 6400 N_CACHED_ALN: 25215 N_COMPUTED_NOTALN: 6477 N_CACHED_NOTALN: 21908 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:43:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 3990 N_CACHED_ALN: 13723 N_COMPUTED_NOTALN: 5386 N_CACHED_NOTALN: 16901 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:43:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 5963 N_CACHED_ALN: 21388 N_COMPUTED_NOTALN: 5474 N_CACHED_NOTALN: 27175 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:44:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4301 N_CACHED_ALN: 14519 N_COMPUTED_NOTALN: 5169 N_CACHED_NOTALN: 16011 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:44:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 8994 N_CACHED_ALN: 34846 N_COMPUTED_NOTALN: 10000 N_CACHED_NOTALN: 46160 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:44:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 6830 N_CACHED_ALN: 24268 N_COMPUTED_NOTALN: 7726 N_CACHED_NOTALN: 31176 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:44:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 4508 N_CACHED_ALN: 15570 N_COMPUTED_NOTALN: 6010 N_CACHED_NOTALN: 23912 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:45:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 11272 N_CACHED_ALN: 42326 N_COMPUTED_NOTALN: 10608 N_CACHED_NOTALN: 45794 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:45:\n",
      "\t Processing reads; N_TOT_READS: 40000 N_COMPUTED_ALN: 4408 N_CACHED_ALN: 14201 N_COMPUTED_NOTALN: 5485 N_CACHED_NOTALN: 15906 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:45:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 7394 N_CACHED_ALN: 29772 N_COMPUTED_NOTALN: 7412 N_CACHED_NOTALN: 25422 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:45:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 6816 N_CACHED_ALN: 25228 N_COMPUTED_NOTALN: 6131 N_CACHED_NOTALN: 31825 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:45:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 4878 N_CACHED_ALN: 17369 N_COMPUTED_NOTALN: 6402 N_CACHED_NOTALN: 21351 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:46:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5188 N_CACHED_ALN: 18398 N_COMPUTED_NOTALN: 6105 N_CACHED_NOTALN: 20309 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:46:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 9907 N_CACHED_ALN: 38344 N_COMPUTED_NOTALN: 10906 N_CACHED_NOTALN: 50843 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:46:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 5251 N_CACHED_ALN: 18884 N_COMPUTED_NOTALN: 6889 N_CACHED_NOTALN: 28976 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:46:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 7680 N_CACHED_ALN: 28077 N_COMPUTED_NOTALN: 8670 N_CACHED_NOTALN: 35573 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:47:\n",
      "\t Processing reads; N_TOT_READS: 50000 N_COMPUTED_ALN: 5361 N_CACHED_ALN: 18036 N_COMPUTED_NOTALN: 6430 N_CACHED_NOTALN: 20173 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:47:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 7695 N_CACHED_ALN: 29173 N_COMPUTED_NOTALN: 6762 N_CACHED_NOTALN: 36370 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:47:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 8258 N_CACHED_ALN: 34500 N_COMPUTED_NOTALN: 8162 N_CACHED_NOTALN: 29080 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:47:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 5668 N_CACHED_ALN: 21075 N_COMPUTED_NOTALN: 7308 N_CACHED_NOTALN: 25949 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:47:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 6097 N_CACHED_ALN: 22372 N_COMPUTED_NOTALN: 6950 N_CACHED_NOTALN: 24581 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:48:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 12770 N_CACHED_ALN: 45776 N_COMPUTED_NOTALN: 11877 N_CACHED_NOTALN: 49577 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:48:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 6036 N_CACHED_ALN: 22313 N_COMPUTED_NOTALN: 7705 N_CACHED_NOTALN: 33946 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:48:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 11095 N_CACHED_ALN: 41701 N_COMPUTED_NOTALN: 11895 N_CACHED_NOTALN: 55309 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:48:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 8550 N_CACHED_ALN: 31939 N_COMPUTED_NOTALN: 9514 N_CACHED_NOTALN: 39997 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:48:\n",
      "\t Processing reads; N_TOT_READS: 60000 N_COMPUTED_ALN: 6270 N_CACHED_ALN: 21893 N_COMPUTED_NOTALN: 7296 N_CACHED_NOTALN: 24541 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:49:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 9142 N_CACHED_ALN: 39151 N_COMPUTED_NOTALN: 8954 N_CACHED_NOTALN: 32753 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:49:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 8604 N_CACHED_ALN: 33306 N_COMPUTED_NOTALN: 7549 N_CACHED_NOTALN: 40541 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:49:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 6509 N_CACHED_ALN: 24858 N_COMPUTED_NOTALN: 8197 N_CACHED_NOTALN: 30436 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:49:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 6946 N_CACHED_ALN: 26509 N_COMPUTED_NOTALN: 7784 N_CACHED_NOTALN: 28761 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:49:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 6795 N_CACHED_ALN: 25695 N_COMPUTED_NOTALN: 8515 N_CACHED_NOTALN: 38995 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:50:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 9323 N_CACHED_ALN: 35951 N_COMPUTED_NOTALN: 10276 N_CACHED_NOTALN: 44450 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:50:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 12005 N_CACHED_ALN: 44614 N_COMPUTED_NOTALN: 12943 N_CACHED_NOTALN: 60438 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:50:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 14426 N_CACHED_ALN: 49817 N_COMPUTED_NOTALN: 12710 N_CACHED_NOTALN: 53047 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:50:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 9397 N_CACHED_ALN: 37305 N_COMPUTED_NOTALN: 8186 N_CACHED_NOTALN: 45112 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:50:\n",
      "\t Processing reads; N_TOT_READS: 70000 N_COMPUTED_ALN: 7131 N_CACHED_ALN: 25791 N_COMPUTED_NOTALN: 8151 N_CACHED_NOTALN: 28927 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:51:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 10275 N_CACHED_ALN: 43578 N_COMPUTED_NOTALN: 9834 N_CACHED_NOTALN: 36313 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:51:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 7255 N_CACHED_ALN: 28724 N_COMPUTED_NOTALN: 9093 N_CACHED_NOTALN: 34928 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:51:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 7813 N_CACHED_ALN: 30509 N_COMPUTED_NOTALN: 8706 N_CACHED_NOTALN: 32972 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:51:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 7527 N_CACHED_ALN: 29195 N_COMPUTED_NOTALN: 9412 N_CACHED_NOTALN: 43866 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:52:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 10171 N_CACHED_ALN: 39777 N_COMPUTED_NOTALN: 11023 N_CACHED_NOTALN: 49029 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:52:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 10209 N_CACHED_ALN: 41439 N_COMPUTED_NOTALN: 8786 N_CACHED_NOTALN: 49566 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:52:\n",
      "\t Processing reads; N_TOT_READS: 80000 N_COMPUTED_ALN: 7992 N_CACHED_ALN: 29824 N_COMPUTED_NOTALN: 9021 N_CACHED_NOTALN: 33163 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:52:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 11205 N_CACHED_ALN: 48118 N_COMPUTED_NOTALN: 10678 N_CACHED_NOTALN: 39999 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:53:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 13200 N_CACHED_ALN: 47443 N_COMPUTED_NOTALN: 14363 N_CACHED_NOTALN: 64994 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:53:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 8063 N_CACHED_ALN: 32558 N_COMPUTED_NOTALN: 10101 N_CACHED_NOTALN: 39278 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:53:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 15695 N_CACHED_ALN: 54236 N_COMPUTED_NOTALN: 13726 N_CACHED_NOTALN: 56343 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:53:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 11055 N_CACHED_ALN: 45504 N_COMPUTED_NOTALN: 9415 N_CACHED_NOTALN: 54026 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:53:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 8665 N_CACHED_ALN: 34631 N_COMPUTED_NOTALN: 9668 N_CACHED_NOTALN: 37036 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:53:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 8274 N_CACHED_ALN: 32770 N_COMPUTED_NOTALN: 10370 N_CACHED_NOTALN: 48586 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:54:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 11147 N_CACHED_ALN: 43179 N_COMPUTED_NOTALN: 11996 N_CACHED_NOTALN: 53678 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:54:\n",
      "\t Finished reads; N_TOT_READS: 143243 N_COMPUTED_ALN: 16268 N_CACHED_ALN: 55687 N_COMPUTED_NOTALN: 14000 N_CACHED_NOTALN: 57288 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:54:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:54:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:54:\n",
      "\t Processing reads; N_TOT_READS: 90000 N_COMPUTED_ALN: 9001 N_CACHED_ALN: 33902 N_COMPUTED_NOTALN: 10037 N_CACHED_NOTALN: 37060 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:55:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 8864 N_CACHED_ALN: 36477 N_COMPUTED_NOTALN: 10901 N_CACHED_NOTALN: 43758 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:55:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 8994 N_CACHED_ALN: 36339 N_COMPUTED_NOTALN: 11075 N_CACHED_NOTALN: 53592 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:55:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 12037 N_CACHED_ALN: 49347 N_COMPUTED_NOTALN: 10115 N_CACHED_NOTALN: 58501 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:55:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 12709 N_CACHED_ALN: 52267 N_COMPUTED_NOTALN: 11611 N_CACHED_NOTALN: 43413 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:55:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 9497 N_CACHED_ALN: 38823 N_COMPUTED_NOTALN: 10416 N_CACHED_NOTALN: 41264 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:55:\n",
      "\t Processing reads; N_TOT_READS: 150000 N_COMPUTED_ALN: 14776 N_CACHED_ALN: 51012 N_COMPUTED_NOTALN: 15217 N_CACHED_NOTALN: 68995 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:56:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 11864 N_CACHED_ALN: 46674 N_COMPUTED_NOTALN: 12950 N_CACHED_NOTALN: 58512 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:56:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 9591 N_CACHED_ALN: 40623 N_COMPUTED_NOTALN: 11711 N_CACHED_NOTALN: 48075 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:56:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 9704 N_CACHED_ALN: 40038 N_COMPUTED_NOTALN: 11777 N_CACHED_NOTALN: 58481 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:56:\n",
      "\t Processing reads; N_TOT_READS: 100000 N_COMPUTED_ALN: 9865 N_CACHED_ALN: 37982 N_COMPUTED_NOTALN: 10884 N_CACHED_NOTALN: 41269 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:57:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 12791 N_CACHED_ALN: 53364 N_COMPUTED_NOTALN: 10899 N_CACHED_NOTALN: 62946 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:57:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:57:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 10379 N_CACHED_ALN: 43136 N_COMPUTED_NOTALN: 11165 N_CACHED_NOTALN: 45320 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:57:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:57:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:57:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:57:\n",
      "\t Begin processing plots for amplicon cd19 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:57:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 13748 N_CACHED_ALN: 55627 N_COMPUTED_NOTALN: 12745 N_CACHED_NOTALN: 47880 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:58:\n",
      "\t Processing reads; N_TOT_READS: 160000 N_COMPUTED_ALN: 16000 N_CACHED_ALN: 54897 N_COMPUTED_NOTALN: 16147 N_CACHED_NOTALN: 72956 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:58:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 10418 N_CACHED_ALN: 43664 N_COMPUTED_NOTALN: 12577 N_CACHED_NOTALN: 63341 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:59:\n",
      "\t Processing reads; N_TOT_READS: 110000 N_COMPUTED_ALN: 10678 N_CACHED_ALN: 42094 N_COMPUTED_NOTALN: 11709 N_CACHED_NOTALN: 45519 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:59:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 10389 N_CACHED_ALN: 44249 N_COMPUTED_NOTALN: 12695 N_CACHED_NOTALN: 52667 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:59:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 13160 N_CACHED_ALN: 49891 N_COMPUTED_NOTALN: 13975 N_CACHED_NOTALN: 62974 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:59:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 11253 N_CACHED_ALN: 47030 N_COMPUTED_NOTALN: 12104 N_CACHED_NOTALN: 49613 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:16:59:\n",
      "\t Processing reads; N_TOT_READS: 150000 N_COMPUTED_ALN: 13993 N_CACHED_ALN: 57044 N_COMPUTED_NOTALN: 11701 N_CACHED_NOTALN: 67262 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:02:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 11275 N_CACHED_ALN: 47030 N_COMPUTED_NOTALN: 13472 N_CACHED_NOTALN: 68223 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:03:\n",
      "\t Finished reads; N_TOT_READS: 168408 N_COMPUTED_ALN: 17139 N_CACHED_ALN: 58116 N_COMPUTED_NOTALN: 17127 N_CACHED_NOTALN: 76026 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:03:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:03:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:03:\n",
      "\t Processing reads; N_TOT_READS: 120000 N_COMPUTED_ALN: 11516 N_CACHED_ALN: 46100 N_COMPUTED_NOTALN: 12583 N_CACHED_NOTALN: 49801 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:03:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 11247 N_CACHED_ALN: 47898 N_COMPUTED_NOTALN: 13661 N_CACHED_NOTALN: 57194 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:03:\n",
      "\t Processing reads; N_TOT_READS: 150000 N_COMPUTED_ALN: 14012 N_CACHED_ALN: 52691 N_COMPUTED_NOTALN: 15065 N_CACHED_NOTALN: 68232 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:04:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 15413 N_CACHED_ALN: 59686 N_COMPUTED_NOTALN: 14072 N_CACHED_NOTALN: 50829 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:04:\n",
      "\t Processing reads; N_TOT_READS: 160000 N_COMPUTED_ALN: 15178 N_CACHED_ALN: 60956 N_COMPUTED_NOTALN: 12492 N_CACHED_NOTALN: 71374 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:04:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 12174 N_CACHED_ALN: 50800 N_COMPUTED_NOTALN: 13077 N_CACHED_NOTALN: 53949 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:06:\n",
      "\t Processing reads; N_TOT_READS: 150000 N_COMPUTED_ALN: 11952 N_CACHED_ALN: 50513 N_COMPUTED_NOTALN: 14395 N_CACHED_NOTALN: 73140 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:07:\n",
      "\t Processing reads; N_TOT_READS: 170000 N_COMPUTED_ALN: 16055 N_CACHED_ALN: 63350 N_COMPUTED_NOTALN: 13524 N_CACHED_NOTALN: 77071 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:07:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 13130 N_CACHED_ALN: 54706 N_COMPUTED_NOTALN: 14041 N_CACHED_NOTALN: 58123 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:07:\n",
      "\t Processing reads; N_TOT_READS: 160000 N_COMPUTED_ALN: 15272 N_CACHED_ALN: 55286 N_COMPUTED_NOTALN: 16531 N_CACHED_NOTALN: 72911 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:07:\n",
      "\t Processing reads; N_TOT_READS: 150000 N_COMPUTED_ALN: 17197 N_CACHED_ALN: 63862 N_COMPUTED_NOTALN: 14948 N_CACHED_NOTALN: 53993 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:07:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 12120 N_CACHED_ALN: 51579 N_COMPUTED_NOTALN: 14650 N_CACHED_NOTALN: 61651 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:07:\n",
      "\t Processing reads; N_TOT_READS: 130000 N_COMPUTED_ALN: 12551 N_CACHED_ALN: 49692 N_COMPUTED_NOTALN: 13575 N_CACHED_NOTALN: 54182 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:09:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:09:\n",
      "\t Processing reads; N_TOT_READS: 160000 N_COMPUTED_ALN: 12906 N_CACHED_ALN: 53897 N_COMPUTED_NOTALN: 15293 N_CACHED_NOTALN: 77904 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:09:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:09:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:09:\n",
      "\t Processing reads; N_TOT_READS: 150000 N_COMPUTED_ALN: 14442 N_CACHED_ALN: 58549 N_COMPUTED_NOTALN: 15020 N_CACHED_NOTALN: 61989 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:09:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:09:\n",
      "\t Processing reads; N_TOT_READS: 180000 N_COMPUTED_ALN: 17717 N_CACHED_ALN: 67068 N_COMPUTED_NOTALN: 14541 N_CACHED_NOTALN: 80674 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:09:\n",
      "\t Begin processing plots for amplicon cd19 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:10:\n",
      "\t Processing reads; N_TOT_READS: 170000 N_COMPUTED_ALN: 16882 N_CACHED_ALN: 58612 N_COMPUTED_NOTALN: 17563 N_CACHED_NOTALN: 76943 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:10:\n",
      "\t Processing reads; N_TOT_READS: 160000 N_COMPUTED_ALN: 18331 N_CACHED_ALN: 68629 N_COMPUTED_NOTALN: 15957 N_CACHED_NOTALN: 57083 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:10:\n",
      "\t Processing reads; N_TOT_READS: 140000 N_COMPUTED_ALN: 13276 N_CACHED_ALN: 53408 N_COMPUTED_NOTALN: 14512 N_CACHED_NOTALN: 58804 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:12:\n",
      "\t Processing reads; N_TOT_READS: 160000 N_COMPUTED_ALN: 15307 N_CACHED_ALN: 61256 N_COMPUTED_NOTALN: 16125 N_CACHED_NOTALN: 67312 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:12:\n",
      "\t Processing reads; N_TOT_READS: 150000 N_COMPUTED_ALN: 13454 N_CACHED_ALN: 54960 N_COMPUTED_NOTALN: 15715 N_CACHED_NOTALN: 65871 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:12:\n",
      "\t Processing reads; N_TOT_READS: 190000 N_COMPUTED_ALN: 19239 N_CACHED_ALN: 70953 N_COMPUTED_NOTALN: 15264 N_CACHED_NOTALN: 84544 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:12:\n",
      "\t Finished reads; N_TOT_READS: 167822 N_COMPUTED_ALN: 19563 N_CACHED_ALN: 72349 N_COMPUTED_NOTALN: 16675 N_CACHED_NOTALN: 59235 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:12:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:12:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:13:\n",
      "\t Processing reads; N_TOT_READS: 180000 N_COMPUTED_ALN: 18154 N_CACHED_ALN: 62435 N_COMPUTED_NOTALN: 18461 N_CACHED_NOTALN: 80950 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:13:\n",
      "\t Processing reads; N_TOT_READS: 170000 N_COMPUTED_ALN: 14016 N_CACHED_ALN: 57236 N_COMPUTED_NOTALN: 16271 N_CACHED_NOTALN: 82477 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:15:\n",
      "\t Processing reads; N_TOT_READS: 200000 N_COMPUTED_ALN: 20431 N_CACHED_ALN: 75357 N_COMPUTED_NOTALN: 15971 N_CACHED_NOTALN: 88241 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:15:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:15:\n",
      "\t Processing reads; N_TOT_READS: 170000 N_COMPUTED_ALN: 16811 N_CACHED_ALN: 64621 N_COMPUTED_NOTALN: 17577 N_CACHED_NOTALN: 70991 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:16:\n",
      "\t Processing reads; N_TOT_READS: 150000 N_COMPUTED_ALN: 14438 N_CACHED_ALN: 56957 N_COMPUTED_NOTALN: 15555 N_CACHED_NOTALN: 63050 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:17:\n",
      "\t Processing reads; N_TOT_READS: 190000 N_COMPUTED_ALN: 19351 N_CACHED_ALN: 66134 N_COMPUTED_NOTALN: 19835 N_CACHED_NOTALN: 84680 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:18:\n",
      "\t Processing reads; N_TOT_READS: 160000 N_COMPUTED_ALN: 14362 N_CACHED_ALN: 58097 N_COMPUTED_NOTALN: 16786 N_CACHED_NOTALN: 70755 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:18:\n",
      "\t Finished reads; N_TOT_READS: 193379 N_COMPUTED_ALN: 19902 N_CACHED_ALN: 67489 N_COMPUTED_NOTALN: 20160 N_CACHED_NOTALN: 85828 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:18:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:18:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:19:\n",
      "\t Processing reads; N_TOT_READS: 210000 N_COMPUTED_ALN: 21783 N_CACHED_ALN: 79631 N_COMPUTED_NOTALN: 16968 N_CACHED_NOTALN: 91618 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:19:\n",
      "\t Finished reads; N_TOT_READS: 212212 N_COMPUTED_ALN: 22107 N_CACHED_ALN: 80639 N_COMPUTED_NOTALN: 17115 N_CACHED_NOTALN: 92351 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:19:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:19:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:19:\n",
      "\t Processing reads; N_TOT_READS: 180000 N_COMPUTED_ALN: 14722 N_CACHED_ALN: 59633 N_COMPUTED_NOTALN: 17376 N_CACHED_NOTALN: 88269 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:19:\n",
      "\t Processing reads; N_TOT_READS: 180000 N_COMPUTED_ALN: 18435 N_CACHED_ALN: 68380 N_COMPUTED_NOTALN: 18534 N_CACHED_NOTALN: 74651 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:21:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:21:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:22:\n",
      "\t Processing reads; N_TOT_READS: 160000 N_COMPUTED_ALN: 15687 N_CACHED_ALN: 60537 N_COMPUTED_NOTALN: 16642 N_CACHED_NOTALN: 67134 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:22:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:22:\n",
      "\t Processing reads; N_TOT_READS: 170000 N_COMPUTED_ALN: 15351 N_CACHED_ALN: 60673 N_COMPUTED_NOTALN: 18287 N_CACHED_NOTALN: 75689 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:22:\n",
      "\t Processing reads; N_TOT_READS: 190000 N_COMPUTED_ALN: 19618 N_CACHED_ALN: 72549 N_COMPUTED_NOTALN: 19449 N_CACHED_NOTALN: 78384 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:22:\n",
      "\t Begin processing plots for amplicon cd19 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:24:\n",
      "\t Processing reads; N_TOT_READS: 190000 N_COMPUTED_ALN: 15788 N_CACHED_ALN: 62330 N_COMPUTED_NOTALN: 18812 N_CACHED_NOTALN: 93070 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:25:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:25:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:25:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:25:\n",
      "\t Processing reads; N_TOT_READS: 170000 N_COMPUTED_ALN: 16411 N_CACHED_ALN: 62522 N_COMPUTED_NOTALN: 18048 N_CACHED_NOTALN: 73019 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:26:\n",
      "\t Processing reads; N_TOT_READS: 180000 N_COMPUTED_ALN: 16860 N_CACHED_ALN: 64102 N_COMPUTED_NOTALN: 19537 N_CACHED_NOTALN: 79501 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:26:\n",
      "\t Processing reads; N_TOT_READS: 200000 N_COMPUTED_ALN: 20673 N_CACHED_ALN: 76623 N_COMPUTED_NOTALN: 20778 N_CACHED_NOTALN: 81926 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:27:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:27:\n",
      "\t Begin processing plots for amplicon cd19 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:28:\n",
      "\t Finished reads; N_TOT_READS: 206462 N_COMPUTED_ALN: 21616 N_CACHED_ALN: 79431 N_COMPUTED_NOTALN: 21376 N_CACHED_NOTALN: 84039 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:28:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:28:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:29:\n",
      "\t Processing reads; N_TOT_READS: 190000 N_COMPUTED_ALN: 18253 N_CACHED_ALN: 67814 N_COMPUTED_NOTALN: 20614 N_CACHED_NOTALN: 83319 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:29:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:29:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:29:\n",
      "\t Processing reads; N_TOT_READS: 180000 N_COMPUTED_ALN: 17975 N_CACHED_ALN: 65978 N_COMPUTED_NOTALN: 19396 N_CACHED_NOTALN: 76651 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:30:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:31:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:31:\n",
      "\t Begin processing plots for amplicon cd19 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:32:\n",
      "\t Processing reads; N_TOT_READS: 200000 N_COMPUTED_ALN: 19296 N_CACHED_ALN: 71997 N_COMPUTED_NOTALN: 21550 N_CACHED_NOTALN: 87157 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:33:\n",
      "\t Processing reads; N_TOT_READS: 190000 N_COMPUTED_ALN: 19477 N_CACHED_ALN: 69738 N_COMPUTED_NOTALN: 20350 N_CACHED_NOTALN: 80435 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:35:\n",
      "\t Processing reads; N_TOT_READS: 200000 N_COMPUTED_ALN: 17267 N_CACHED_ALN: 65600 N_COMPUTED_NOTALN: 19857 N_CACHED_NOTALN: 97276 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:35:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:35:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:17:35:\n",
      "\t  <=90.0% of reads were aligned. Total reads: 143243, Aligned reads: 71955. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:17:35:\n",
      "\t  >=1.0% of reads have modifications at the start or end. Total reads: 71955, Irregular reads: 71828. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:17:35:\n",
      "\t  >=0.2% of substitutions were outside of the quantification window. Total substitutions: 278734, Substitutions outside window: 1308. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:17:35:\n",
      "\t  >=30.0% of modifications were substitutions. This could potentially indicate poor sequencing quality. Total modifications: 294765.0, Substitutions: 278734. \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:35:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                               \n",
      "                                        _                                      \n",
      "                                       '  )                                    \n",
      "                                       .-'                                     \n",
      "                                      (____                                    \n",
      "                                   C)|     \\                                   \n",
      "                                     \\     /                                   \n",
      "                                      \\___/                                    \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:36:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:36:\n",
      "\t Completed 1/9 runs \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:37:\n",
      "\t Processing reads; N_TOT_READS: 200000 N_COMPUTED_ALN: 20706 N_CACHED_ALN: 73898 N_COMPUTED_NOTALN: 21284 N_CACHED_NOTALN: 84112 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:37:\n",
      "\t Processing reads; N_TOT_READS: 210000 N_COMPUTED_ALN: 20557 N_CACHED_ALN: 75685 N_COMPUTED_NOTALN: 23028 N_CACHED_NOTALN: 90730 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:38:\n",
      "\t Finished reads; N_TOT_READS: 212410 N_COMPUTED_ALN: 20908 N_CACHED_ALN: 76668 N_COMPUTED_NOTALN: 23262 N_CACHED_NOTALN: 91572 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:38:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:38:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:40:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:42:\n",
      "\t Processing reads; N_TOT_READS: 210000 N_COMPUTED_ALN: 21884 N_CACHED_ALN: 77644 N_COMPUTED_NOTALN: 22716 N_CACHED_NOTALN: 87756 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:43:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:43:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:43:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:43:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:44:\n",
      "\t Finished reads; N_TOT_READS: 213707 N_COMPUTED_ALN: 22489 N_CACHED_ALN: 79206 N_COMPUTED_NOTALN: 23079 N_CACHED_NOTALN: 88933 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:44:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:44:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:44:\n",
      "\t Processing reads; N_TOT_READS: 210000 N_COMPUTED_ALN: 18542 N_CACHED_ALN: 69160 N_COMPUTED_NOTALN: 20765 N_CACHED_NOTALN: 101533 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:44:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:44:\n",
      "\t Begin processing plots for amplicon cd19 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:47:\n",
      "\t Processing reads; N_TOT_READS: 220000 N_COMPUTED_ALN: 19497 N_CACHED_ALN: 73096 N_COMPUTED_NOTALN: 21648 N_CACHED_NOTALN: 105759 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:48:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:48:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:48:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:49:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:49:\n",
      "\t Begin processing plots for amplicon cd19 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:53:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:53:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:53:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:55:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:55:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:55:\n",
      "\t Begin processing plots for amplicon cd19 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:17:58:\n",
      "\t Processing reads; N_TOT_READS: 230000 N_COMPUTED_ALN: 20723 N_CACHED_ALN: 76741 N_COMPUTED_NOTALN: 22982 N_CACHED_NOTALN: 109554 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:00:\n",
      "\t Finished reads; N_TOT_READS: 232256 N_COMPUTED_ALN: 21033 N_CACHED_ALN: 77682 N_COMPUTED_NOTALN: 23177 N_CACHED_NOTALN: 110364 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:00:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:00:\n",
      "\t Quantifying indels/substitutions... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:01:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:04:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:04:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:04:\n",
      "\t  <=90.0% of reads were aligned. Total reads: 167822, Aligned reads: 91912. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:04:\n",
      "\t  >=1.0% of reads have modifications at the start or end. Total reads: 91912, Irregular reads: 91750. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:04:\n",
      "\t  >=0.2% of substitutions were outside of the quantification window. Total substitutions: 311859, Substitutions outside window: 1571. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:04:\n",
      "\t  >=30.0% of modifications were substitutions. This could potentially indicate poor sequencing quality. Total modifications: 327509.0, Substitutions: 311859. \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:04:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                               \n",
      "                                        _                                      \n",
      "                                       '  )                                    \n",
      "                                       .-'                                     \n",
      "                                      (____                                    \n",
      "                                   C)|     \\                                   \n",
      "                                     \\     /                                   \n",
      "                                      \\___/                                    \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:05:\n",
      "\t Completed 2/9 runs \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:07:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:07:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:07:\n",
      "\t  <=90.0% of reads were aligned. Total reads: 168408, Aligned reads: 75255. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:07:\n",
      "\t  >=1.0% of reads have modifications at the start or end. Total reads: 75255, Irregular reads: 75132. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:07:\n",
      "\t  >=0.2% of substitutions were outside of the quantification window. Total substitutions: 297584, Substitutions outside window: 1399. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:07:\n",
      "\t  >=30.0% of modifications were substitutions. This could potentially indicate poor sequencing quality. Total modifications: 318128.0, Substitutions: 297584. \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:07:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                               \n",
      "                                        _                                      \n",
      "                                       '  )                                    \n",
      "                                       .-'                                     \n",
      "                                      (____                                    \n",
      "                                   C)|     \\                                   \n",
      "                                     \\     /                                   \n",
      "                                      \\___/                                    \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:07:\n",
      "\t Completed 3/9 runs \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:08:\n",
      "\t Calculating allele frequencies... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:08:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:08:\n",
      "\t Saving processed data... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:09:\n",
      "\t Making Plots... \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:09:\n",
      "\t Begin processing plots for amplicon cd19 \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:11:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:11:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:12:\n",
      "\t  <=90.0% of reads were aligned. Total reads: 212212, Aligned reads: 102746. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:12:\n",
      "\t  >=1.0% of reads have modifications at the start or end. Total reads: 102746, Irregular reads: 102539. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:12:\n",
      "\t  >=0.2% of substitutions were outside of the quantification window. Total substitutions: 366494, Substitutions outside window: 1758. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:12:\n",
      "\t  >=30.0% of modifications were substitutions. This could potentially indicate poor sequencing quality. Total modifications: 384667.0, Substitutions: 366494. \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:12:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                               \n",
      "                                        _                                      \n",
      "                                       '  )                                    \n",
      "                                       .-'                                     \n",
      "                                      (____                                    \n",
      "                                   C)|     \\                                   \n",
      "                                     \\     /                                   \n",
      "                                      \\___/                                    \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:13:\n",
      "\t Completed 4/9 runs \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:13:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:17:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:17:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:17:\n",
      "\t  <=90.0% of reads were aligned. Total reads: 193379, Aligned reads: 87391. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:17:\n",
      "\t  >=1.0% of reads have modifications at the start or end. Total reads: 87391, Irregular reads: 87206. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:17:\n",
      "\t  >=0.2% of substitutions were outside of the quantification window. Total substitutions: 312535, Substitutions outside window: 1772. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:17:\n",
      "\t  >=30.0% of modifications were substitutions. This could potentially indicate poor sequencing quality. Total modifications: 329669.0, Substitutions: 312535. \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:17:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                               \n",
      "                                        _                                      \n",
      "                                       '  )                                    \n",
      "                                       .-'                                     \n",
      "                                      (____                                    \n",
      "                                   C)|     \\                                   \n",
      "                                     \\     /                                   \n",
      "                                      \\___/                                    \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:18:\n",
      "\t Completed 5/9 runs \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:21:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:21:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:21:\n",
      "\t  <=90.0% of reads were aligned. Total reads: 206462, Aligned reads: 101047. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:21:\n",
      "\t  >=1.0% of reads have modifications at the start or end. Total reads: 101047, Irregular reads: 100862. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:21:\n",
      "\t  >=0.2% of substitutions were outside of the quantification window. Total substitutions: 389442, Substitutions outside window: 1771. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:21:\n",
      "\t  >=30.0% of modifications were substitutions. This could potentially indicate poor sequencing quality. Total modifications: 416365.0, Substitutions: 389442. \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:21:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                               \n",
      "                                        _                                      \n",
      "                                       '  )                                    \n",
      "                                       .-'                                     \n",
      "                                      (____                                    \n",
      "                                   C)|     \\                                   \n",
      "                                     \\     /                                   \n",
      "                                      \\___/                                    \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:21:\n",
      "\t Completed 6/9 runs \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t  <=90.0% of reads were aligned. Total reads: 213707, Aligned reads: 101695. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t  >=1.0% of reads have modifications at the start or end. Total reads: 101695, Irregular reads: 101526. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t  >=0.2% of substitutions were outside of the quantification window. Total substitutions: 337241, Substitutions outside window: 1644. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t  >=30.0% of modifications were substitutions. This could potentially indicate poor sequencing quality. Total modifications: 354682.0, Substitutions: 337241. \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                               \n",
      "                                        _                                      \n",
      "                                       '  )                                    \n",
      "                                       .-'                                     \n",
      "                                      (____                                    \n",
      "                                   C)|     \\                                   \n",
      "                                     \\     /                                   \n",
      "                                      \\___/                                    \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t  <=90.0% of reads were aligned. Total reads: 212410, Aligned reads: 97576. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t  >=1.0% of reads have modifications at the start or end. Total reads: 97576, Irregular reads: 97386. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t  >=0.2% of substitutions were outside of the quantification window. Total substitutions: 310625, Substitutions outside window: 1726. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t  >=30.0% of modifications were substitutions. This could potentially indicate poor sequencing quality. Total modifications: 337318.0, Substitutions: 310625. \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                               \n",
      "                                        _                                      \n",
      "                                       '  )                                    \n",
      "                                       .-'                                     \n",
      "                                      (____                                    \n",
      "                                   C)|     \\                                   \n",
      "                                     \\     /                                   \n",
      "                                      \\___/                                    \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t Completed 7/9 runs \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:24:\n",
      "\t Completed 8/9 runs \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t Done! \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t Removing Intermediate files... \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t  <=90.0% of reads were aligned. Total reads: 232256, Aligned reads: 98715. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t  >=1.0% of reads have modifications at the start or end. Total reads: 98715, Irregular reads: 98522. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t  >=0.2% of substitutions were outside of the quantification window. Total substitutions: 361076, Substitutions outside window: 1619. \n",
      "\n",
      "WARNING @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t  >=30.0% of modifications were substitutions. This could potentially indicate poor sequencing quality. Total modifications: 381335.0, Substitutions: 361076. \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                               \n",
      "                                        _                                      \n",
      "                                       '  )                                    \n",
      "                                       .-'                                     \n",
      "                                      (____                                    \n",
      "                                   C)|     \\                                   \n",
      "                                     \\     /                                   \n",
      "                                      \\___/                                    \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t Completed 9/9 runs \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t Finished all batches \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t Reporting summary for amplicon: \"cd19\" \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:30:\n",
      "\t All guides are equal. Performing comparison of batches for amplicon 'cd19' \n",
      "\n",
      "INFO  @ Thu, 24 Oct 2024 11:18:36:\n",
      "\t Analysis Complete! \n",
      "\n",
      "                                                                               \n",
      "                                        _                                      \n",
      "                                       '  )                                    \n",
      "                                       .-'                                     \n",
      "                                      (____                                    \n",
      "                                   C)|     \\                                   \n",
      "                                     \\     /                                   \n",
      "                                      \\___/                                    \n",
      "\n",
      "CPU times: user 2.13 s, sys: 674 ms, total: 2.81 s\n",
      "Wall time: 2min 51s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "for _, validation_guide_row in CD19_guide_validation_sequences_df.iterrows():\n",
    "    validation_name = validation_guide_row[\"editor\"] + \"_sg\" + validation_guide_row[\"sgID\"]\n",
    "    crispresso_name = validation_guide_row[\"crispresso_name\"]\n",
    "    \n",
    "    validation_names_enumeration = [(index, name) for index, name in enumerate(names) if crispresso_name in name]\n",
    "    if len(validation_names_enumeration) > 0:\n",
    "        print(\"Processing \" + validation_name)\n",
    "        validate_results_output_dir = \"./20231005_v1_CRISPResso2Batch_Results_q30/\" + validation_name\n",
    "\n",
    "        unmerged_PE_results_dir = validate_results_output_dir + \"/unmerged_PE\"\n",
    "\n",
    "        os.makedirs(unmerged_PE_results_dir)\n",
    "\n",
    "        unmerged_PE_settings_fn = 'Davide_CD19_enhancer_BE_screen_{}_unmerged_PE.settings'.format(validation_name)\n",
    "\n",
    "        #ABE8e_pooled_unmerged_PE_middle\n",
    "        !CRISPRessoBatch -bs {settings_output_dir + \"/\" + unmerged_PE_settings_fn} -a {pooled_amplicon_sequence} \\\n",
    "        -an cd19 -q 30 \\\n",
    "        --exclude_bp_from_left 3 \\\n",
    "        --exclude_bp_from_right 3 --no_rerun -n {validation_name + \"_unmerged_PE\"} \\\n",
    "        --min_frequency_alleles_around_cut_to_plot 0.001 --max_rows_alleles_around_cut_to_plot 500 -p 64  \\\n",
    "        --plot_window_size 20 --base_editor_output -w 0 -bo {unmerged_PE_results_dir}\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
